<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Alexander Margetts" />

<meta name="date" content="2023-10-19" />

<title>Microglial METH IVSA DeSeq2 RNA-Sequencing Analysis – Samples Removed – Final Product</title>

<script src="site_libs/header-attrs-2.23/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Methamphetamine_MicroglialRNASequencing_Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/avm27/Methamphetamine_MicroglialRNASequencing_Analysis">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Microglial METH IVSA DeSeq2 RNA-Sequencing
Analysis – Samples Removed – Final Product</h1>
<h4 class="author">Alexander Margetts</h4>
<h4 class="date">2023-10-19</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2023-10-19
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong>
<code>Methamphetamine_MicroglialRNASequencing_Analysis/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231019code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231019)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231019code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231019)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomavm27MethamphetamineMicroglialRNASequencingAnalysistreee92ae3ee9535578dff07df5fd59852c6c0241b87targetblanke92ae3ea">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/avm27/Methamphetamine_MicroglialRNASequencing_Analysis/tree/e92ae3ee9535578dff07df5fd59852c6c0241b87" target="_blank">e92ae3e</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomavm27MethamphetamineMicroglialRNASequencingAnalysistreee92ae3ee9535578dff07df5fd59852c6c0241b87targetblanke92ae3ea"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/avm27/Methamphetamine_MicroglialRNASequencing_Analysis/tree/e92ae3ee9535578dff07df5fd59852c6c0241b87" target="_blank">e92ae3e</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
working directory clean
</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown
(<code>analysis/MIVSA_RAnalysis_WorkFlowR.Rmd</code>) and HTML
(<code>docs/MIVSA_RAnalysis_WorkFlowR.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/avm27/Methamphetamine_MicroglialRNASequencing_Analysis/blob/e92ae3ee9535578dff07df5fd59852c6c0241b87/analysis/MIVSA_RAnalysis_WorkFlowR.Rmd" target="_blank">e92ae3e</a>
</td>
<td>
avm27
</td>
<td>
2023-10-19
</td>
<td>
Quickstart commit from wflow_quickstart()
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="methamphetamine-ivsa" class="section level1">
<h1>Methamphetamine IVSA</h1>
<p>Male C57BL/6J mice underwent methamphetamine intravenous
self-administration (METH IVSA). The striatum was dissected, and
microglia were isolated immediately following the last maintenance
session (Maintenance and Saline) or following 21 days of forced
home-cage abstinence (Abstinence). The samples were then processed for
RNA-sequencing. The subsequent analysis is the result of this
RNA-sequencing experiment.</p>
</div>
<div id="deseq2-analysis" class="section level1">
<h1>DESeq2 Analysis</h1>
<div id="setup" class="section level2">
<h2>Setup</h2>
<p>Building DESeq2 input files from gene count matrices output by
StringTie following removal of samples that are outliers.</p>
<p>Running DeSeq2 analysis with factor levels of Saline and
Maintenance</p>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/DESeq2%20Analysis-1.png" width="672" style="display: block; margin: auto;" />
# Plot Estimates and Gather Results</p>
</div>
<div id="gather-results" class="section level2 tabset">
<h2 class="tabset">Gather Results</h2>
<p>These results include overview for each result file and dispersion
estimates based on count values.</p>
<div id="maintenance-vs-saline" class="section level3">
<h3>Maintenance vs Saline</h3>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:1000px; ">
<table class="table table-striped table-hover table-condensed table-responsive" style="font-size: 10px; font-family: Arial; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Maintenance vs Saline Microglia Results Overview
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
baseMean
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
log2FoldChange
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
lfcSE
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
stat
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
padj
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Sox17|Sox17
</td>
<td style="text-align:left;">
1005.050408
</td>
<td style="text-align:left;">
-0.2892391
</td>
<td style="text-align:left;">
0.3824676
</td>
<td style="text-align:left;">
-0.7562447
</td>
<td style="text-align:left;">
0.4495025
</td>
<td style="text-align:left;">
0.8622696
</td>
</tr>
<tr>
<td style="text-align:left;">
Xkr4|Xkr4
</td>
<td style="text-align:left;">
27.487221
</td>
<td style="text-align:left;">
2.1094096
</td>
<td style="text-align:left;">
1.7498633
</td>
<td style="text-align:left;">
1.2054711
</td>
<td style="text-align:left;">
0.2280215
</td>
<td style="text-align:left;">
0.6912576
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm39585|Gm39585
</td>
<td style="text-align:left;">
20.122567
</td>
<td style="text-align:left;">
-1.4161909
</td>
<td style="text-align:left;">
1.9739949
</td>
<td style="text-align:left;">
-0.7174238
</td>
<td style="text-align:left;">
0.4731126
</td>
<td style="text-align:left;">
0.8743361
</td>
</tr>
<tr>
<td style="text-align:left;">
Mrpl15|Mrpl15
</td>
<td style="text-align:left;">
2772.826990
</td>
<td style="text-align:left;">
0.0920304
</td>
<td style="text-align:left;">
0.1493648
</td>
<td style="text-align:left;">
0.6161448
</td>
<td style="text-align:left;">
0.5377989
</td>
<td style="text-align:left;">
0.9043487
</td>
</tr>
<tr>
<td style="text-align:left;">
Lypla1|Lypla1
</td>
<td style="text-align:left;">
8825.610732
</td>
<td style="text-align:left;">
0.1191790
</td>
<td style="text-align:left;">
0.1002528
</td>
<td style="text-align:left;">
1.1887856
</td>
<td style="text-align:left;">
0.2345240
</td>
<td style="text-align:left;">
0.6999918
</td>
</tr>
<tr>
<td style="text-align:left;">
Tcea1|Tcea1
</td>
<td style="text-align:left;">
8191.296553
</td>
<td style="text-align:left;">
0.1704652
</td>
<td style="text-align:left;">
0.0630445
</td>
<td style="text-align:left;">
2.7038851
</td>
<td style="text-align:left;">
0.0068534
</td>
<td style="text-align:left;">
0.1300812
</td>
</tr>
<tr>
<td style="text-align:left;">
Rgs20|Rgs20
</td>
<td style="text-align:left;">
127.389956
</td>
<td style="text-align:left;">
0.6658835
</td>
<td style="text-align:left;">
0.7526546
</td>
<td style="text-align:left;">
0.8847132
</td>
<td style="text-align:left;">
0.3763113
</td>
<td style="text-align:left;">
0.8194094
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm16041|Gm16041
</td>
<td style="text-align:left;">
0.000000
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
Oprk1|Oprk1
</td>
<td style="text-align:left;">
13.475225
</td>
<td style="text-align:left;">
3.5221169
</td>
<td style="text-align:left;">
2.8760340
</td>
<td style="text-align:left;">
1.2246437
</td>
<td style="text-align:left;">
0.2207095
</td>
<td style="text-align:left;">
0.6831799
</td>
</tr>
<tr>
<td style="text-align:left;">
Atp6v1h|Atp6v1h
</td>
<td style="text-align:left;">
6521.251908
</td>
<td style="text-align:left;">
0.0229262
</td>
<td style="text-align:left;">
0.0875939
</td>
<td style="text-align:left;">
0.2617323
</td>
<td style="text-align:left;">
0.7935278
</td>
<td style="text-align:left;">
0.9727107
</td>
</tr>
<tr>
<td style="text-align:left;">
St18|St18
</td>
<td style="text-align:left;">
209.697800
</td>
<td style="text-align:left;">
0.3202529
</td>
<td style="text-align:left;">
0.6296535
</td>
<td style="text-align:left;">
0.5086177
</td>
<td style="text-align:left;">
0.6110203
</td>
<td style="text-align:left;">
0.9283226
</td>
</tr>
<tr>
<td style="text-align:left;">
Rb1cc1|Rb1cc1
</td>
<td style="text-align:left;">
4516.928293
</td>
<td style="text-align:left;">
-0.2118859
</td>
<td style="text-align:left;">
0.3319474
</td>
<td style="text-align:left;">
-0.6383117
</td>
<td style="text-align:left;">
0.5232708
</td>
<td style="text-align:left;">
0.8983380
</td>
</tr>
<tr>
<td style="text-align:left;">
4732440D04Rik|4732440D04Rik
</td>
<td style="text-align:left;">
174.005530
</td>
<td style="text-align:left;">
0.1897164
</td>
<td style="text-align:left;">
0.6386101
</td>
<td style="text-align:left;">
0.2970771
</td>
<td style="text-align:left;">
0.7664077
</td>
<td style="text-align:left;">
0.9688947
</td>
</tr>
<tr>
<td style="text-align:left;">
Pcmtd1|Pcmtd1
</td>
<td style="text-align:left;">
5252.070527
</td>
<td style="text-align:left;">
0.0051816
</td>
<td style="text-align:left;">
0.1197234
</td>
<td style="text-align:left;">
0.0432800
</td>
<td style="text-align:left;">
0.9654784
</td>
<td style="text-align:left;">
0.9983380
</td>
</tr>
<tr>
<td style="text-align:left;">
Mybl1|Mybl1
</td>
<td style="text-align:left;">
8.488892
</td>
<td style="text-align:left;">
5.8571818
</td>
<td style="text-align:left;">
3.8759977
</td>
<td style="text-align:left;">
1.5111417
</td>
<td style="text-align:left;">
0.1307523
</td>
<td style="text-align:left;">
0.5647701
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm39590|Gm39590
</td>
<td style="text-align:left;">
1.865610
</td>
<td style="text-align:left;">
-0.0586868
</td>
<td style="text-align:left;">
3.9771414
</td>
<td style="text-align:left;">
-0.0147560
</td>
<td style="text-align:left;">
0.9882268
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
Vcpip1|Vcpip1
</td>
<td style="text-align:left;">
4388.265061
</td>
<td style="text-align:left;">
0.0465268
</td>
<td style="text-align:left;">
0.1355168
</td>
<td style="text-align:left;">
0.3433285
</td>
<td style="text-align:left;">
0.7313513
</td>
<td style="text-align:left;">
0.9626087
</td>
</tr>
<tr>
<td style="text-align:left;">
1700034P13Rik|1700034P13Rik
</td>
<td style="text-align:left;">
13.695416
</td>
<td style="text-align:left;">
-0.6166189
</td>
<td style="text-align:left;">
0.8899140
</td>
<td style="text-align:left;">
-0.6928972
</td>
<td style="text-align:left;">
0.4883741
</td>
<td style="text-align:left;">
0.8802916
</td>
</tr>
<tr>
<td style="text-align:left;">
Adhfe1|Adhfe1
</td>
<td style="text-align:left;">
267.004148
</td>
<td style="text-align:left;">
0.0432398
</td>
<td style="text-align:left;">
0.3663710
</td>
<td style="text-align:left;">
0.1180218
</td>
<td style="text-align:left;">
0.9060504
</td>
<td style="text-align:left;">
0.9938121
</td>
</tr>
<tr>
<td style="text-align:left;">
Rrs1|Rrs1
</td>
<td style="text-align:left;">
1345.749254
</td>
<td style="text-align:left;">
-0.6954956
</td>
<td style="text-align:left;">
0.1888328
</td>
<td style="text-align:left;">
-3.6831298
</td>
<td style="text-align:left;">
0.0002304
</td>
<td style="text-align:left;">
0.0127600
</td>
</tr>
</tbody>
</table>
</div>
<p>out of 21824 with nonzero total read count adjusted p-value &lt; 0.1
LFC &gt; 0 (up) : 489, 2.2% LFC &lt; 0 (down) : 311, 1.4% outliers [1] :
675, 3.1% low counts [2] : 3936, 18% (mean count &lt; 4) [1] see
‘cooksCutoff’ argument of ?results [2] see ‘independentFiltering’
argument of ?results</p>
</div>
<div id="abstinence-vs-saline" class="section level3">
<h3>Abstinence vs Saline</h3>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:1000px; ">
<table class="table table-striped table-hover table-condensed table-responsive" style="font-size: 10px; font-family: Arial; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Abstinence vs Saline Microglia Results Overview
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
baseMean
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
log2FoldChange
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
lfcSE
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
stat
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
padj
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Sox17|Sox17
</td>
<td style="text-align:left;">
1005.050408
</td>
<td style="text-align:left;">
-0.4706693
</td>
<td style="text-align:left;">
0.3540696
</td>
<td style="text-align:left;">
-1.3293128
</td>
<td style="text-align:left;">
0.1837448
</td>
<td style="text-align:left;">
0.8044312
</td>
</tr>
<tr>
<td style="text-align:left;">
Xkr4|Xkr4
</td>
<td style="text-align:left;">
27.487221
</td>
<td style="text-align:left;">
2.2910497
</td>
<td style="text-align:left;">
1.6213699
</td>
<td style="text-align:left;">
1.4130333
</td>
<td style="text-align:left;">
0.1576459
</td>
<td style="text-align:left;">
0.7669589
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm39585|Gm39585
</td>
<td style="text-align:left;">
20.122567
</td>
<td style="text-align:left;">
-1.1661676
</td>
<td style="text-align:left;">
1.8248782
</td>
<td style="text-align:left;">
-0.6390386
</td>
<td style="text-align:left;">
0.5227978
</td>
<td style="text-align:left;">
0.9927696
</td>
</tr>
<tr>
<td style="text-align:left;">
Mrpl15|Mrpl15
</td>
<td style="text-align:left;">
2772.826990
</td>
<td style="text-align:left;">
0.1660281
</td>
<td style="text-align:left;">
0.1382171
</td>
<td style="text-align:left;">
1.2012119
</td>
<td style="text-align:left;">
0.2296690
</td>
<td style="text-align:left;">
0.8547000
</td>
</tr>
<tr>
<td style="text-align:left;">
Lypla1|Lypla1
</td>
<td style="text-align:left;">
8825.610732
</td>
<td style="text-align:left;">
0.0335897
</td>
<td style="text-align:left;">
0.0928040
</td>
<td style="text-align:left;">
0.3619429
</td>
<td style="text-align:left;">
0.7173947
</td>
<td style="text-align:left;">
0.9990707
</td>
</tr>
<tr>
<td style="text-align:left;">
Tcea1|Tcea1
</td>
<td style="text-align:left;">
8191.296553
</td>
<td style="text-align:left;">
-0.0661473
</td>
<td style="text-align:left;">
0.0583806
</td>
<td style="text-align:left;">
-1.1330362
</td>
<td style="text-align:left;">
0.2571991
</td>
<td style="text-align:left;">
0.8820779
</td>
</tr>
<tr>
<td style="text-align:left;">
Rgs20|Rgs20
</td>
<td style="text-align:left;">
127.389956
</td>
<td style="text-align:left;">
1.0565561
</td>
<td style="text-align:left;">
0.6964308
</td>
<td style="text-align:left;">
1.5171013
</td>
<td style="text-align:left;">
0.1292411
</td>
<td style="text-align:left;">
0.7248640
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm16041|Gm16041
</td>
<td style="text-align:left;">
0.000000
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
Oprk1|Oprk1
</td>
<td style="text-align:left;">
13.475225
</td>
<td style="text-align:left;">
7.2944794
</td>
<td style="text-align:left;">
2.6556624
</td>
<td style="text-align:left;">
2.7467646
</td>
<td style="text-align:left;">
0.0060186
</td>
<td style="text-align:left;">
0.1787199
</td>
</tr>
<tr>
<td style="text-align:left;">
Atp6v1h|Atp6v1h
</td>
<td style="text-align:left;">
6521.251908
</td>
<td style="text-align:left;">
-0.0021371
</td>
<td style="text-align:left;">
0.0810617
</td>
<td style="text-align:left;">
-0.0263641
</td>
<td style="text-align:left;">
0.9789669
</td>
<td style="text-align:left;">
0.9990707
</td>
</tr>
<tr>
<td style="text-align:left;">
St18|St18
</td>
<td style="text-align:left;">
209.697800
</td>
<td style="text-align:left;">
0.4939166
</td>
<td style="text-align:left;">
0.5827141
</td>
<td style="text-align:left;">
0.8476139
</td>
<td style="text-align:left;">
0.3966530
</td>
<td style="text-align:left;">
0.9560401
</td>
</tr>
<tr>
<td style="text-align:left;">
Rb1cc1|Rb1cc1
</td>
<td style="text-align:left;">
4516.928293
</td>
<td style="text-align:left;">
-0.2426487
</td>
<td style="text-align:left;">
0.3073065
</td>
<td style="text-align:left;">
-0.7895983
</td>
<td style="text-align:left;">
0.4297624
</td>
<td style="text-align:left;">
0.9668337
</td>
</tr>
<tr>
<td style="text-align:left;">
4732440D04Rik|4732440D04Rik
</td>
<td style="text-align:left;">
174.005530
</td>
<td style="text-align:left;">
0.3296424
</td>
<td style="text-align:left;">
0.5909528
</td>
<td style="text-align:left;">
0.5578152
</td>
<td style="text-align:left;">
0.5769706
</td>
<td style="text-align:left;">
0.9965434
</td>
</tr>
<tr>
<td style="text-align:left;">
Pcmtd1|Pcmtd1
</td>
<td style="text-align:left;">
5252.070527
</td>
<td style="text-align:left;">
0.1878934
</td>
<td style="text-align:left;">
0.1107750
</td>
<td style="text-align:left;">
1.6961714
</td>
<td style="text-align:left;">
0.0898534
</td>
<td style="text-align:left;">
0.6518587
</td>
</tr>
<tr>
<td style="text-align:left;">
Mybl1|Mybl1
</td>
<td style="text-align:left;">
8.488892
</td>
<td style="text-align:left;">
5.9663637
</td>
<td style="text-align:left;">
3.6034746
</td>
<td style="text-align:left;">
1.6557252
</td>
<td style="text-align:left;">
0.0977775
</td>
<td style="text-align:left;">
0.6681910
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm39590|Gm39590
</td>
<td style="text-align:left;">
1.865610
</td>
<td style="text-align:left;">
4.5165793
</td>
<td style="text-align:left;">
3.6086749
</td>
<td style="text-align:left;">
1.2515894
</td>
<td style="text-align:left;">
0.2107195
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
Vcpip1|Vcpip1
</td>
<td style="text-align:left;">
4388.265061
</td>
<td style="text-align:left;">
0.2808433
</td>
<td style="text-align:left;">
0.1253862
</td>
<td style="text-align:left;">
2.2398268
</td>
<td style="text-align:left;">
0.0251022
</td>
<td style="text-align:left;">
0.3842919
</td>
</tr>
<tr>
<td style="text-align:left;">
1700034P13Rik|1700034P13Rik
</td>
<td style="text-align:left;">
13.695416
</td>
<td style="text-align:left;">
-0.5085417
</td>
<td style="text-align:left;">
0.8194923
</td>
<td style="text-align:left;">
-0.6205570
</td>
<td style="text-align:left;">
0.5348911
</td>
<td style="text-align:left;">
0.9944680
</td>
</tr>
<tr>
<td style="text-align:left;">
Adhfe1|Adhfe1
</td>
<td style="text-align:left;">
267.004148
</td>
<td style="text-align:left;">
0.6784805
</td>
<td style="text-align:left;">
0.3382766
</td>
<td style="text-align:left;">
2.0056976
</td>
<td style="text-align:left;">
0.0448885
</td>
<td style="text-align:left;">
0.5085780
</td>
</tr>
<tr>
<td style="text-align:left;">
Rrs1|Rrs1
</td>
<td style="text-align:left;">
1345.749254
</td>
<td style="text-align:left;">
-0.2018134
</td>
<td style="text-align:left;">
0.1744082
</td>
<td style="text-align:left;">
-1.1571327
</td>
<td style="text-align:left;">
0.2472181
</td>
<td style="text-align:left;">
0.8747460
</td>
</tr>
</tbody>
</table>
</div>
<p>out of 21824 with nonzero total read count adjusted p-value &lt; 0.1
LFC &gt; 0 (up) : 295, 1.4% LFC &lt; 0 (down) : 107, 0.49% outliers [1]
: 675, 3.1% low counts [2] : 3320, 15% (mean count &lt; 2) [1] see
‘cooksCutoff’ argument of ?results [2] see ‘independentFiltering’
argument of ?results</p>
</div>
<div id="abstinence-vs-maintenance" class="section level3">
<h3>Abstinence vs Maintenance</h3>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:1000px; ">
<table class="table table-striped table-hover table-condensed table-responsive" style="font-size: 10px; font-family: Arial; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Abstinence vs Maintenance Microglia Results Overview
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
baseMean
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
log2FoldChange
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
lfcSE
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
stat
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
padj
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Sox17|Sox17
</td>
<td style="text-align:left;">
1005.050408
</td>
<td style="text-align:left;">
-0.1814295
</td>
<td style="text-align:left;">
0.3541645
</td>
<td style="text-align:left;">
-0.5122747
</td>
<td style="text-align:left;">
0.6084588
</td>
<td style="text-align:left;">
0.9453446
</td>
</tr>
<tr>
<td style="text-align:left;">
Xkr4|Xkr4
</td>
<td style="text-align:left;">
27.487221
</td>
<td style="text-align:left;">
0.1816497
</td>
<td style="text-align:left;">
1.6071314
</td>
<td style="text-align:left;">
0.1130273
</td>
<td style="text-align:left;">
0.9100089
</td>
<td style="text-align:left;">
0.9948756
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm39585|Gm39585
</td>
<td style="text-align:left;">
20.122567
</td>
<td style="text-align:left;">
0.2500280
</td>
<td style="text-align:left;">
1.8305919
</td>
<td style="text-align:left;">
0.1365832
</td>
<td style="text-align:left;">
0.8913603
</td>
<td style="text-align:left;">
0.9927533
</td>
</tr>
<tr>
<td style="text-align:left;">
Mrpl15|Mrpl15
</td>
<td style="text-align:left;">
2772.826990
</td>
<td style="text-align:left;">
0.0739978
</td>
<td style="text-align:left;">
0.1381566
</td>
<td style="text-align:left;">
0.5356085
</td>
<td style="text-align:left;">
0.5922291
</td>
<td style="text-align:left;">
0.9399536
</td>
</tr>
<tr>
<td style="text-align:left;">
Lypla1|Lypla1
</td>
<td style="text-align:left;">
8825.610732
</td>
<td style="text-align:left;">
-0.0855892
</td>
<td style="text-align:left;">
0.0927717
</td>
<td style="text-align:left;">
-0.9225791
</td>
<td style="text-align:left;">
0.3562266
</td>
<td style="text-align:left;">
0.8326814
</td>
</tr>
<tr>
<td style="text-align:left;">
Tcea1|Tcea1
</td>
<td style="text-align:left;">
8191.296553
</td>
<td style="text-align:left;">
-0.2366124
</td>
<td style="text-align:left;">
0.0583112
</td>
<td style="text-align:left;">
-4.0577556
</td>
<td style="text-align:left;">
0.0000495
</td>
<td style="text-align:left;">
0.0035076
</td>
</tr>
<tr>
<td style="text-align:left;">
Rgs20|Rgs20
</td>
<td style="text-align:left;">
127.389956
</td>
<td style="text-align:left;">
0.3906746
</td>
<td style="text-align:left;">
0.6948027
</td>
<td style="text-align:left;">
0.5622813
</td>
<td style="text-align:left;">
0.5739244
</td>
<td style="text-align:left;">
0.9336932
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm16041|Gm16041
</td>
<td style="text-align:left;">
0.000000
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
Oprk1|Oprk1
</td>
<td style="text-align:left;">
13.475225
</td>
<td style="text-align:left;">
3.7723620
</td>
<td style="text-align:left;">
2.5322852
</td>
<td style="text-align:left;">
1.4897066
</td>
<td style="text-align:left;">
0.1363014
</td>
<td style="text-align:left;">
0.5847465
</td>
</tr>
<tr>
<td style="text-align:left;">
Atp6v1h|Atp6v1h
</td>
<td style="text-align:left;">
6521.251908
</td>
<td style="text-align:left;">
-0.0250632
</td>
<td style="text-align:left;">
0.0810397
</td>
<td style="text-align:left;">
-0.3092711
</td>
<td style="text-align:left;">
0.7571153
</td>
<td style="text-align:left;">
0.9728842
</td>
</tr>
<tr>
<td style="text-align:left;">
St18|St18
</td>
<td style="text-align:left;">
209.697800
</td>
<td style="text-align:left;">
0.1736652
</td>
<td style="text-align:left;">
0.5821876
</td>
<td style="text-align:left;">
0.2982977
</td>
<td style="text-align:left;">
0.7654760
</td>
<td style="text-align:left;">
0.9746060
</td>
</tr>
<tr>
<td style="text-align:left;">
Rb1cc1|Rb1cc1
</td>
<td style="text-align:left;">
4516.928293
</td>
<td style="text-align:left;">
-0.0307621
</td>
<td style="text-align:left;">
0.3073236
</td>
<td style="text-align:left;">
-0.1000966
</td>
<td style="text-align:left;">
0.9202676
</td>
<td style="text-align:left;">
0.9954082
</td>
</tr>
<tr>
<td style="text-align:left;">
4732440D04Rik|4732440D04Rik
</td>
<td style="text-align:left;">
174.005530
</td>
<td style="text-align:left;">
0.1399275
</td>
<td style="text-align:left;">
0.5905605
</td>
<td style="text-align:left;">
0.2369402
</td>
<td style="text-align:left;">
0.8127032
</td>
<td style="text-align:left;">
0.9833817
</td>
</tr>
<tr>
<td style="text-align:left;">
Pcmtd1|Pcmtd1
</td>
<td style="text-align:left;">
5252.070527
</td>
<td style="text-align:left;">
0.1827118
</td>
<td style="text-align:left;">
0.1107588
</td>
<td style="text-align:left;">
1.6496379
</td>
<td style="text-align:left;">
0.0990170
</td>
<td style="text-align:left;">
0.5148197
</td>
</tr>
<tr>
<td style="text-align:left;">
Mybl1|Mybl1
</td>
<td style="text-align:left;">
8.488892
</td>
<td style="text-align:left;">
0.1092359
</td>
<td style="text-align:left;">
3.4915286
</td>
<td style="text-align:left;">
0.0312860
</td>
<td style="text-align:left;">
0.9750415
</td>
<td style="text-align:left;">
0.9982936
</td>
</tr>
<tr>
<td style="text-align:left;">
Gm39590|Gm39590
</td>
<td style="text-align:left;">
1.865610
</td>
<td style="text-align:left;">
4.5752107
</td>
<td style="text-align:left;">
3.6086749
</td>
<td style="text-align:left;">
1.2678368
</td>
<td style="text-align:left;">
0.2048563
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
Vcpip1|Vcpip1
</td>
<td style="text-align:left;">
4388.265061
</td>
<td style="text-align:left;">
0.2343167
</td>
<td style="text-align:left;">
0.1253561
</td>
<td style="text-align:left;">
1.8692089
</td>
<td style="text-align:left;">
0.0615938
</td>
<td style="text-align:left;">
0.4157637
</td>
</tr>
<tr>
<td style="text-align:left;">
1700034P13Rik|1700034P13Rik
</td>
<td style="text-align:left;">
13.695416
</td>
<td style="text-align:left;">
0.1080784
</td>
<td style="text-align:left;">
0.8269067
</td>
<td style="text-align:left;">
0.1307021
</td>
<td style="text-align:left;">
0.8960110
</td>
<td style="text-align:left;">
0.9934490
</td>
</tr>
<tr>
<td style="text-align:left;">
Adhfe1|Adhfe1
</td>
<td style="text-align:left;">
267.004148
</td>
<td style="text-align:left;">
0.6352412
</td>
<td style="text-align:left;">
0.3380731
</td>
<td style="text-align:left;">
1.8790055
</td>
<td style="text-align:left;">
0.0602437
</td>
<td style="text-align:left;">
0.4126802
</td>
</tr>
<tr>
<td style="text-align:left;">
Rrs1|Rrs1
</td>
<td style="text-align:left;">
1345.749254
</td>
<td style="text-align:left;">
0.4936823
</td>
<td style="text-align:left;">
0.1748608
</td>
<td style="text-align:left;">
2.8232870
</td>
<td style="text-align:left;">
0.0047534
</td>
<td style="text-align:left;">
0.0925227
</td>
</tr>
</tbody>
</table>
</div>
<p>out of 21824 with nonzero total read count adjusted p-value &lt; 0.1
LFC &gt; 0 (up) : 465, 2.1% LFC &lt; 0 (down) : 481, 2.2% outliers [1] :
675, 3.1% low counts [2] : 3936, 18% (mean count &lt; 4) [1] see
‘cooksCutoff’ argument of ?results [2] see ‘independentFiltering’
argument of ?results</p>
</div>
</div>
<div id="annotate-results-files-and-pull-significant-degs"
class="section level2 tabset">
<h2 class="tabset">Annotate Results Files and Pull Significant DEGs</h2>
<p>This subsection is for annotation the results files to include all
necessary information for downstream analyses.</p>
<p>After annotation I subset the results to only view the significantly
differentially expressed genes with an adjusted p-value &gt; 0.05.</p>
<pre class="r"><code>## Subset for significant genes only

results_sig &lt;- subset(res, padj &lt; 0.05)
results_sig2 &lt;- subset(res2, padj &lt; 0.05)
results_sig3 &lt;- subset(res3, padj &lt; 0.05)</code></pre>
<div id="maintenance-vs-saline-1" class="section level3">
<h3>Maintenance vs Saline</h3>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:1000px; ">
<table class="table table-striped table-hover table-condensed table-responsive" style="font-size: 10px; font-family: Arial; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Significant DEGs Maintenance vs Saline
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
baseMean
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
log2FoldChange
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
lfcSE
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
stat
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
padj
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
symbol
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
entrez
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ensembl
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Rrs1|Rrs1
</td>
<td style="text-align:left;">
1345.74925
</td>
<td style="text-align:left;">
-0.6954956
</td>
<td style="text-align:left;">
0.1888328
</td>
<td style="text-align:left;">
-3.683130
</td>
<td style="text-align:left;">
0.0002304
</td>
<td style="text-align:left;">
0.0127600
</td>
<td style="text-align:left;">
Rrs1
</td>
<td style="text-align:left;">
ribosome biogenesis regulator 1
</td>
<td style="text-align:left;">
59014
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Sgk3|Sgk3
</td>
<td style="text-align:left;">
8049.66688
</td>
<td style="text-align:left;">
0.5236573
</td>
<td style="text-align:left;">
0.1177588
</td>
<td style="text-align:left;">
4.446864
</td>
<td style="text-align:left;">
0.0000087
</td>
<td style="text-align:left;">
0.0009160
</td>
<td style="text-align:left;">
Sgk3
</td>
<td style="text-align:left;">
serum/glucocorticoid regulated kinase 3
</td>
<td style="text-align:left;">
170755
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Sulf1|Sulf1
</td>
<td style="text-align:left;">
50.66850
</td>
<td style="text-align:left;">
21.8892776
</td>
<td style="text-align:left;">
2.7822808
</td>
<td style="text-align:left;">
7.867386
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
Sulf1
</td>
<td style="text-align:left;">
sulfatase 1
</td>
<td style="text-align:left;">
240725
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Phf3|Phf3
</td>
<td style="text-align:left;">
1225.06289
</td>
<td style="text-align:left;">
0.8004524
</td>
<td style="text-align:left;">
0.1939269
</td>
<td style="text-align:left;">
4.127599
</td>
<td style="text-align:left;">
0.0000367
</td>
<td style="text-align:left;">
0.0030358
</td>
<td style="text-align:left;">
Phf3
</td>
<td style="text-align:left;">
PHD finger protein 3
</td>
<td style="text-align:left;">
213109
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Ptp4a1|Ptp4a1
</td>
<td style="text-align:left;">
10992.32954
</td>
<td style="text-align:left;">
0.2390088
</td>
<td style="text-align:left;">
0.0701230
</td>
<td style="text-align:left;">
3.408424
</td>
<td style="text-align:left;">
0.0006534
</td>
<td style="text-align:left;">
0.0275723
</td>
<td style="text-align:left;">
Ptp4a1
</td>
<td style="text-align:left;">
protein tyrosine phosphatase 4a1
</td>
<td style="text-align:left;">
19243
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Dst|Dst
</td>
<td style="text-align:left;">
2769.47797
</td>
<td style="text-align:left;">
0.6444525
</td>
<td style="text-align:left;">
0.1796336
</td>
<td style="text-align:left;">
3.587594
</td>
<td style="text-align:left;">
0.0003337
</td>
<td style="text-align:left;">
0.0169842
</td>
<td style="text-align:left;">
Dst
</td>
<td style="text-align:left;">
dystonin
</td>
<td style="text-align:left;">
13518
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Il1rl1|Il1rl1
</td>
<td style="text-align:left;">
767.85351
</td>
<td style="text-align:left;">
-0.6675801
</td>
<td style="text-align:left;">
0.2001417
</td>
<td style="text-align:left;">
-3.335537
</td>
<td style="text-align:left;">
0.0008513
</td>
<td style="text-align:left;">
0.0332898
</td>
<td style="text-align:left;">
Il1rl1
</td>
<td style="text-align:left;">
interleukin 1 receptor-like 1
</td>
<td style="text-align:left;">
17082
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Tmeff2|Tmeff2
</td>
<td style="text-align:left;">
1851.52483
</td>
<td style="text-align:left;">
1.5905441
</td>
<td style="text-align:left;">
0.3999840
</td>
<td style="text-align:left;">
3.976519
</td>
<td style="text-align:left;">
0.0000699
</td>
<td style="text-align:left;">
0.0050114
</td>
<td style="text-align:left;">
Tmeff2
</td>
<td style="text-align:left;">
transmembrane protein with EGF-like and two follistatin-like domains 2
</td>
<td style="text-align:left;">
56363
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Gtf3c3|Gtf3c3
</td>
<td style="text-align:left;">
1892.36540
</td>
<td style="text-align:left;">
-0.4870701
</td>
<td style="text-align:left;">
0.1428300
</td>
<td style="text-align:left;">
-3.410138
</td>
<td style="text-align:left;">
0.0006493
</td>
<td style="text-align:left;">
0.0274693
</td>
<td style="text-align:left;">
Gtf3c3
</td>
<td style="text-align:left;">
general transcription factor IIIC, polypeptide 3
</td>
<td style="text-align:left;">
98488
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Coq10b|Coq10b
</td>
<td style="text-align:left;">
3416.27810
</td>
<td style="text-align:left;">
0.7241932
</td>
<td style="text-align:left;">
0.1366191
</td>
<td style="text-align:left;">
5.300820
</td>
<td style="text-align:left;">
0.0000001
</td>
<td style="text-align:left;">
0.0000270
</td>
<td style="text-align:left;">
Coq10b
</td>
<td style="text-align:left;">
coenzyme Q10B
</td>
<td style="text-align:left;">
67876
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Hspd1|Hspd1
</td>
<td style="text-align:left;">
13992.02859
</td>
<td style="text-align:left;">
0.4306525
</td>
<td style="text-align:left;">
0.0876478
</td>
<td style="text-align:left;">
4.913444
</td>
<td style="text-align:left;">
0.0000009
</td>
<td style="text-align:left;">
0.0001511
</td>
<td style="text-align:left;">
Hspd1
</td>
<td style="text-align:left;">
heat shock protein 1 (chaperonin)
</td>
<td style="text-align:left;">
15510
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Wdr12|Wdr12
</td>
<td style="text-align:left;">
2234.91895
</td>
<td style="text-align:left;">
0.4341932
</td>
<td style="text-align:left;">
0.1246503
</td>
<td style="text-align:left;">
3.483289
</td>
<td style="text-align:left;">
0.0004953
</td>
<td style="text-align:left;">
0.0229442
</td>
<td style="text-align:left;">
Wdr12
</td>
<td style="text-align:left;">
WD repeat domain 12
</td>
<td style="text-align:left;">
57750
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Map2|Map2
</td>
<td style="text-align:left;">
649.58161
</td>
<td style="text-align:left;">
1.7126502
</td>
<td style="text-align:left;">
0.4851971
</td>
<td style="text-align:left;">
3.529803
</td>
<td style="text-align:left;">
0.0004159
</td>
<td style="text-align:left;">
0.0201697
</td>
<td style="text-align:left;">
Map2
</td>
<td style="text-align:left;">
microtubule-associated protein 2
</td>
<td style="text-align:left;">
17756
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Abcb6|Abcb6
</td>
<td style="text-align:left;">
1043.57686
</td>
<td style="text-align:left;">
-1.0128883
</td>
<td style="text-align:left;">
0.2699762
</td>
<td style="text-align:left;">
-3.751769
</td>
<td style="text-align:left;">
0.0001756
</td>
<td style="text-align:left;">
0.0102762
</td>
<td style="text-align:left;">
Abcb6
</td>
<td style="text-align:left;">
ATP-binding cassette, sub-family B (MDR/TAP), member 6
</td>
<td style="text-align:left;">
74104
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Efhd1|Efhd1
</td>
<td style="text-align:left;">
915.13609
</td>
<td style="text-align:left;">
1.0882417
</td>
<td style="text-align:left;">
0.3208173
</td>
<td style="text-align:left;">
3.392092
</td>
<td style="text-align:left;">
0.0006936
</td>
<td style="text-align:left;">
0.0288572
</td>
<td style="text-align:left;">
Efhd1
</td>
<td style="text-align:left;">
EF hand domain containing 1
</td>
<td style="text-align:left;">
98363
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Ugt1a6a|Ugt1a6a
</td>
<td style="text-align:left;">
28.49494
</td>
<td style="text-align:left;">
-7.2900944
</td>
<td style="text-align:left;">
1.4156636
</td>
<td style="text-align:left;">
-5.149595
</td>
<td style="text-align:left;">
0.0000003
</td>
<td style="text-align:left;">
0.0000516
</td>
<td style="text-align:left;">
Ugt1a6a
</td>
<td style="text-align:left;">
UDP glucuronosyltransferase 1 family, polypeptide A6A
</td>
<td style="text-align:left;">
94284
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Cntn2|Cntn2
</td>
<td style="text-align:left;">
352.60489
</td>
<td style="text-align:left;">
2.5104391
</td>
<td style="text-align:left;">
0.5415527
</td>
<td style="text-align:left;">
4.635632
</td>
<td style="text-align:left;">
0.0000036
</td>
<td style="text-align:left;">
0.0004400
</td>
<td style="text-align:left;">
Cntn2
</td>
<td style="text-align:left;">
contactin 2
</td>
<td style="text-align:left;">
21367
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Adora1|Adora1
</td>
<td style="text-align:left;">
400.20827
</td>
<td style="text-align:left;">
1.7436188
</td>
<td style="text-align:left;">
0.3745739
</td>
<td style="text-align:left;">
4.654939
</td>
<td style="text-align:left;">
0.0000032
</td>
<td style="text-align:left;">
0.0004064
</td>
<td style="text-align:left;">
Adora1
</td>
<td style="text-align:left;">
adenosine A1 receptor
</td>
<td style="text-align:left;">
11539
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Shisa4|Shisa4
</td>
<td style="text-align:left;">
907.41309
</td>
<td style="text-align:left;">
1.4450193
</td>
<td style="text-align:left;">
0.3905135
</td>
<td style="text-align:left;">
3.700306
</td>
<td style="text-align:left;">
0.0002153
</td>
<td style="text-align:left;">
0.0121177
</td>
<td style="text-align:left;">
Shisa4
</td>
<td style="text-align:left;">
shisa family member 4
</td>
<td style="text-align:left;">
77552
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Phlda3|Phlda3
</td>
<td style="text-align:left;">
607.95039
</td>
<td style="text-align:left;">
1.3654343
</td>
<td style="text-align:left;">
0.4095723
</td>
<td style="text-align:left;">
3.333805
</td>
<td style="text-align:left;">
0.0008567
</td>
<td style="text-align:left;">
0.0334233
</td>
<td style="text-align:left;">
Phlda3
</td>
<td style="text-align:left;">
pleckstrin homology like domain, family A, member 3
</td>
<td style="text-align:left;">
27280
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="abstinence-vs-saline-1" class="section level3">
<h3>Abstinence vs Saline</h3>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:1000px; ">
<table class="table table-striped table-hover table-condensed table-responsive" style="font-size: 10px; font-family: Arial; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Significant DEGs Abstinence vs Saline
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
baseMean
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
log2FoldChange
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
lfcSE
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
stat
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
padj
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
symbol
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
entrez
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ensembl
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Sulf1|Sulf1
</td>
<td style="text-align:left;">
5.066850e+01
</td>
<td style="text-align:left;">
21.9195329
</td>
<td style="text-align:left;">
2.5985677
</td>
<td style="text-align:left;">
8.435237
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
Sulf1
</td>
<td style="text-align:left;">
sulfatase 1
</td>
<td style="text-align:left;">
240725
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Tmeff2|Tmeff2
</td>
<td style="text-align:left;">
1.851525e+03
</td>
<td style="text-align:left;">
1.3996169
</td>
<td style="text-align:left;">
0.3703824
</td>
<td style="text-align:left;">
3.778843
</td>
<td style="text-align:left;">
0.0001576
</td>
<td style="text-align:left;">
0.0138324
</td>
<td style="text-align:left;">
Tmeff2
</td>
<td style="text-align:left;">
transmembrane protein with EGF-like and two follistatin-like domains 2
</td>
<td style="text-align:left;">
56363
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Fzd5|Fzd5
</td>
<td style="text-align:left;">
1.406694e+02
</td>
<td style="text-align:left;">
1.8591904
</td>
<td style="text-align:left;">
0.4420074
</td>
<td style="text-align:left;">
4.206242
</td>
<td style="text-align:left;">
0.0000260
</td>
<td style="text-align:left;">
0.0030443
</td>
<td style="text-align:left;">
Fzd5
</td>
<td style="text-align:left;">
frizzled class receptor 5
</td>
<td style="text-align:left;">
14367
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Ptprn|Ptprn
</td>
<td style="text-align:left;">
9.746532e+01
</td>
<td style="text-align:left;">
1.8603752
</td>
<td style="text-align:left;">
0.5503598
</td>
<td style="text-align:left;">
3.380289
</td>
<td style="text-align:left;">
0.0007241
</td>
<td style="text-align:left;">
0.0441578
</td>
<td style="text-align:left;">
Ptprn
</td>
<td style="text-align:left;">
protein tyrosine phosphatase, receptor type, N
</td>
<td style="text-align:left;">
19275
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Tmem37|Tmem37
</td>
<td style="text-align:left;">
8.065723e+03
</td>
<td style="text-align:left;">
-0.3977212
</td>
<td style="text-align:left;">
0.1075395
</td>
<td style="text-align:left;">
-3.698373
</td>
<td style="text-align:left;">
0.0002170
</td>
<td style="text-align:left;">
0.0180059
</td>
<td style="text-align:left;">
Tmem37
</td>
<td style="text-align:left;">
transmembrane protein 37
</td>
<td style="text-align:left;">
170706
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Cntn2|Cntn2
</td>
<td style="text-align:left;">
3.526049e+02
</td>
<td style="text-align:left;">
2.4908656
</td>
<td style="text-align:left;">
0.5019458
</td>
<td style="text-align:left;">
4.962420
</td>
<td style="text-align:left;">
0.0000007
</td>
<td style="text-align:left;">
0.0001454
</td>
<td style="text-align:left;">
Cntn2
</td>
<td style="text-align:left;">
contactin 2
</td>
<td style="text-align:left;">
21367
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Rbbp5|Rbbp5
</td>
<td style="text-align:left;">
2.067301e+03
</td>
<td style="text-align:left;">
0.7515691
</td>
<td style="text-align:left;">
0.1861912
</td>
<td style="text-align:left;">
4.036545
</td>
<td style="text-align:left;">
0.0000542
</td>
<td style="text-align:left;">
0.0058330
</td>
<td style="text-align:left;">
Rbbp5
</td>
<td style="text-align:left;">
retinoblastoma binding protein 5, histone lysine methyltransferase
complex subunit
</td>
<td style="text-align:left;">
213464
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Adora1|Adora1
</td>
<td style="text-align:left;">
4.002083e+02
</td>
<td style="text-align:left;">
1.3125207
</td>
<td style="text-align:left;">
0.3472911
</td>
<td style="text-align:left;">
3.779310
</td>
<td style="text-align:left;">
0.0001573
</td>
<td style="text-align:left;">
0.0138324
</td>
<td style="text-align:left;">
Adora1
</td>
<td style="text-align:left;">
adenosine A1 receptor
</td>
<td style="text-align:left;">
11539
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Shisa4|Shisa4
</td>
<td style="text-align:left;">
9.074131e+02
</td>
<td style="text-align:left;">
1.5961330
</td>
<td style="text-align:left;">
0.3616009
</td>
<td style="text-align:left;">
4.414073
</td>
<td style="text-align:left;">
0.0000101
</td>
<td style="text-align:left;">
0.0014516
</td>
<td style="text-align:left;">
Shisa4
</td>
<td style="text-align:left;">
shisa family member 4
</td>
<td style="text-align:left;">
77552
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Lad1|Lad1
</td>
<td style="text-align:left;">
1.055933e+01
</td>
<td style="text-align:left;">
20.2855251
</td>
<td style="text-align:left;">
3.6022097
</td>
<td style="text-align:left;">
5.631411
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000060
</td>
<td style="text-align:left;">
Lad1
</td>
<td style="text-align:left;">
ladinin
</td>
<td style="text-align:left;">
16763
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Prdx6|Prdx6
</td>
<td style="text-align:left;">
6.519144e+03
</td>
<td style="text-align:left;">
0.5079789
</td>
<td style="text-align:left;">
0.1320362
</td>
<td style="text-align:left;">
3.847271
</td>
<td style="text-align:left;">
0.0001194
</td>
<td style="text-align:left;">
0.0111886
</td>
<td style="text-align:left;">
Prdx6
</td>
<td style="text-align:left;">
peroxiredoxin 6
</td>
<td style="text-align:left;">
11758
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Fcgr3|Fcgr3
</td>
<td style="text-align:left;">
2.645967e+05
</td>
<td style="text-align:left;">
-0.1785064
</td>
<td style="text-align:left;">
0.0431478
</td>
<td style="text-align:left;">
-4.137086
</td>
<td style="text-align:left;">
0.0000352
</td>
<td style="text-align:left;">
0.0040203
</td>
<td style="text-align:left;">
Fcgr3
</td>
<td style="text-align:left;">
Fc receptor, IgG, low affinity III
</td>
<td style="text-align:left;">
14131
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Kcnj10|Kcnj10
</td>
<td style="text-align:left;">
4.180340e+02
</td>
<td style="text-align:left;">
0.7997097
</td>
<td style="text-align:left;">
0.2355706
</td>
<td style="text-align:left;">
3.394778
</td>
<td style="text-align:left;">
0.0006868
</td>
<td style="text-align:left;">
0.0431903
</td>
<td style="text-align:left;">
Kcnj10
</td>
<td style="text-align:left;">
potassium inwardly-rectifying channel, subfamily J, member 10
</td>
<td style="text-align:left;">
16513
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
1700034H15Rik|1700034H15Rik
</td>
<td style="text-align:left;">
9.732449e+00
</td>
<td style="text-align:left;">
20.4270038
</td>
<td style="text-align:left;">
3.6021048
</td>
<td style="text-align:left;">
5.670852
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000050
</td>
<td style="text-align:left;">
1700034H15Rik
</td>
<td style="text-align:left;">
RIKEN cDNA 1700034H15 gene
</td>
<td style="text-align:left;">
98736
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
LOC108167797|LOC108167797
</td>
<td style="text-align:left;">
7.830883e+00
</td>
<td style="text-align:left;">
19.6809315
</td>
<td style="text-align:left;">
3.6031327
</td>
<td style="text-align:left;">
5.462172
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000142
</td>
<td style="text-align:left;">
LOC108167797
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Ifngr1|Ifngr1
</td>
<td style="text-align:left;">
9.796039e+04
</td>
<td style="text-align:left;">
-0.2495816
</td>
<td style="text-align:left;">
0.0666201
</td>
<td style="text-align:left;">
-3.746340
</td>
<td style="text-align:left;">
0.0001794
</td>
<td style="text-align:left;">
0.0155278
</td>
<td style="text-align:left;">
Ifngr1
</td>
<td style="text-align:left;">
interferon gamma receptor 1
</td>
<td style="text-align:left;">
15979
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Gja1|Gja1
</td>
<td style="text-align:left;">
4.212005e+03
</td>
<td style="text-align:left;">
1.0115368
</td>
<td style="text-align:left;">
0.2662060
</td>
<td style="text-align:left;">
3.799828
</td>
<td style="text-align:left;">
0.0001448
</td>
<td style="text-align:left;">
0.0130267
</td>
<td style="text-align:left;">
Gja1
</td>
<td style="text-align:left;">
gap junction protein, alpha 1
</td>
<td style="text-align:left;">
14609
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Unc5b|Unc5b
</td>
<td style="text-align:left;">
2.024884e+02
</td>
<td style="text-align:left;">
1.6036437
</td>
<td style="text-align:left;">
0.4048530
</td>
<td style="text-align:left;">
3.961052
</td>
<td style="text-align:left;">
0.0000746
</td>
<td style="text-align:left;">
0.0077490
</td>
<td style="text-align:left;">
Unc5b
</td>
<td style="text-align:left;">
unc-5 netrin receptor B
</td>
<td style="text-align:left;">
107449
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Ank3|Ank3
</td>
<td style="text-align:left;">
3.950925e+02
</td>
<td style="text-align:left;">
1.4819121
</td>
<td style="text-align:left;">
0.3442522
</td>
<td style="text-align:left;">
4.304729
</td>
<td style="text-align:left;">
0.0000167
</td>
<td style="text-align:left;">
0.0022017
</td>
<td style="text-align:left;">
Ank3
</td>
<td style="text-align:left;">
ankyrin 3, epithelial
</td>
<td style="text-align:left;">
11735
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Phyhipl|Phyhipl
</td>
<td style="text-align:left;">
2.026671e+03
</td>
<td style="text-align:left;">
1.5678817
</td>
<td style="text-align:left;">
0.2661561
</td>
<td style="text-align:left;">
5.890834
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000023
</td>
<td style="text-align:left;">
Phyhipl
</td>
<td style="text-align:left;">
phytanoyl-CoA hydroxylase interacting protein-like
</td>
<td style="text-align:left;">
70911
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="abstinence-vs-maintenance-1" class="section level3">
<h3>Abstinence vs Maintenance</h3>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:1000px; ">
<table class="table table-striped table-hover table-condensed table-responsive" style="font-size: 10px; font-family: Arial; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Significant Abstinence vs Maintenance
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
baseMean
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
log2FoldChange
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
lfcSE
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
stat
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
padj
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
symbol
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
entrez
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ensembl
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Tcea1|Tcea1
</td>
<td style="text-align:left;">
8191.29655
</td>
<td style="text-align:left;">
-0.2366124
</td>
<td style="text-align:left;">
0.0583112
</td>
<td style="text-align:left;">
-4.057756
</td>
<td style="text-align:left;">
0.0000495
</td>
<td style="text-align:left;">
0.0035076
</td>
<td style="text-align:left;">
Tcea1
</td>
<td style="text-align:left;">
transcription elongation factor A (SII) 1
</td>
<td style="text-align:left;">
21399
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Sgk3|Sgk3
</td>
<td style="text-align:left;">
8049.66688
</td>
<td style="text-align:left;">
-0.5398574
</td>
<td style="text-align:left;">
0.1089754
</td>
<td style="text-align:left;">
-4.953939
</td>
<td style="text-align:left;">
0.0000007
</td>
<td style="text-align:left;">
0.0000946
</td>
<td style="text-align:left;">
Sgk3
</td>
<td style="text-align:left;">
serum/glucocorticoid regulated kinase 3
</td>
<td style="text-align:left;">
170755
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Tceb1|Tceb1
</td>
<td style="text-align:left;">
7975.02657
</td>
<td style="text-align:left;">
0.2184331
</td>
<td style="text-align:left;">
0.0695727
</td>
<td style="text-align:left;">
3.139636
</td>
<td style="text-align:left;">
0.0016916
</td>
<td style="text-align:left;">
0.0452040
</td>
<td style="text-align:left;">
Tceb1
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Fam135a|Fam135a
</td>
<td style="text-align:left;">
1287.34784
</td>
<td style="text-align:left;">
0.6104730
</td>
<td style="text-align:left;">
0.1797700
</td>
<td style="text-align:left;">
3.395856
</td>
<td style="text-align:left;">
0.0006841
</td>
<td style="text-align:left;">
0.0235520
</td>
<td style="text-align:left;">
Fam135a
</td>
<td style="text-align:left;">
family with sequence similarity 135, member A
</td>
<td style="text-align:left;">
68187
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Phf3|Phf3
</td>
<td style="text-align:left;">
1225.06289
</td>
<td style="text-align:left;">
-0.6373891
</td>
<td style="text-align:left;">
0.1792120
</td>
<td style="text-align:left;">
-3.556620
</td>
<td style="text-align:left;">
0.0003757
</td>
<td style="text-align:left;">
0.0153382
</td>
<td style="text-align:left;">
Phf3
</td>
<td style="text-align:left;">
PHD finger protein 3
</td>
<td style="text-align:left;">
213109
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Ptp4a1|Ptp4a1
</td>
<td style="text-align:left;">
10992.32954
</td>
<td style="text-align:left;">
-0.2408100
</td>
<td style="text-align:left;">
0.0648710
</td>
<td style="text-align:left;">
-3.712138
</td>
<td style="text-align:left;">
0.0002055
</td>
<td style="text-align:left;">
0.0099409
</td>
<td style="text-align:left;">
Ptp4a1
</td>
<td style="text-align:left;">
protein tyrosine phosphatase 4a1
</td>
<td style="text-align:left;">
19243
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Dst|Dst
</td>
<td style="text-align:left;">
2769.47797
</td>
<td style="text-align:left;">
-0.9424648
</td>
<td style="text-align:left;">
0.1662848
</td>
<td style="text-align:left;">
-5.667774
</td>
<td style="text-align:left;">
0.0000000
</td>
<td style="text-align:left;">
0.0000031
</td>
<td style="text-align:left;">
Dst
</td>
<td style="text-align:left;">
dystonin
</td>
<td style="text-align:left;">
13518
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Unc50|Unc50
</td>
<td style="text-align:left;">
7164.14519
</td>
<td style="text-align:left;">
-0.1959583
</td>
<td style="text-align:left;">
0.0598255
</td>
<td style="text-align:left;">
-3.275497
</td>
<td style="text-align:left;">
0.0010548
</td>
<td style="text-align:left;">
0.0321111
</td>
<td style="text-align:left;">
Unc50
</td>
<td style="text-align:left;">
unc-50 homolog
</td>
<td style="text-align:left;">
67387
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Il1rl1|Il1rl1
</td>
<td style="text-align:left;">
767.85351
</td>
<td style="text-align:left;">
0.6680319
</td>
<td style="text-align:left;">
0.1852259
</td>
<td style="text-align:left;">
3.606580
</td>
<td style="text-align:left;">
0.0003103
</td>
<td style="text-align:left;">
0.0134218
</td>
<td style="text-align:left;">
Il1rl1
</td>
<td style="text-align:left;">
interleukin 1 receptor-like 1
</td>
<td style="text-align:left;">
17082
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Stk17b|Stk17b
</td>
<td style="text-align:left;">
11796.44360
</td>
<td style="text-align:left;">
-0.4213730
</td>
<td style="text-align:left;">
0.1010894
</td>
<td style="text-align:left;">
-4.168320
</td>
<td style="text-align:left;">
0.0000307
</td>
<td style="text-align:left;">
0.0023526
</td>
<td style="text-align:left;">
Stk17b
</td>
<td style="text-align:left;">
serine/threonine kinase 17b (apoptosis-inducing)
</td>
<td style="text-align:left;">
98267
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Hspd1|Hspd1
</td>
<td style="text-align:left;">
13992.02859
</td>
<td style="text-align:left;">
-0.2769954
</td>
<td style="text-align:left;">
0.0810943
</td>
<td style="text-align:left;">
-3.415718
</td>
<td style="text-align:left;">
0.0006361
</td>
<td style="text-align:left;">
0.0222929
</td>
<td style="text-align:left;">
Hspd1
</td>
<td style="text-align:left;">
heat shock protein 1 (chaperonin)
</td>
<td style="text-align:left;">
15510
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Nrp2|Nrp2
</td>
<td style="text-align:left;">
871.34143
</td>
<td style="text-align:left;">
-0.5085451
</td>
<td style="text-align:left;">
0.1387695
</td>
<td style="text-align:left;">
-3.664675
</td>
<td style="text-align:left;">
0.0002477
</td>
<td style="text-align:left;">
0.0114422
</td>
<td style="text-align:left;">
Nrp2
</td>
<td style="text-align:left;">
neuropilin 2
</td>
<td style="text-align:left;">
18187
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Pard3b|Pard3b
</td>
<td style="text-align:left;">
262.31577
</td>
<td style="text-align:left;">
0.9405450
</td>
<td style="text-align:left;">
0.2928996
</td>
<td style="text-align:left;">
3.211152
</td>
<td style="text-align:left;">
0.0013220
</td>
<td style="text-align:left;">
0.0381761
</td>
<td style="text-align:left;">
Pard3b
</td>
<td style="text-align:left;">
par-3 family cell polarity regulator beta
</td>
<td style="text-align:left;">
72823
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Abcb6|Abcb6
</td>
<td style="text-align:left;">
1043.57686
</td>
<td style="text-align:left;">
1.1106563
</td>
<td style="text-align:left;">
0.2499278
</td>
<td style="text-align:left;">
4.443909
</td>
<td style="text-align:left;">
0.0000088
</td>
<td style="text-align:left;">
0.0008206
</td>
<td style="text-align:left;">
Abcb6
</td>
<td style="text-align:left;">
ATP-binding cassette, sub-family B (MDR/TAP), member 6
</td>
<td style="text-align:left;">
74104
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Rhbdd1|Rhbdd1
</td>
<td style="text-align:left;">
3405.91366
</td>
<td style="text-align:left;">
0.7278433
</td>
<td style="text-align:left;">
0.1872318
</td>
<td style="text-align:left;">
3.887392
</td>
<td style="text-align:left;">
0.0001013
</td>
<td style="text-align:left;">
0.0060717
</td>
<td style="text-align:left;">
Rhbdd1
</td>
<td style="text-align:left;">
rhomboid domain containing 1
</td>
<td style="text-align:left;">
76867
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Dock10|Dock10
</td>
<td style="text-align:left;">
14056.33423
</td>
<td style="text-align:left;">
-0.3079567
</td>
<td style="text-align:left;">
0.0961073
</td>
<td style="text-align:left;">
-3.204302
</td>
<td style="text-align:left;">
0.0013539
</td>
<td style="text-align:left;">
0.0388407
</td>
<td style="text-align:left;">
Dock10
</td>
<td style="text-align:left;">
dedicator of cytokinesis 10
</td>
<td style="text-align:left;">
210293
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Pid1|Pid1
</td>
<td style="text-align:left;">
6100.89696
</td>
<td style="text-align:left;">
-0.5207357
</td>
<td style="text-align:left;">
0.1248731
</td>
<td style="text-align:left;">
-4.170119
</td>
<td style="text-align:left;">
0.0000304
</td>
<td style="text-align:left;">
0.0023526
</td>
<td style="text-align:left;">
Pid1
</td>
<td style="text-align:left;">
phosphotyrosine interaction domain containing 1
</td>
<td style="text-align:left;">
98496
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
C130026I21Rik|C130026I21Rik
</td>
<td style="text-align:left;">
10.78415
</td>
<td style="text-align:left;">
6.8108008
</td>
<td style="text-align:left;">
2.0397198
</td>
<td style="text-align:left;">
3.339086
</td>
<td style="text-align:left;">
0.0008405
</td>
<td style="text-align:left;">
0.0275325
</td>
<td style="text-align:left;">
C130026I21Rik
</td>
<td style="text-align:left;">
RIKEN cDNA C130026I21 gene
</td>
<td style="text-align:left;">
620078
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Kcnj13|Kcnj13
</td>
<td style="text-align:left;">
17.91513
</td>
<td style="text-align:left;">
7.6155817
</td>
<td style="text-align:left;">
1.9974216
</td>
<td style="text-align:left;">
3.812706
</td>
<td style="text-align:left;">
0.0001375
</td>
<td style="text-align:left;">
0.0075050
</td>
<td style="text-align:left;">
Kcnj13
</td>
<td style="text-align:left;">
potassium inwardly-rectifying channel, subfamily J, member 13
</td>
<td style="text-align:left;">
100040591
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
<tr>
<td style="text-align:left;">
Ugt1a6a|Ugt1a6a
</td>
<td style="text-align:left;">
28.49494
</td>
<td style="text-align:left;">
5.6541170
</td>
<td style="text-align:left;">
1.3572152
</td>
<td style="text-align:left;">
4.165970
</td>
<td style="text-align:left;">
0.0000310
</td>
<td style="text-align:left;">
0.0023666
</td>
<td style="text-align:left;">
Ugt1a6a
</td>
<td style="text-align:left;">
UDP glucuronosyltransferase 1 family, polypeptide A6A
</td>
<td style="text-align:left;">
94284
</td>
<td style="text-align:left;">
ENSMUSG0….
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="visualize-shrinkage-estimations" class="section level2">
<h2>Visualize Shrinkage Estimations</h2>
<p>Following subsetting, I want to see how the data looks based on
effect size so I have run a shrinkage model using the “apeglm” method
which normalizes counts based on effect size without major changes to
data structure. I have included non-shrunk models as well for
comparison.</p>
<div id="non-shrunken-l2fc" class="section level3 tabset">
<h3 class="tabset">Non-Shrunken L2FC</h3>
<div id="maintenance-vs-saline-2" class="section level4">
<h4>Maintenance vs Saline</h4>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Non-Shrunken%20Estimate%20Plots-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="abstinence-vs-saline-2" class="section level4">
<h4>Abstinence vs Saline</h4>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Non-Shrunken%20Estimate%20Plots2-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="abstinence-vs-maintenance-2" class="section level4">
<h4>Abstinence vs Maintenance</h4>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Non-Shrunken%20Estimate%20Plots3-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="shrunken-l2fc" class="section level3 tabset">
<h3 class="tabset">Shrunken L2FC</h3>
<div id="maintenance-vs-saline-3" class="section level4">
<h4>Maintenance vs Saline</h4>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Shrunken%20Estimate%20Plots-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="abstinence-vs-saline-3" class="section level4">
<h4>Abstinence vs Saline</h4>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Shrunken%20Estimate%20Plots2-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="abstinence-vs-maintenance-3" class="section level4">
<h4>Abstinence vs Maintenance</h4>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Shrunken%20Estimate%20Plots3-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
</div>
<div id="looking-at-differentially-expressed-genes"
class="section level1">
<h1>Looking at Differentially Expressed Genes</h1>
<p>We can generate a matrix to identify where our differentially
expressed genes lie between each group based on an alpha value of
0.05.</p>
<pre class="r"><code>## Matrix Visualizaattion
x &lt;- vsDEGMatrix(
  data = dds, padj = 0.05, d.factor = &quot;condition&quot;,
  type = &quot;deseq&quot;, title = TRUE, legend = TRUE, grid = TRUE
)</code></pre>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Matrix%20of%20Differentially%20Expressed%20genes%20across%20groups-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="data-visualization" class="section level1">
<h1>Data Visualization</h1>
<div id="normalized-counts-and-plots-of-genes-of-interest"
class="section level2 tabset">
<h2 class="tabset">Normalized Counts and Plots of Genes of Interest</h2>
<pre class="r"><code>level_order &lt;- c(&quot;Saline&quot;, &quot;Maintenance&quot;, &quot;Abstinence&quot;) # this vector might be useful for other plots/analyses

normalized_counts &lt;- counts(dds, normalized = TRUE)</code></pre>
<div id="hspd1" class="section level3">
<h3>Hspd1</h3>
<pre class="r"><code>####################################### HSPA8

d2 &lt;- plotCounts(dds,
  gene = &quot;Hspd1|Hspd1&quot;, intgroup = &quot;condition&quot;,
  returnData = TRUE
)

a &lt;- ggplot(d2, aes(x = factor(condition, level = level_order), y = count, color = condition)) +
  geom_point(position = position_jitter(w = 0.1, h = 0)) +
  theme_bw() +
  ggtitle(
    label = &quot;Hspd1 Expression&quot;,
    subtitle = &quot;Normalized Gene Expression&quot;
  )

a</code></pre>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Selected%20Genes-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="cirbp" class="section level3">
<h3>Cirbp</h3>
<pre class="r"><code>####################################### HSP90ab1


d2 &lt;- plotCounts(dds,
  gene = &quot;Cirbp|Cirbp&quot;, intgroup = &quot;condition&quot;,
  returnData = TRUE
)

a &lt;- ggplot(d2, aes(x = factor(condition, level = level_order), y = count, color = condition)) +
  geom_point(position = position_jitter(w = 0.1, h = 0)) +
  theme_bw() +
  ggtitle(
    label = &quot;Cirbp Expression&quot;,
    subtitle = &quot;Normalized Gene Expression&quot;
  )

a</code></pre>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Selected%20Genes2-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="fus" class="section level3">
<h3>Fus</h3>
<pre class="r"><code>####################################### Dnajb1


d2 &lt;- plotCounts(dds,
  gene = &quot;Fus|Fus&quot;, intgroup = &quot;condition&quot;,
  returnData = TRUE
)

a &lt;- ggplot(d2, aes(x = factor(condition, level = level_order), y = count, color = condition)) +
  geom_point(position = position_jitter(w = 0.1, h = 0)) +
  theme_bw() +
  ggtitle(
    label = &quot;Fus Expression&quot;,
    subtitle = &quot;Normalized Gene Expression&quot;
  )

a</code></pre>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Selected%20Genes3-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="cryab" class="section level3">
<h3>Cryab</h3>
<pre class="r"><code>####################################### Dnajb1


d2 &lt;- plotCounts(dds,
  gene = &quot;Cryab|Cryab&quot;, intgroup = &quot;condition&quot;,
  returnData = TRUE
)

a &lt;- ggplot(d2, aes(x = factor(condition, level = level_order), y = count, color = condition)) +
  geom_point(position = position_jitter(w = 0.1, h = 0)) +
  theme_bw() +
  ggtitle(
    label = &quot;Cryab Expression&quot;,
    subtitle = &quot;Normalized Gene Expression&quot;
  )

a</code></pre>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Selected%20Genes4-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="nr1d1" class="section level3">
<h3>Nr1d1</h3>
<pre class="r"><code>####################################### Dnajb1


d2 &lt;- plotCounts(dds,
  gene = &quot;Nr1d1|Nr1d1&quot;, intgroup = &quot;condition&quot;,
  returnData = TRUE
)

a &lt;- ggplot(d2, aes(x = factor(condition, level = level_order), y = count, color = condition)) +
  geom_point(position = position_jitter(w = 0.1, h = 0)) +
  theme_bw() +
  ggtitle(
    label = &quot;Nr1d1 Expression&quot;,
    subtitle = &quot;Normalized Gene Expression&quot;
  )

a</code></pre>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Selected%20Genes5-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="syt11" class="section level3">
<h3>Syt11</h3>
<pre class="r"><code>####################################### Dnajb1


d2 &lt;- plotCounts(dds,
  gene = &quot;Syt11|Syt11&quot;, intgroup = &quot;condition&quot;,
  returnData = TRUE
)

a &lt;- ggplot(d2, aes(x = factor(condition, level = level_order), y = count, color = condition)) +
  geom_point(position = position_jitter(w = 0.1, h = 0)) +
  theme_bw() +
  ggtitle(
    label = &quot;Syt11 Expression&quot;,
    subtitle = &quot;Normalized Gene Expression&quot;
  )

a</code></pre>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Selected%20Genes6-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="visualize-results-pca" class="section level2">
<h2>Visualize Results: PCA</h2>
<p>For visualization I am converting all analyses to logarithmic.
Following this I have completed a PCA for each analyses based on all
genes together.</p>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Plot%20PCA-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="visualize-results-heatmap-data-generation"
class="section level2">
<h2>Visualize Results: Heatmap (Data Generation)</h2>
<p>This heatmap is generated by taking the row means of the normalized
counts from the primary analysis (dds, All vs Control). The top genes
were placed in decreasing order and selected for analysis after
logarithmic transformation.The gene name column is then split by the “|”
delimeter in order to obtain a single gene name for each count. The
heatmap was generated by scaling to genes for the z-scores following
row/column analysis using a complete pearson correlation. The heatmap is
clustered by row but not by column to maintain correct order of
samples.The legend indicates logfold change values for each gene.
Representative heatmaps are included for a number of selected genes.</p>
</div>
<div id="visualize-results-heatmap" class="section level2 tabset">
<h2 class="tabset">Visualize Results: Heatmap</h2>
<pre class="r"><code>ressig1 &lt;- as.data.frame(results_sig)
ressig2 &lt;- as.data.frame(results_sig2)
ressig3 &lt;- as.data.frame(results_sig3)

ressig1$Comparison &lt;- &quot;Maintenance vs Saline&quot;
ressig2$Comparison &lt;- &quot;Abstinence vs Saline&quot;
ressig3$Comparison &lt;- &quot;Abstinence vs Maintenance&quot;

ressig1_highlfc_pos &lt;- ressig1[ressig1$log2FoldChange &gt;= 1.3, ]
ressig1_highlfc_neg &lt;- ressig1[ressig1$log2FoldChange &lt;= -1.3, ]
ressig2_highlfc_pos &lt;- ressig2[ressig2$log2FoldChange &gt;= 1.3, ]
ressig2_highlfc_neg &lt;- ressig2[ressig2$log2FoldChange &lt;= -1.3, ]
ressig3_highlfc_pos &lt;- ressig3[ressig3$log2FoldChange &gt;= 1.3, ]
ressig3_highlfc_neg &lt;- ressig3[ressig3$log2FoldChange &lt;= -1.3, ]

temp_sigdegshm &lt;- rbind(ressig1_highlfc_pos, ressig1_highlfc_neg)
temp_sigdegshm2 &lt;- rbind(temp_sigdegshm, ressig2_highlfc_pos)
temp_sigdegshm3 &lt;- rbind(temp_sigdegshm2, ressig2_highlfc_neg)
temp_sigdegshm4 &lt;- rbind(temp_sigdegshm3, ressig3_highlfc_pos)
temp_sigdegshm5 &lt;- rbind(temp_sigdegshm4, ressig3_highlfc_neg)

temp_sigdegshm6 &lt;- temp_sigdegshm5[temp_sigdegshm5$lfcSE &lt;= 1.50, ]

# sum(is.na(temp_sigdegshm6$symbol))
# sum(duplicated(temp_sigdegshm6$symbol))

dup_rows &lt;- duplicated(temp_sigdegshm6$symbol)
data_dedup &lt;- temp_sigdegshm6[!dup_rows, ]

rows &lt;- rownames(data_dedup)

select &lt;- order(rowMeans(counts(dds, normalized = TRUE)),
  decreasing = TRUE
)

mat &lt;- assay(ddsMat_rlog)[select, ]

data_mod &lt;- mat[rownames(mat) %in% rows, ]

matrownames_data_mod &lt;- data.frame(rownames(data_mod))

matrownames_data_mod &lt;- matrownames_data_mod %&gt;% separate(rownames.data_mod., c(&quot;id&quot;, &quot;symbol&quot;), sep = &quot;([|])&quot;)


samples &lt;- as.tibble(read.csv(&quot;/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/final_samples_removed/sample_info_final.csv&quot;, sep = &quot;,&quot;))

annot_col &lt;- samples %&gt;%
  column_to_rownames(&quot;samplename&quot;) %&gt;%
  as.data.frame()

ann_colors &lt;- list(
  condition = as.factor(c(Saline = &quot;slateblue2&quot;, Maintenance = &quot;limegreen&quot;, Abstinence = &quot;violetred2&quot;)),
  Group = as.factor(c(Saline = &quot;slateblue2&quot;, Maintenance = &quot;limegreen&quot;, Abstinence = &quot;violetred2&quot;))
)</code></pre>
<div id="unsupervised-clustering" class="section level3">
<h3>Unsupervised Clustering</h3>
<pre class="r"><code>pheatmap::pheatmap(
  mat = data_mod,
  color = colorRampPalette(rev(brewer.pal(11, &quot;RdYlBu&quot;)))(155),
  scale = &quot;row&quot;, # Scale genes to Z-score (how many standard deviations),
  annotation_col = annot_col,
  # annotation_colors = ann_colors,# Change the default colors of the annotations
  fontsize = 3, # Make fonts smaller
  cellwidth = 15, # Make the cells wider
  cellheight = 3, # Make the cells wider
  show_colnames = T,
  cluster_rows = T,
  cluster_cols = T,
  annotation_legend = T,
  legend = T,
  show_rownames = T,
  clustering_distance_rows = &quot;correlation&quot;,
  clustering_method = &quot;ward.D&quot;,
  main = &quot;Heatmap of Significant DEGs -- Unsupervised Clustering&quot;,
  cluster_row_slices = TRUE,
  show_row_dend = FALSE,
  cutree_rows = 4,
  cutree_cols = 3,
  treeheight_row = 10,
  treeheight_col = 10,
  # kmeans_k = 4,
  # display_numbers = TRUE,
  # filename = &quot;plots/MIVSA_Heatmap_DEGs_Final.pdf&quot;,
  legend_labels = c(&quot;Saline&quot;, &quot;Maintenance&quot;, &quot;Abstinence&quot;),
  labels_row = matrownames_data_mod$symbol
)</code></pre>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Filter%20Heatmaps%20by%20DEGs2Unsupervised-1.png" width="768" style="display: block; margin: auto;" /></p>
</div>
<div id="supervised-clustering" class="section level3">
<h3>Supervised Clustering</h3>
<pre class="r"><code>pheatmap::pheatmap(
  mat = data_mod,
  color = colorRampPalette(rev(brewer.pal(11, &quot;RdYlBu&quot;)))(155),
  scale = &quot;row&quot;, # Scale genes to Z-score (how many standard deviations),
  annotation_col = annot_col,
  # annotation_colors = ann_colors,# Change the default colors of the annotations
  fontsize = 8, # Make fonts smaller
  cellwidth = 15, # Make the cells wider
  cellheight = 3, # Make the cells wider
  show_colnames = F,
  cluster_rows = T,
  cluster_cols = F,
  annotation_legend = T,
  legend = T,
  show_rownames = F,
  clustering_distance_rows = &quot;correlation&quot;,
  clustering_method = &quot;ward.D&quot;,
  main = &quot;Heatmap of Significant DEGs -- Supervised Clustering&quot;,
  cluster_row_slices = TRUE,
  show_row_dend = FALSE,
  cutree_rows = 4,
  cutree_cols = 3,
  gaps_col = c(5, 10),
  treeheight_row = 0,
  treeheight_col = 10,
  # kmeans_k = 4,
  # display_numbers = TRUE,
  # filename = &quot;plots/MIVSA_Heatmap_DEGs_Final_Unclustered_Nolabs.pdf&quot;,
  legend_labels = c(&quot;Saline&quot;, &quot;Maintenance&quot;, &quot;Abstinence&quot;)
)</code></pre>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Filter%20Heatmaps%20by%20DEGs2-1.png" width="768" style="display: block; margin: auto;" /></p>
<pre class="r"><code># labels_row = matrownames_data_mod$symbol)</code></pre>
</div>
<div id="supervised-clustering-and-condensed" class="section level3">
<h3>Supervised Clustering and Condensed</h3>
<pre class="r"><code>pheatmap::pheatmap(
  mat = data_mod,
  color = colorRampPalette(rev(brewer.pal(11, &quot;RdYlBu&quot;)))(155),
  scale = &quot;row&quot;, # Scale genes to Z-score (how many standard deviations),
  annotation_col = annot_col,
  # annotation_colors = ann_colors,# Change the default colors of the annotations
  fontsize = 12, # Make fonts smaller
  # cellwidth = 15, # Make the cells wider
  # cellheight = 3, # Make the cells wider
  show_colnames = F,
  cluster_rows = T,
  cluster_cols = F,
  annotation_legend = T,
  legend = T,
  show_rownames = F,
  clustering_distance_rows = &quot;correlation&quot;,
  clustering_method = &quot;ward.D&quot;,
  main = &quot;Heatmap of Significant DEGs -- Supervised and Condensed&quot;,
  legend_labels = c(&quot;Saline&quot;, &quot;Maintenance&quot;, &quot;Abstinence&quot;),
  cluster_row_slices = TRUE,
  show_row_dend = FALSE,
  gaps_col = c(5, 10),
  cutree_rows = 4,
  cutree_cols = 3,
  treeheight_row = 15,
  treeheight_col = 10
)</code></pre>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Filter%20Heatmaps%20by%20DEGs3-1.png" width="768" style="display: block; margin: auto;" /></p>
<pre class="r"><code># kmeans_k = 4,
# display_numbers = TRUE,
# filename = &quot;plots/MIVSA_Heatmap_DEGs_Final_Unclustered_Nolabs_Condensed.pdf&quot;)
# labels_row = matrownames_data_mod$symbol)</code></pre>
</div>
</div>
<div id="volcano-plots" class="section level2">
<h2>Volcano Plots</h2>
<div id="visualize-results-volcano-plots-non-interactive"
class="section level3">
<h3>Visualize Results: Volcano Plots (Non-Interactive)</h3>
<p>Volcano plots are generated by gathering the FDR corrected p-values
from each analysis. The adjusted p-values undergo a -log10
transformation to generate these FDR corrected values. Any rows
containing “NA” are ommitted from analysis. Data points are colored
based on increasing or decreasing value and plotted using ggplot2.
Adjusted p-value cutoff is 1.3 following log transformation.(-log10 0.05
= 1.3)</p>
<pre class="r"><code># Gather Log-fold change and FDR-corrected pvalues from DESeq2 results
# Change pvalues to -log10 (1.3 = 0.05)
data &lt;- data.frame(
  gene = (res$symbol),
  pval = -log10(res$padj),
  lfc = res$log2FoldChange
)

data2 &lt;- data.frame(
  gene = (res2$symbol),
  pval = -log10(res2$padj),
  lfc = res2$log2FoldChange
)

data3 &lt;- data.frame(
  gene = (res3$symbol),
  pval = -log10(res3$padj),
  lfc = res3$log2FoldChange
)


# Remove any rows that have NA as an entry
data &lt;- na.omit(data)

data2 &lt;- na.omit(data2)

data3 &lt;- na.omit(data3)

# Color the points which are up or down
## If fold-change &gt; 0 and pvalue &gt; 1.3 (Increased significant)
## If fold-change &lt; 0 and pvalue &gt; 1.3 (Decreased significant)

data &lt;- dplyr::mutate(data, color = case_when(
  data$lfc &gt; 0 &amp; data$pval &gt; 1.3 ~ &quot;Increased in Maintenance&quot;,
  data$lfc &lt; 0 &amp; data$pval &gt; 1.3 ~ &quot;Decreased in Maintenance&quot;,
  data$pval &lt; 1.3 ~ &quot;Nonsignificant&quot;
))

data2 &lt;- dplyr::mutate(data2, color = case_when(
  data2$lfc &gt; 0 &amp; data2$pval &gt; 1.3 ~ &quot;Increased in Abstinence&quot;,
  data2$lfc &lt; 0 &amp; data2$pval &gt; 1.3 ~ &quot;Decreased in Abstinence&quot;,
  data2$pval &lt; 1.3 ~ &quot;Nonsignificant&quot;
))

data3 &lt;- dplyr::mutate(data3, color = case_when(
  data3$lfc &gt; 0 &amp; data3$pval &gt; 1.3 ~ &quot;Increased in Abstinence&quot;,
  data3$lfc &lt; 0 &amp; data3$pval &gt; 1.3 ~ &quot;Decreased in Abstinence&quot;,
  data3$pval &lt; 1.3 ~ &quot;Nonsignificant&quot;
))



# Make a basic ggplot2 object with x-y values
vol &lt;- ggplot(data, aes(x = lfc, y = pval, color = color))

vol2 &lt;- ggplot(data2, aes(x = lfc, y = pval, color = color))

vol3 &lt;- ggplot(data3, aes(x = lfc, y = pval, color = color))



# Add ggplot2 layers

y.axis.text &lt;- element_text(face = &quot;bold&quot;, color = &quot;black&quot;, size = 7)
x.axis.text &lt;- element_text(face = &quot;bold&quot;, color = &quot;black&quot;, size = 7)
plot.title.text &lt;- element_text(face = &quot;bold&quot;, color = &quot;black&quot;, size = 9, hjust = 0.5)
legend.text &lt;- element_text(face = &quot;bold&quot;, color = &quot;black&quot;, size = 5)
legend.title.text &lt;- element_text(face = &quot;bold&quot;, color = &quot;black&quot;, size = 6)
axis.title &lt;- element_text(face = &quot;bold&quot;, color = &quot;black&quot;, size = 6)</code></pre>
<pre class="r"><code>vol_final &lt;- vol +
  ggtitle(label = &quot;Maintenance vs Saline&quot;) +
  geom_point(size = 1.5, alpha = 0.8, na.rm = T) +
  scale_color_manual(
    name = &quot;Directionality&quot;,
    values = c(&quot;Increased in Maintenance&quot; = &quot;#008B00&quot;, &quot;Decreased in Maintenance&quot; = &quot;#CD4F39&quot;, &quot;Nonsignificant&quot; = &quot;darkgray&quot;)
  ) +
  theme_bw(base_size = 14) + # change overall theme+
  theme(
    legend.position = &quot;right&quot;,
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, &quot;cm&quot;), # change legend key size
    legend.key.height = unit(0.5, &quot;cm&quot;), # change legend key height
    legend.key.width = unit(0.5, &quot;cm&quot;)
  ) + # change legend key width
  xlab(expression(log[2](&quot;Fold Change&quot;))) + # Change X-Axis label
  ylab(expression(-log[10](&quot;adjusted p-value&quot;))) + # Change Y-Axis label
  geom_hline(yintercept = 1.3, colour = &quot;darkgrey&quot;) + # Add p-adj value cutoff line
  scale_y_continuous(trans = &quot;log1p&quot;) # Scale y-axis due to large p-values

vol2_final &lt;- vol2 +
  ggtitle(label = &quot;Abstinence vs Saline&quot;) +
  geom_point(size = 1.5, alpha = 0.8, na.rm = T) +
  scale_color_manual(
    name = &quot;Directionality&quot;,
    values = c(&quot;Increased in Abstinence&quot; = &quot;#008B00&quot;, &quot;Decreased in Abstinence&quot; = &quot;#CD4F39&quot;, &quot;Nonsignificant&quot; = &quot;darkgray&quot;)
  ) +
  theme_bw(base_size = 14) + # change overall theme
  theme(
    legend.position = &quot;right&quot;,
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, &quot;cm&quot;), # change legend key size
    legend.key.height = unit(0.5, &quot;cm&quot;), # change legend key height
    legend.key.width = unit(0.5, &quot;cm&quot;)
  ) + # change legend key width
  xlab(expression(log[2](&quot;Fold Change&quot;))) + # Change X-Axis label
  ylab(expression(-log[10](&quot;adjusted p-value&quot;))) + # Change Y-Axis label
  geom_hline(yintercept = 1.3, colour = &quot;darkgrey&quot;) + # Add p-adj value cutoff line
  scale_y_continuous(trans = &quot;log1p&quot;) # Scale y-axis due to large p-values

vol3_final &lt;- vol3 +
  ggtitle(label = &quot;Abstinence vs Maintenance&quot;) +
  geom_point(size = 1.5, alpha = 0.8, na.rm = T) +
  scale_color_manual(
    name = &quot;Directionality&quot;,
    values = c(&quot;Increased in Abstinence&quot; = &quot;#008B00&quot;, &quot;Decreased in Abstinence&quot; = &quot;#CD4F39&quot;, &quot;Nonsignificant&quot; = &quot;darkgray&quot;)
  ) +
  theme_bw(base_size = 14) + # change overall theme
  theme(
    legend.position = &quot;right&quot;,
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, &quot;cm&quot;), # change legend key size
    legend.key.height = unit(0.5, &quot;cm&quot;), # change legend key height
    legend.key.width = unit(0.5, &quot;cm&quot;)
  ) + # change legend key width
  xlab(expression(log[2](&quot;fold change&quot;))) + # Change X-Axis label
  ylab(expression(-log[10](&quot;adjusted p-value&quot;))) + # Change Y-Axis label
  geom_hline(yintercept = 1.3, colour = &quot;darkgrey&quot;) + # Add p-adj value cutoff line
  scale_y_continuous(trans = &quot;log1p&quot;) # Scale y-axis due to large p-values

ggarrange(vol_final, vol2_final, vol3_final, labels = c(&quot;A&quot;, &quot;B&quot;, &quot;C&quot;))</code></pre>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Volcano%20Plots%20--%20Visualize-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="pathway-analysis" class="section level1">
<h1>Pathway Analysis</h1>
<div id="pathway-analysis-go" class="section level2 tabset">
<h2 class="tabset">Pathway Analysis GO</h2>
<p>First, the background for the GO universe must be set. In order to do
so we will subset the first results file “res” for any genes that map to
entrez IDs and then we only take the unique ones. This leaves us with
~31,000 genes for the remaining background which we list as our
microglial expressed genes.</p>
<p>Pathway analysis is conducted using enrichGo and enrichKEGG based on
a p-value of 0.05 and a q-value of 0.10 and including all significantly
differentially expressed genes.</p>
<pre class="r"><code># Remove any genes that have greater than adj.pvalue 0.05
results_sig_entrez &lt;- subset(results_sig, lfcSE &lt;= 1.5)
results_sig_entrez2 &lt;- subset(results_sig2, lfcSE &lt;= 1.5)
results_sig_entrez3 &lt;- subset(results_sig3, lfcSE &lt;= 1.5)

# Remove any genes that have greater than 1.5 LFC Standard Error
results_sig_entrez &lt;- subset(results_sig_entrez, padj &lt;= 0.05)
results_sig_entrez2 &lt;- subset(results_sig_entrez2, padj &lt;= 0.05)
results_sig_entrez3 &lt;- subset(results_sig_entrez3, padj &lt;= 0.05)

# Remove any genes that do not have any entrez identifiers
results_sig_entrez &lt;- subset(results_sig_entrez, is.na(entrez) == FALSE)
results_sig_entrez2 &lt;- subset(results_sig_entrez2, is.na(entrez) == FALSE)
results_sig_entrez3 &lt;- subset(results_sig_entrez3, is.na(entrez) == FALSE)

# Create a matrix of gene log2 fold changes
gene_matrix &lt;- results_sig_entrez$log2FoldChange
gene_matrix2 &lt;- results_sig_entrez2$log2FoldChange
gene_matrix3 &lt;- results_sig_entrez3$log2FoldChange

# Add the entrezID&#39;s as names for each logFC entry
names(gene_matrix) &lt;- results_sig_entrez$entrez
names(gene_matrix2) &lt;- results_sig_entrez2$entrez
names(gene_matrix3) &lt;- results_sig_entrez3$entrez


go &lt;- enrichGO(
  gene = names(gene_matrix),
  OrgDb = &quot;org.Mm.eg.db&quot;,
  readable = T,
  universe = gene_matrix_background,
  ont = &quot;ALL&quot;,
  pAdjustMethod = &quot;fdr&quot;,
  pvalueCutoff = 0.05,
  qvalueCutoff = 0.10
)

go2 &lt;- enrichGO(
  gene = names(gene_matrix2),
  OrgDb = &quot;org.Mm.eg.db&quot;,
  readable = T,
  universe = gene_matrix_background,
  ont = &quot;ALL&quot;,
  pAdjustMethod = &quot;fdr&quot;,
  pvalueCutoff = 0.05,
  qvalueCutoff = 0.10
)

go3 &lt;- enrichGO(
  gene = names(gene_matrix3),
  OrgDb = &quot;org.Mm.eg.db&quot;,
  readable = T,
  universe = gene_matrix_background,
  ont = &quot;ALL&quot;,
  pAdjustMethod = &quot;fdr&quot;,
  pvalueCutoff = 0.05,
  qvalueCutoff = 0.10
)</code></pre>
<div id="maintenance-vs-saline-4" class="section level3">
<h3>Maintenance vs Saline</h3>
<pre class="r"><code>kable(head(go, 10), caption = &quot;Top 10 GO Pathways for Maintenance vs Saline&quot;, format = &quot;html&quot;, align = &quot;l&quot;, booktabs = TRUE) %&gt;%
  kable_styling(html_font = &quot;Arial&quot;, font_size = 10, bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;, &quot;condensed&quot;, &quot;responsive&quot;)) %&gt;%
  scroll_box(width = &quot;1000px&quot;, height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:1000px; ">
<table class="table table-striped table-hover table-condensed table-responsive" style="font-size: 10px; font-family: Arial; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Top 10 GO Pathways for Maintenance vs Saline
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ONTOLOGY
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
GeneRatio
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
BgRatio
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
p.adjust
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
qvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
<a href="GO:0048709" class="uri">GO:0048709</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0048709" class="uri">GO:0048709</a>
</td>
<td style="text-align:left;">
oligodendrocyte differentiation
</td>
<td style="text-align:left;">
18/417
</td>
<td style="text-align:left;">
118/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
1e-07
</td>
<td style="text-align:left;">
1e-07
</td>
<td style="text-align:left;">
Cntn2/Aspa/Omg/Cnp/Tppp/Enpp2/Sox10/Tnfrsf21/Myrf/Il33/Opalin/Abca2/Mal/Tspan2/Hes5/Mag/Fa2h/Plp1
</td>
<td style="text-align:left;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0022010" class="uri">GO:0022010</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0022010" class="uri">GO:0022010</a>
</td>
<td style="text-align:left;">
central nervous system myelination
</td>
<td style="text-align:left;">
10/417
</td>
<td style="text-align:left;">
29/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
1e-07
</td>
<td style="text-align:left;">
1e-07
</td>
<td style="text-align:left;">
Cntn2/Omg/Sox10/Myrf/Abca2/Mal/Hes5/Mag/Fa2h/Plp1
</td>
<td style="text-align:left;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0032291" class="uri">GO:0032291</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0032291" class="uri">GO:0032291</a>
</td>
<td style="text-align:left;">
axon ensheathment in central nervous system
</td>
<td style="text-align:left;">
10/417
</td>
<td style="text-align:left;">
29/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
1e-07
</td>
<td style="text-align:left;">
1e-07
</td>
<td style="text-align:left;">
Cntn2/Omg/Sox10/Myrf/Abca2/Mal/Hes5/Mag/Fa2h/Plp1
</td>
<td style="text-align:left;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0050684" class="uri">GO:0050684</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0050684" class="uri">GO:0050684</a>
</td>
<td style="text-align:left;">
regulation of mRNA processing
</td>
<td style="text-align:left;">
19/417
</td>
<td style="text-align:left;">
149/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
1e-07
</td>
<td style="text-align:left;">
1e-07
</td>
<td style="text-align:left;">
Ptbp1/Cirbp/Fxr2/Zfp36l1/Hnrnpk/Ccnb1/Acin1/Tra2b/Srsf3/Srsf7/Rbm39/Fxr1/Srsf10/Srpk2/Tra2a/Tia1/Hspa8/Rbm5/Rbm3
</td>
<td style="text-align:left;">
19
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0006397" class="uri">GO:0006397</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0006397" class="uri">GO:0006397</a>
</td>
<td style="text-align:left;">
mRNA processing
</td>
<td style="text-align:left;">
32/417
</td>
<td style="text-align:left;">
452/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
3e-07
</td>
<td style="text-align:left;">
3e-07
</td>
<td style="text-align:left;">
Ptbp1/Cirbp/Fxr2/Prpf39/Zfp36l1/Srsf5/Hnrnpk/Ccnb1/Acin1/Tra2b/Luc7l/Srsf3/U2af1/Akap8l/Hnrnpm/Cdc5l/Srsf7/Thoc1/Cir1/Mfap1a/Rbm39/Fxr1/Prpf38b/Srsf10/Srpk2/Fip1l1/Tra2a/Tia1/Aplp1/Hspa8/Rbm5/Rbm3
</td>
<td style="text-align:left;">
32
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0006457" class="uri">GO:0006457</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0006457" class="uri">GO:0006457</a>
</td>
<td style="text-align:left;">
protein folding
</td>
<td style="text-align:left;">
19/417
</td>
<td style="text-align:left;">
164/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
3e-07
</td>
<td style="text-align:left;">
3e-07
</td>
<td style="text-align:left;">
Hspd1/Cct4/Ahsa2/Ahsa1/Cct5/Cct8/Tcp1/Hsp90ab1/Cdc37l1/Hspa5/Ppig/Ppid/Dnaja1/Hsph1/Cct7/Fkbp4/Chordc1/Hspa8/Cryab
</td>
<td style="text-align:left;">
19
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0048024" class="uri">GO:0048024</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0048024" class="uri">GO:0048024</a>
</td>
<td style="text-align:left;">
regulation of mRNA splicing, via spliceosome
</td>
<td style="text-align:left;">
16/417
</td>
<td style="text-align:left;">
112/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
3e-07
</td>
<td style="text-align:left;">
3e-07
</td>
<td style="text-align:left;">
Ptbp1/Cirbp/Fxr2/Hnrnpk/Acin1/Tra2b/Srsf3/Srsf7/Rbm39/Fxr1/Srsf10/Tra2a/Tia1/Hspa8/Rbm5/Rbm3
</td>
<td style="text-align:left;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0000375" class="uri">GO:0000375</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0000375" class="uri">GO:0000375</a>
</td>
<td style="text-align:left;">
RNA splicing, via transesterification reactions
</td>
<td style="text-align:left;">
24/417
</td>
<td style="text-align:left;">
270/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
3e-07
</td>
<td style="text-align:left;">
3e-07
</td>
<td style="text-align:left;">
Ptbp1/Cirbp/Fxr2/Prpf39/Srsf5/Hnrnpk/Acin1/Tra2b/Luc7l/Srsf3/U2af1/Hnrnpm/Cdc5l/Srsf7/Mfap1a/Rbm39/Fxr1/Srsf10/Srpk2/Tra2a/Tia1/Hspa8/Rbm5/Rbm3
</td>
<td style="text-align:left;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0000377" class="uri">GO:0000377</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0000377" class="uri">GO:0000377</a>
</td>
<td style="text-align:left;">
RNA splicing, via transesterification reactions with bulged adenosine as
nucleophile
</td>
<td style="text-align:left;">
24/417
</td>
<td style="text-align:left;">
270/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
3e-07
</td>
<td style="text-align:left;">
3e-07
</td>
<td style="text-align:left;">
Ptbp1/Cirbp/Fxr2/Prpf39/Srsf5/Hnrnpk/Acin1/Tra2b/Luc7l/Srsf3/U2af1/Hnrnpm/Cdc5l/Srsf7/Mfap1a/Rbm39/Fxr1/Srsf10/Srpk2/Tra2a/Tia1/Hspa8/Rbm5/Rbm3
</td>
<td style="text-align:left;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0000398" class="uri">GO:0000398</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0000398" class="uri">GO:0000398</a>
</td>
<td style="text-align:left;">
mRNA splicing, via spliceosome
</td>
<td style="text-align:left;">
24/417
</td>
<td style="text-align:left;">
270/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
3e-07
</td>
<td style="text-align:left;">
3e-07
</td>
<td style="text-align:left;">
Ptbp1/Cirbp/Fxr2/Prpf39/Srsf5/Hnrnpk/Acin1/Tra2b/Luc7l/Srsf3/U2af1/Hnrnpm/Cdc5l/Srsf7/Mfap1a/Rbm39/Fxr1/Srsf10/Srpk2/Tra2a/Tia1/Hspa8/Rbm5/Rbm3
</td>
<td style="text-align:left;">
24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="abstinence-vs-saline-4" class="section level3">
<h3>Abstinence vs Saline</h3>
<pre class="r"><code>kable(head(go2, 10), caption = &quot;Top 10 GO Pathways for Abstinence vs Saline&quot;, format = &quot;html&quot;, align = &quot;l&quot;, booktabs = TRUE) %&gt;%
  kable_styling(html_font = &quot;Arial&quot;, font_size = 10, bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;, &quot;condensed&quot;, &quot;responsive&quot;)) %&gt;%
  scroll_box(width = &quot;1000px&quot;, height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:1000px; ">
<table class="table table-striped table-hover table-condensed table-responsive" style="font-size: 10px; font-family: Arial; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Top 10 GO Pathways for Abstinence vs Saline
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ONTOLOGY
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
GeneRatio
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
BgRatio
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
p.adjust
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
qvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
<a href="GO:0048709" class="uri">GO:0048709</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0048709" class="uri">GO:0048709</a>
</td>
<td style="text-align:left;">
oligodendrocyte differentiation
</td>
<td style="text-align:left;">
20/218
</td>
<td style="text-align:left;">
118/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
Cntn2/Kcnj10/Aspa/Omg/Cnp/Tppp/Clu/Enpp2/Sox10/Cntn1/Olig1/Sox8/Il33/Opalin/Mal/Tspan2/Hes5/Mag/Fa2h/Plp1
</td>
<td style="text-align:left;">
20
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0010001" class="uri">GO:0010001</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0010001" class="uri">GO:0010001</a>
</td>
<td style="text-align:left;">
glial cell differentiation
</td>
<td style="text-align:left;">
25/218
</td>
<td style="text-align:left;">
257/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
Cntn2/Kcnj10/Ifngr1/Aspa/Omg/Nr1d1/Cnp/Tppp/Clu/Enpp2/Sox10/Cntn1/Nrros/Olig1/Sox8/Il33/Opalin/Mal/Tspan2/Plpp3/Hes5/Mag/Fa2h/Cspg5/Plp1
</td>
<td style="text-align:left;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0021782" class="uri">GO:0021782</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0021782" class="uri">GO:0021782</a>
</td>
<td style="text-align:left;">
glial cell development
</td>
<td style="text-align:left;">
18/218
</td>
<td style="text-align:left;">
120/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
Cntn2/Kcnj10/Ifngr1/Omg/Nr1d1/Tppp/Clu/Sox10/Cntn1/Nrros/Olig1/Mal/Tspan2/Hes5/Mag/Fa2h/Cspg5/Plp1
</td>
<td style="text-align:left;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0014003" class="uri">GO:0014003</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0014003" class="uri">GO:0014003</a>
</td>
<td style="text-align:left;">
oligodendrocyte development
</td>
<td style="text-align:left;">
13/218
</td>
<td style="text-align:left;">
48/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
Cntn2/Kcnj10/Omg/Tppp/Clu/Sox10/Cntn1/Olig1/Mal/Hes5/Mag/Fa2h/Plp1
</td>
<td style="text-align:left;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0022010" class="uri">GO:0022010</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0022010" class="uri">GO:0022010</a>
</td>
<td style="text-align:left;">
central nervous system myelination
</td>
<td style="text-align:left;">
11/218
</td>
<td style="text-align:left;">
29/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
Cntn2/Kcnj10/Omg/Clu/Sox10/Cntn1/Mal/Hes5/Mag/Fa2h/Plp1
</td>
<td style="text-align:left;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0032291" class="uri">GO:0032291</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0032291" class="uri">GO:0032291</a>
</td>
<td style="text-align:left;">
axon ensheathment in central nervous system
</td>
<td style="text-align:left;">
11/218
</td>
<td style="text-align:left;">
29/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
Cntn2/Kcnj10/Omg/Clu/Sox10/Cntn1/Mal/Hes5/Mag/Fa2h/Plp1
</td>
<td style="text-align:left;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0042063" class="uri">GO:0042063</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0042063" class="uri">GO:0042063</a>
</td>
<td style="text-align:left;">
gliogenesis
</td>
<td style="text-align:left;">
26/218
</td>
<td style="text-align:left;">
358/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
Cntn2/Kcnj10/Ifngr1/Aspa/Omg/Nr1d1/Cnp/Tppp/Clu/Enpp2/Myc/Sox10/Cntn1/Nrros/Olig1/Sox8/Il33/Opalin/Mal/Tspan2/Plpp3/Hes5/Mag/Fa2h/Cspg5/Plp1
</td>
<td style="text-align:left;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0007272" class="uri">GO:0007272</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0007272" class="uri">GO:0007272</a>
</td>
<td style="text-align:left;">
ensheathment of neurons
</td>
<td style="text-align:left;">
16/218
</td>
<td style="text-align:left;">
174/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
Cntn2/Kcnj10/Omg/Tppp/Clu/Sox10/Cntn1/Mal/Cldn11/Tspan2/Ugt8a/Hes5/Mag/Pllp/Fa2h/Plp1
</td>
<td style="text-align:left;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0008366" class="uri">GO:0008366</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0008366" class="uri">GO:0008366</a>
</td>
<td style="text-align:left;">
axon ensheathment
</td>
<td style="text-align:left;">
16/218
</td>
<td style="text-align:left;">
174/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
Cntn2/Kcnj10/Omg/Tppp/Clu/Sox10/Cntn1/Mal/Cldn11/Tspan2/Ugt8a/Hes5/Mag/Pllp/Fa2h/Plp1
</td>
<td style="text-align:left;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0042552" class="uri">GO:0042552</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0042552" class="uri">GO:0042552</a>
</td>
<td style="text-align:left;">
myelination
</td>
<td style="text-align:left;">
15/218
</td>
<td style="text-align:left;">
172/21359
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
1e-07
</td>
<td style="text-align:left;">
1e-07
</td>
<td style="text-align:left;">
Cntn2/Kcnj10/Omg/Tppp/Clu/Sox10/Cntn1/Mal/Tspan2/Ugt8a/Hes5/Mag/Pllp/Fa2h/Plp1
</td>
<td style="text-align:left;">
15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="abstinence-vs-maintenance-4" class="section level3">
<h3>Abstinence vs Maintenance</h3>
<pre class="r"><code>kable(head(go3, 10), caption = &quot;Top 10 GO Pathways for Abstinence vs Maintenance&quot;, format = &quot;html&quot;, align = &quot;l&quot;, booktabs = TRUE) %&gt;%
  kable_styling(html_font = &quot;Arial&quot;, font_size = 10, bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;, &quot;condensed&quot;, &quot;responsive&quot;)) %&gt;%
  scroll_box(width = &quot;1000px&quot;, height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:1000px; ">
<table class="table table-striped table-hover table-condensed table-responsive" style="font-size: 10px; font-family: Arial; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Top 10 GO Pathways for Abstinence vs Maintenance
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ONTOLOGY
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
GeneRatio
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
BgRatio
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
p.adjust
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
qvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
<a href="GO:0002764" class="uri">GO:0002764</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0002764" class="uri">GO:0002764</a>
</td>
<td style="text-align:left;">
immune response-regulating signaling pathway
</td>
<td style="text-align:left;">
32/530
</td>
<td style="text-align:left;">
381/21359
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
9.50e-06
</td>
<td style="text-align:left;">
8.00e-06
</td>
<td style="text-align:left;">
Hspd1/Lilrb4a/Lcp2/Havcr2/Gps2/Sarm1/Lgals9/Nr1d1/Cd300a/Sec14l1/Pum2/Psen1/Cd180/Fyb/Tnfrsf21/Nfkbil1/Cd14/Slc15a3/Lpxn/Blnk/Dgkz/Rbck1/Tifa/Sh2b2/Zc3hav1/Clec4n/C5ar1/Cd33/Bax/Cd276/Tlr13/Tlr7
</td>
<td style="text-align:left;">
32
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0006457" class="uri">GO:0006457</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0006457" class="uri">GO:0006457</a>
</td>
<td style="text-align:left;">
protein folding
</td>
<td style="text-align:left;">
20/530
</td>
<td style="text-align:left;">
164/21359
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
1.08e-05
</td>
<td style="text-align:left;">
9.20e-06
</td>
<td style="text-align:left;">
Hspd1/Cct4/Ahsa2/Ahsa1/Tbce/Cct5/Ppil2/Cct8/Tcp1/Fkbp5/Hsp90ab1/Hspa5/Ppig/Dnajc10/Cct3/Dnajb6/Cct7/Fkbp4/Chordc1/Hspa8
</td>
<td style="text-align:left;">
20
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0000375" class="uri">GO:0000375</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0000375" class="uri">GO:0000375</a>
</td>
<td style="text-align:left;">
RNA splicing, via transesterification reactions
</td>
<td style="text-align:left;">
25/530
</td>
<td style="text-align:left;">
270/21359
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
1.65e-05
</td>
<td style="text-align:left;">
1.39e-05
</td>
<td style="text-align:left;">
Ptbp1/Cirbp/Zc3h10/Nsrp1/Srsf5/Ddx41/Hnrnpk/Acin1/Srsf3/U2af1/Hnrnpm/Cdc5l/Usp49/Rbm17/Fxr1/Mbnl1/Larp7/Sf3a3/Thrap3/Syf2/Srsf10/Srpk2/Hspa8/Rbm5/Srpk3
</td>
<td style="text-align:left;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0000377" class="uri">GO:0000377</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0000377" class="uri">GO:0000377</a>
</td>
<td style="text-align:left;">
RNA splicing, via transesterification reactions with bulged adenosine as
nucleophile
</td>
<td style="text-align:left;">
25/530
</td>
<td style="text-align:left;">
270/21359
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
1.65e-05
</td>
<td style="text-align:left;">
1.39e-05
</td>
<td style="text-align:left;">
Ptbp1/Cirbp/Zc3h10/Nsrp1/Srsf5/Ddx41/Hnrnpk/Acin1/Srsf3/U2af1/Hnrnpm/Cdc5l/Usp49/Rbm17/Fxr1/Mbnl1/Larp7/Sf3a3/Thrap3/Syf2/Srsf10/Srpk2/Hspa8/Rbm5/Srpk3
</td>
<td style="text-align:left;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0000398" class="uri">GO:0000398</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0000398" class="uri">GO:0000398</a>
</td>
<td style="text-align:left;">
mRNA splicing, via spliceosome
</td>
<td style="text-align:left;">
25/530
</td>
<td style="text-align:left;">
270/21359
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
1.65e-05
</td>
<td style="text-align:left;">
1.39e-05
</td>
<td style="text-align:left;">
Ptbp1/Cirbp/Zc3h10/Nsrp1/Srsf5/Ddx41/Hnrnpk/Acin1/Srsf3/U2af1/Hnrnpm/Cdc5l/Usp49/Rbm17/Fxr1/Mbnl1/Larp7/Sf3a3/Thrap3/Syf2/Srsf10/Srpk2/Hspa8/Rbm5/Srpk3
</td>
<td style="text-align:left;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0006986" class="uri">GO:0006986</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0006986" class="uri">GO:0006986</a>
</td>
<td style="text-align:left;">
response to unfolded protein
</td>
<td style="text-align:left;">
16/530
</td>
<td style="text-align:left;">
114/21359
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
1.68e-05
</td>
<td style="text-align:left;">
1.42e-05
</td>
<td style="text-align:left;">
Hspd1/Rhbdd1/Ubxn4/Ddit3/Xbp1/Dnajb9/Hsf1/Hspa5/Dnajc10/Bax/Ppp1r15a/Ccnd1/Herpud1/Abcb10/Hspa8/Manf
</td>
<td style="text-align:left;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0061077" class="uri">GO:0061077</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0061077" class="uri">GO:0061077</a>
</td>
<td style="text-align:left;">
chaperone-mediated protein folding
</td>
<td style="text-align:left;">
12/530
</td>
<td style="text-align:left;">
63/21359
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
2.44e-05
</td>
<td style="text-align:left;">
2.06e-05
</td>
<td style="text-align:left;">
Cct4/Cct5/Cct8/Tcp1/Fkbp5/Hspa5/Cct3/Dnajb6/Cct7/Fkbp4/Chordc1/Hspa8
</td>
<td style="text-align:left;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:1904851" class="uri">GO:1904851</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:1904851" class="uri">GO:1904851</a>
</td>
<td style="text-align:left;">
positive regulation of establishment of protein localization to telomere
</td>
<td style="text-align:left;">
6/530
</td>
<td style="text-align:left;">
10/21359
</td>
<td style="text-align:left;">
0e+00
</td>
<td style="text-align:left;">
2.44e-05
</td>
<td style="text-align:left;">
2.06e-05
</td>
<td style="text-align:left;">
Cct4/Cct5/Cct8/Tcp1/Cct3/Cct7
</td>
<td style="text-align:left;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0072594" class="uri">GO:0072594</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0072594" class="uri">GO:0072594</a>
</td>
<td style="text-align:left;">
establishment of protein localization to organelle
</td>
<td style="text-align:left;">
31/530
</td>
<td style="text-align:left;">
416/21359
</td>
<td style="text-align:left;">
1e-07
</td>
<td style="text-align:left;">
3.04e-05
</td>
<td style="text-align:left;">
2.57e-05
</td>
<td style="text-align:left;">
Hspd1/Tpr/Pex3/Pttg1ip/Ap3d1/Ddit3/Xbp1/Cct4/Rpain/Akap1/Srp68/Psen1/Ipo4/Nipbl/Cct5/Cct8/Tcp1/Hsp90ab1/Pik3c3/Nolc1/Hspa5/Pkig/Sec62/Fbxw7/Cct3/Cct7/Nup62/Bax/Vps37a/Herpud1/Hspa8
</td>
<td style="text-align:left;">
31
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="GO:0051054" class="uri">GO:0051054</a>
</td>
<td style="text-align:left;">
BP
</td>
<td style="text-align:left;">
<a href="GO:0051054" class="uri">GO:0051054</a>
</td>
<td style="text-align:left;">
positive regulation of DNA metabolic process
</td>
<td style="text-align:left;">
26/530
</td>
<td style="text-align:left;">
314/21359
</td>
<td style="text-align:left;">
1e-07
</td>
<td style="text-align:left;">
3.82e-05
</td>
<td style="text-align:left;">
3.22e-05
</td>
<td style="text-align:left;">
Cacybp/Eya4/Pkib/Cct4/Ube2b/Ctc1/Smarce1/Pnp/Cct5/Myc/Hsf1/Cct8/Tcp1/Hsp90ab1/Msh2/Cct3/Pogz/Nbn/Anxa3/Trrap/Cct7/Tigar/Ercc1/Bax/Fus/Brd7
</td>
<td style="text-align:left;">
26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="go-pathways-visualization" class="section level1 tabset">
<h1 class="tabset">GO Pathways Visualization</h1>
<div id="go-dotplot" class="section level2">
<h2>GO DotPlot</h2>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Combined%20GO%20DotPlots-1.png" width="672" style="display: block; margin: auto;" />
## Chord Plots</p>
<pre class="r"><code>library(GOplot)
library(data.table)

go_circle &lt;- read.delim(&quot;/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/final_samples_removed/results/PathwayAnalysis/GO_MIVSA_Maintenance_vs_Saline_circle.txt&quot;, sep = &quot;\t&quot;)
go2_circle &lt;- read.delim(&quot;/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/final_samples_removed/results/PathwayAnalysis/GO_MIVSA_Abstinence_vs_Saline_circle.txt&quot;, sep = &quot;\t&quot;)
go3_circle &lt;- read.delim(&quot;/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/final_samples_removed/results/PathwayAnalysis/GO_MIVSA_Abstinence_vs_Maintenance_circle.txt&quot;, sep = &quot;\t&quot;)

processes1 &lt;- data[c(1:7), ]

processes1_final_circle &lt;- processes1$Description

ids_final_circle &lt;- processes1$ID


processes2 &lt;- data[c(8:14), ]

processes2_final_circle &lt;- processes2$Description

ids2_final_circle &lt;- processes2$ID


processes3 &lt;- data[c(15:21), ]

processes3_final_circle &lt;- processes3$Description

ids3_final_circle &lt;- processes3$ID



circletest_genes1 &lt;- data.frame(matrix(ncol = 2, nrow = 432))
colnames(circletest_genes1) &lt;- c(&quot;Genes&quot;, &quot;L2FC&quot;)

circletest_DEGsgenes1 &lt;- as.data.table(results_sig)
circletest_DEGsgenes1$ID &lt;- circletest_DEGsgenes1$symbol
circletest_DEGsgenes1$logFC &lt;- circletest_DEGsgenes1$log2FoldChange
circletest_DEGsgenes1$AveExpr &lt;- circletest_DEGsgenes1$baseMean
circletest_DEGsgenes1$t &lt;- circletest_DEGsgenes1$stat
circletest_DEGsgenes1$P.Value &lt;- circletest_DEGsgenes1$pvalue
circletest_DEGsgenes1$adj.P.Value &lt;- circletest_DEGsgenes1$padj
circletest_DEGsgenes1$B &lt;- circletest_DEGsgenes1$stat



circletest_DEGsgenes1_final &lt;- circletest_DEGsgenes1[, 11:17, with = FALSE]

circletest_genes1$Genes &lt;- names(gene_matrix)
circletest_genes1$ID &lt;- mapIds(
  x = org.Mm.eg.db,
  keys = circletest_genes1$Genes,
  column = &quot;SYMBOL&quot;,
  keytype = &quot;ENTREZID&quot;,
  multiVals = &quot;first&quot;
)
circletest_genes1$logFC &lt;- (gene_matrix)


circletest_genes1 &lt;- mutate_all(circletest_genes1, .funs = toupper)
circletest_genes1_final &lt;- circletest_genes1[, 3:4]
circletest_genes1_final$logFC &lt;- as.numeric(circletest_genes1_final$logFC)



circletest_genes2 &lt;- data.frame(matrix(ncol = 2, nrow = 227))
colnames(circletest_genes2) &lt;- c(&quot;Genes&quot;, &quot;L2FC&quot;)

circletest_DEGsgenes2 &lt;- as.data.table(results_sig2)
circletest_DEGsgenes2$ID &lt;- circletest_DEGsgenes2$symbol
circletest_DEGsgenes2$logFC &lt;- circletest_DEGsgenes2$log2FoldChange
circletest_DEGsgenes2$AveExpr &lt;- circletest_DEGsgenes2$baseMean
circletest_DEGsgenes2$t &lt;- circletest_DEGsgenes2$stat
circletest_DEGsgenes2$P.Value &lt;- circletest_DEGsgenes2$pvalue
circletest_DEGsgenes2$adj.P.Value &lt;- circletest_DEGsgenes2$padj
circletest_DEGsgenes2$B &lt;- circletest_DEGsgenes2$stat



circletest_DEGsgenes2_final &lt;- circletest_DEGsgenes2[, 11:17, with = FALSE]

circletest_genes2$Genes &lt;- names(gene_matrix2)
circletest_genes2$ID &lt;- mapIds(
  x = org.Mm.eg.db,
  keys = circletest_genes2$Genes,
  column = &quot;SYMBOL&quot;,
  keytype = &quot;ENTREZID&quot;,
  multiVals = &quot;first&quot;
)
circletest_genes2$logFC &lt;- (gene_matrix2)

circletest_genes2 &lt;- mutate_all(circletest_genes2, .funs = toupper)
circletest_genes2_final &lt;- circletest_genes2[, 3:4]
circletest_genes2_final$logFC &lt;- as.numeric(circletest_genes2_final$logFC)





circletest_genes3 &lt;- data.frame(matrix(ncol = 2, nrow = 549))
colnames(circletest_genes3) &lt;- c(&quot;Genes&quot;, &quot;L2FC&quot;)

circletest_DEGsgenes3 &lt;- as.data.table(results_sig3)
circletest_DEGsgenes3$ID &lt;- circletest_DEGsgenes3$symbol
circletest_DEGsgenes3$logFC &lt;- circletest_DEGsgenes3$log2FoldChange
circletest_DEGsgenes3$AveExpr &lt;- circletest_DEGsgenes3$baseMean
circletest_DEGsgenes3$t &lt;- circletest_DEGsgenes3$stat
circletest_DEGsgenes3$P.Value &lt;- circletest_DEGsgenes3$pvalue
circletest_DEGsgenes3$adj.P.Value &lt;- circletest_DEGsgenes3$padj
circletest_DEGsgenes3$B &lt;- circletest_DEGsgenes3$stat



circletest_DEGsgenes3_final &lt;- circletest_DEGsgenes3[, 11:17, with = FALSE]

circletest_genes3$Genes &lt;- names(gene_matrix3)
circletest_genes3$ID &lt;- mapIds(
  x = org.Mm.eg.db,
  keys = circletest_genes3$Genes,
  column = &quot;SYMBOL&quot;,
  keytype = &quot;ENTREZID&quot;,
  multiVals = &quot;first&quot;
)
circletest_genes3$logFC &lt;- (gene_matrix3)


circletest_genes3 &lt;- mutate_all(circletest_genes3, .funs = toupper)
circletest_genes3_final &lt;- circletest_genes3[, 3:4]
circletest_genes3_final$logFC &lt;- as.numeric(circletest_genes3_final$logFC)</code></pre>
<pre class="r"><code>circ &lt;- circle_dat(go_circle, circletest_DEGsgenes1_final)
circ2 &lt;- circle_dat(go2_circle, circletest_DEGsgenes2_final)
circ3 &lt;- circle_dat(go3_circle, circletest_DEGsgenes3_final)

chord &lt;- chord_dat(circ, circletest_genes1_final, processes1_final_circle)

chord2 &lt;- chord_dat(circ2, circletest_genes2_final, processes2_final_circle)

chord3 &lt;- chord_dat(circ3, circletest_genes3_final, processes3_final_circle)


p5 &lt;- GOChord(chord, space = 0, gene.order = &quot;logFC&quot;, gene.space = 0.15, gene.size = 4, nlfc = 1, process.label = 14, limit = c(1, 0), ribbon.col = c(&quot;#478FCD&quot;, &quot;#FEC111&quot;, &quot;#8D268B&quot;, &quot;#0FB34C&quot;, &quot;#39C0C4&quot;, &quot;#F16739&quot;, &quot;#607D8B&quot;), border.size = 0, lfc.col = c(&quot;#E31A1C&quot;, &quot;#33A02C&quot;, &quot;#125AD9&quot;), lfc.min = -2, lfc.max = 2)
p6 &lt;- GOChord(chord2, space = 0, gene.order = &quot;logFC&quot;, gene.space = 0.15, gene.size = 4, nlfc = 1, process.label = 14, limit = c(1, 0), ribbon.col = c(&quot;#478FCD&quot;, &quot;#FEC111&quot;, &quot;#8D268B&quot;, &quot;#0FB34C&quot;, &quot;#39C0C4&quot;, &quot;#F16739&quot;, &quot;#607D8B&quot;), border.size = 0, lfc.col = c(&quot;#E31A1C&quot;, &quot;#33A02C&quot;, &quot;#125AD9&quot;), lfc.min = -2, lfc.max = 2)
p7 &lt;- GOChord(chord3, space = 0, gene.order = &quot;logFC&quot;, gene.space = 0.15, gene.size = 4, nlfc = 1, process.label = 14, limit = c(1, 0), ribbon.col = c(&quot;#478FCD&quot;, &quot;#FEC111&quot;, &quot;#8D268B&quot;, &quot;#0FB34C&quot;, &quot;#39C0C4&quot;, &quot;#F16739&quot;, &quot;#607D8B&quot;), border.size = 0, lfc.col = c(&quot;#E31A1C&quot;, &quot;#33A02C&quot;, &quot;#125AD9&quot;), lfc.min = -2, lfc.max = 2)



y.axis.text &lt;- element_text(family = &quot;NimbusSanCond&quot;, face = &quot;bold&quot;, color = &quot;black&quot;, size = 7)
x.axis.text &lt;- element_text(family = &quot;NimbusSanCond&quot;, face = &quot;bold&quot;, color = &quot;black&quot;, size = 7)
plot.title.text &lt;- element_text(family = &quot;NimbusSanCond&quot;, face = &quot;bold&quot;, color = &quot;black&quot;, size = 9, hjust = 0.5)
legend.text &lt;- element_text(family = &quot;NimbusSanCond&quot;, face = &quot;bold&quot;, color = &quot;black&quot;, size = 6)
legend.title.text &lt;- element_text(family = &quot;NimbusSanCond&quot;, face = &quot;bold&quot;, color = &quot;black&quot;, size = 10)
axis.title &lt;- element_text(family = &quot;NimbusSanCond&quot;, face = &quot;bold&quot;, color = &quot;black&quot;, size = 6)
text &lt;- element_text(family = &quot;NimbusSanCond&quot;, face = &quot;bold&quot;, color = &quot;black&quot;, size = 6)


p8 &lt;- p5 +
  theme(
    legend.position = &quot;bottom&quot;,
    legend.text = legend.text,
    legend.title = legend.title.text,
    text = text,
    legend.key.width = unit(3, &quot;cm&quot;)
  ) # change legend key width

p9 &lt;- p6 +
  theme(
    legend.position = &quot;bottom&quot;,
    legend.text = legend.text,
    legend.title = legend.title.text,
    text = text,
    legend.key.width = unit(3, &quot;cm&quot;)
  ) # change legend key width

p10 &lt;- p7 +
  theme(
    legend.position = &quot;bottom&quot;,
    legend.text = legend.text,
    legend.title = legend.title.text,
    text = text,
    legend.key.width = unit(3, &quot;cm&quot;)
  ) # change legend key width


cowplot::plot_grid(p8, p9, p10,
  ncol = 3, labels = c(&quot;A. Maintenance vs Saline&quot;, &quot;B. Abstinence vs Saline&quot;, &quot;C. Abstinence vs Maintenance&quot;),
  label_fontfamily = &quot;arial&quot;,
  label_fontface = &quot;bold&quot;,
  label_colour = &quot;black&quot;,
  label_size = 50
)</code></pre>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/unnamed-chunk-3-1.png" width="5760" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="determining-directionality-of-gopathways"
class="section level1 tabset">
<h1 class="tabset">Determining Directionality of GOPathways</h1>
<div
id="log2foldchange-of-genes-in-maintenance-vs-saline-gene-ontologies"
class="section level2">
<h2>Log2FoldChange of Genes in Maintenance vs Saline Gene
Ontologies</h2>
<pre class="r"><code>p9</code></pre>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Barplots%20Maintenance%20vs%20Saline-1.png" width="1152" style="display: block; margin: auto;" /></p>
</div>
<div
id="log2foldchange-of-genes-in-abstinence-vs-saline-gene-ontologies"
class="section level2">
<h2>Log2FoldChange of Genes in Abstinence vs Saline Gene Ontologies</h2>
<pre class="r"><code>p17</code></pre>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Barplots%20Abstinence%20vs%20Saline-1.png" width="1152" style="display: block; margin: auto;" /></p>
</div>
<div
id="log2foldchange-of-genes-in-abstinence-vs-maintenance-gene-ontologies"
class="section level2">
<h2>Log2FoldChange of Genes in Abstinence vs Maintenance Gene
Ontologies</h2>
<pre class="r"><code>p26</code></pre>
<p><img src="figure/MIVSA_RAnalysis_WorkFlowR.Rmd/Barplots%20Abstinence%20vs%20Maintenance-1.png" width="1152" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="session-info" class="section level1">
<h1>Session Info</h1>
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.2.1 (2022-06-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 22.04.3 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.10.0
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.10.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] GOplot_1.0.2                             
 [2] gridExtra_2.3                            
 [3] ggdendro_0.1.23                          
 [4] BiocParallel_1.32.6                      
 [5] viridis_0.6.3                            
 [6] viridisLite_0.4.2                        
 [7] ggbeeswarm_0.7.2                         
 [8] PoiClaClu_1.0.2.1                        
 [9] data.table_1.14.8                        
[10] DEGreport_1.34.0                         
[11] ggrepel_0.9.3                            
[12] conflicted_1.2.0                         
[13] regionReport_1.32.0                      
[14] kableExtra_1.3.4                         
[15] vidger_1.18.0                            
[16] tables_0.9.17                            
[17] knitr_1.43                               
[18] Glimma_2.8.0                             
[19] Mus.musculus_1.3.1                       
[20] TxDb.Mmusculus.UCSC.mm10.knownGene_3.10.0
[21] OrganismDbi_1.40.0                       
[22] GenomicFeatures_1.50.4                   
[23] gplots_3.1.3                             
[24] lubridate_1.9.2                          
[25] forcats_1.0.0                            
[26] stringr_1.5.0                            
[27] purrr_1.0.1                              
[28] readr_2.1.4                              
[29] tibble_3.2.1                             
[30] tidyverse_2.0.0                          
[31] statmod_1.5.0                            
[32] tweeDEseqCountData_1.36.0                
[33] edgeR_3.40.2                             
[34] limma_3.54.2                             
[35] GOSemSim_2.24.0                          
[36] apeglm_1.20.0                            
[37] ggpubr_0.6.0                             
[38] SPIA_2.50.0                              
[39] KEGGgraph_1.58.3                         
[40] ggnewscale_0.4.9                         
[41] enrichplot_1.18.4                        
[42] tidyr_1.3.0                              
[43] WGCNA_1.72-1                             
[44] fastcluster_1.2.3                        
[45] dynamicTreeCut_1.63-1                    
[46] pathview_1.38.0                          
[47] gage_2.48.0                              
[48] dplyr_1.1.2                              
[49] topGO_2.50.0                             
[50] SparseM_1.81                             
[51] graph_1.76.0                             
[52] GO.db_3.16.0                             
[53] RColorBrewer_1.1-3                       
[54] genefilter_1.80.3                        
[55] pheatmap_1.0.12                          
[56] ggsci_3.0.0                              
[57] tximport_1.26.1                          
[58] org.Mm.eg.db_3.16.0                      
[59] AnnotationDbi_1.60.2                     
[60] DOSE_3.24.2                              
[61] ReactomePA_1.42.0                        
[62] biomaRt_2.54.1                           
[63] clusterProfiler_4.6.2                    
[64] ggplot2_3.4.2                            
[65] DESeq2_1.38.3                            
[66] SummarizedExperiment_1.28.0              
[67] Biobase_2.58.0                           
[68] MatrixGenerics_1.10.0                    
[69] matrixStats_1.0.0                        
[70] GenomicRanges_1.50.2                     
[71] GenomeInfoDb_1.34.9                      
[72] IRanges_2.32.0                           
[73] S4Vectors_0.36.2                         
[74] BiocGenerics_0.44.0                      
[75] gprofiler2_0.2.2                         
[76] workflowr_1.7.0                          

loaded via a namespace (and not attached):
  [1] Hmisc_5.1-0                 svglite_2.1.1              
  [3] ps_1.7.5                    Rsamtools_2.14.0           
  [5] foreach_1.5.2               rprojroot_2.0.3            
  [7] crayon_1.5.2                MASS_7.3-60                
  [9] nlme_3.1-162                backports_1.4.1            
 [11] impute_1.72.3               rlang_1.1.1                
 [13] XVector_0.38.0              HDO.db_0.99.1              
 [15] irlba_2.3.5.1               callr_3.7.3                
 [17] filelock_1.0.2              rjson_0.2.21               
 [19] bit64_4.0.5                 glue_1.6.2                 
 [21] mixsqp_0.3-48               rngtools_1.5.2             
 [23] vipor_0.4.5                 parallel_4.2.1             
 [25] processx_3.8.2              DEFormats_1.26.0           
 [27] tidyselect_1.2.0            XML_3.99-0.14              
 [29] GenomicAlignments_1.34.1    xtable_1.8-4               
 [31] magrittr_2.0.3              evaluate_0.21              
 [33] bibtex_0.5.1                cli_3.6.1                  
 [35] zlibbioc_1.44.0             doRNG_1.8.6                
 [37] rstudioapi_0.14             whisker_0.4.1              
 [39] bslib_0.5.0                 rpart_4.1.19               
 [41] derfinderHelper_1.32.0      fastmatch_1.1-3            
 [43] BiocStyle_2.26.0            treeio_1.22.0              
 [45] xfun_0.39                   clue_0.3-64                
 [47] gson_0.1.0                  cluster_2.1.4              
 [49] caTools_1.18.2              tidygraph_1.2.3            
 [51] KEGGREST_1.38.0             logging_0.10-108           
 [53] ape_5.7-1                   Biostrings_2.66.0          
 [55] png_0.1-8                   reshape_0.8.9              
 [57] withr_2.5.0                 bitops_1.0-7               
 [59] ggforce_0.4.1               RBGL_1.74.0                
 [61] plyr_1.8.8                  coda_0.19-4                
 [63] bumphunter_1.40.0           pillar_1.9.0               
 [65] GlobalOptions_0.1.2         cachem_1.0.8               
 [67] fs_1.6.2                    GetoptLong_1.0.5           
 [69] graphite_1.44.0             vctrs_0.6.3                
 [71] generics_0.1.3              tools_4.2.1                
 [73] foreign_0.8-84              beeswarm_0.4.0             
 [75] munsell_0.5.0               tweenr_2.0.2               
 [77] fgsea_1.24.0                DelayedArray_0.24.0        
 [79] fastmap_1.1.1               compiler_4.2.1             
 [81] abind_1.4-5                 httpuv_1.6.11              
 [83] rtracklayer_1.58.0          plotly_4.10.2              
 [85] GenomeInfoDbData_1.2.9      lattice_0.21-8             
 [87] utf8_1.2.3                  later_1.3.1                
 [89] BiocFileCache_2.6.1         jsonlite_1.8.7             
 [91] GGally_2.1.2                scales_1.2.1               
 [93] tidytree_0.4.2              carData_3.0-5              
 [95] lazyeval_0.2.2              promises_1.2.0.1           
 [97] car_3.1-2                   doParallel_1.0.17          
 [99] checkmate_2.2.0             rmarkdown_2.23             
[101] cowplot_1.1.1               webshot_0.5.5              
[103] downloader_0.4              BSgenome_1.66.3            
[105] igraph_1.5.0                survival_3.5-5             
[107] numDeriv_2016.8-1.1         yaml_2.3.7                 
[109] systemfonts_1.0.4           ashr_2.2-54                
[111] SQUAREM_2021.1              htmltools_0.5.5            
[113] memoise_2.0.1               VariantAnnotation_1.44.1   
[115] BiocIO_1.8.0                locfit_1.5-9.8             
[117] graphlayouts_1.0.0          digest_0.6.32              
[119] rappdirs_0.3.3              knitrBootstrap_1.0.2       
[121] emdbook_1.3.13              RSQLite_2.3.1              
[123] yulab.utils_0.0.6           derfinder_1.32.0           
[125] blob_1.2.4                  preprocessCore_1.60.2      
[127] labeling_0.4.2              splines_4.2.1              
[129] Formula_1.2-5               RCurl_1.98-1.12            
[131] broom_1.0.5                 hms_1.1.3                  
[133] ConsensusClusterPlus_1.62.0 colorspace_2.1-0           
[135] base64enc_0.1-3             mnormt_2.1.1               
[137] BiocManager_1.30.21         shape_1.4.6                
[139] aplot_0.1.10                nnet_7.3-19                
[141] sass_0.4.6                  Rcpp_1.0.10                
[143] circlize_0.4.15             mvtnorm_1.2-2              
[145] fansi_1.0.4                 tzdb_0.4.0                 
[147] truncnorm_1.0-9             R6_2.5.1                   
[149] grid_4.2.1                  lifecycle_1.0.3            
[151] curl_5.0.1                  ggsignif_0.6.4             
[153] jquerylib_0.1.4             Matrix_1.5-4.1             
[155] qvalue_2.30.0               org.Hs.eg.db_3.16.0        
[157] iterators_1.0.14            RefManageR_1.4.0           
[159] htmlwidgets_1.6.2           markdown_1.7               
[161] polyclip_1.10-4             shadowtext_0.1.2           
[163] timechange_0.2.0            gridGraphics_0.5-1         
[165] reactome.db_1.82.0          ComplexHeatmap_2.14.0      
[167] rvest_1.0.3                 htmlTable_2.4.1            
[169] patchwork_1.1.2             bdsmatrix_1.3-6            
[171] invgamma_1.1                codetools_0.2-19           
[173] gtools_3.9.4                getPass_0.2-2              
[175] prettyunits_1.1.1           psych_2.3.6                
[177] dbplyr_2.3.2                gtable_0.3.3               
[179] DBI_1.1.3                   git2r_0.32.0               
[181] highr_0.10                  ggfun_0.1.1                
[183] httr_1.4.6                  KernSmooth_2.23-21         
[185] stringi_1.7.12              progress_1.2.2             
[187] reshape2_1.4.4              farver_2.1.1               
[189] annotate_1.76.0             Rgraphviz_2.42.0           
[191] ggtree_3.6.2                xml2_1.3.4                 
[193] bbmle_1.0.25                restfulr_0.0.15            
[195] geneplotter_1.76.0          ggplotify_0.1.1            
[197] bit_4.0.5                   scatterpie_0.2.1           
[199] ggraph_2.1.0                pkgconfig_2.0.3            
[201] rstatix_0.7.2               GenomicFiles_1.34.0        </code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.2.1 (2022-06-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 22.04.3 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.10.0
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.10.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] GOplot_1.0.2                             
 [2] gridExtra_2.3                            
 [3] ggdendro_0.1.23                          
 [4] BiocParallel_1.32.6                      
 [5] viridis_0.6.3                            
 [6] viridisLite_0.4.2                        
 [7] ggbeeswarm_0.7.2                         
 [8] PoiClaClu_1.0.2.1                        
 [9] data.table_1.14.8                        
[10] DEGreport_1.34.0                         
[11] ggrepel_0.9.3                            
[12] conflicted_1.2.0                         
[13] regionReport_1.32.0                      
[14] kableExtra_1.3.4                         
[15] vidger_1.18.0                            
[16] tables_0.9.17                            
[17] knitr_1.43                               
[18] Glimma_2.8.0                             
[19] Mus.musculus_1.3.1                       
[20] TxDb.Mmusculus.UCSC.mm10.knownGene_3.10.0
[21] OrganismDbi_1.40.0                       
[22] GenomicFeatures_1.50.4                   
[23] gplots_3.1.3                             
[24] lubridate_1.9.2                          
[25] forcats_1.0.0                            
[26] stringr_1.5.0                            
[27] purrr_1.0.1                              
[28] readr_2.1.4                              
[29] tibble_3.2.1                             
[30] tidyverse_2.0.0                          
[31] statmod_1.5.0                            
[32] tweeDEseqCountData_1.36.0                
[33] edgeR_3.40.2                             
[34] limma_3.54.2                             
[35] GOSemSim_2.24.0                          
[36] apeglm_1.20.0                            
[37] ggpubr_0.6.0                             
[38] SPIA_2.50.0                              
[39] KEGGgraph_1.58.3                         
[40] ggnewscale_0.4.9                         
[41] enrichplot_1.18.4                        
[42] tidyr_1.3.0                              
[43] WGCNA_1.72-1                             
[44] fastcluster_1.2.3                        
[45] dynamicTreeCut_1.63-1                    
[46] pathview_1.38.0                          
[47] gage_2.48.0                              
[48] dplyr_1.1.2                              
[49] topGO_2.50.0                             
[50] SparseM_1.81                             
[51] graph_1.76.0                             
[52] GO.db_3.16.0                             
[53] RColorBrewer_1.1-3                       
[54] genefilter_1.80.3                        
[55] pheatmap_1.0.12                          
[56] ggsci_3.0.0                              
[57] tximport_1.26.1                          
[58] org.Mm.eg.db_3.16.0                      
[59] AnnotationDbi_1.60.2                     
[60] DOSE_3.24.2                              
[61] ReactomePA_1.42.0                        
[62] biomaRt_2.54.1                           
[63] clusterProfiler_4.6.2                    
[64] ggplot2_3.4.2                            
[65] DESeq2_1.38.3                            
[66] SummarizedExperiment_1.28.0              
[67] Biobase_2.58.0                           
[68] MatrixGenerics_1.10.0                    
[69] matrixStats_1.0.0                        
[70] GenomicRanges_1.50.2                     
[71] GenomeInfoDb_1.34.9                      
[72] IRanges_2.32.0                           
[73] S4Vectors_0.36.2                         
[74] BiocGenerics_0.44.0                      
[75] gprofiler2_0.2.2                         
[76] workflowr_1.7.0                          

loaded via a namespace (and not attached):
  [1] Hmisc_5.1-0                 svglite_2.1.1              
  [3] ps_1.7.5                    Rsamtools_2.14.0           
  [5] foreach_1.5.2               rprojroot_2.0.3            
  [7] crayon_1.5.2                MASS_7.3-60                
  [9] nlme_3.1-162                backports_1.4.1            
 [11] impute_1.72.3               rlang_1.1.1                
 [13] XVector_0.38.0              HDO.db_0.99.1              
 [15] irlba_2.3.5.1               callr_3.7.3                
 [17] filelock_1.0.2              rjson_0.2.21               
 [19] bit64_4.0.5                 glue_1.6.2                 
 [21] mixsqp_0.3-48               rngtools_1.5.2             
 [23] vipor_0.4.5                 parallel_4.2.1             
 [25] processx_3.8.2              DEFormats_1.26.0           
 [27] tidyselect_1.2.0            XML_3.99-0.14              
 [29] GenomicAlignments_1.34.1    xtable_1.8-4               
 [31] magrittr_2.0.3              evaluate_0.21              
 [33] bibtex_0.5.1                cli_3.6.1                  
 [35] zlibbioc_1.44.0             doRNG_1.8.6                
 [37] rstudioapi_0.14             whisker_0.4.1              
 [39] bslib_0.5.0                 rpart_4.1.19               
 [41] derfinderHelper_1.32.0      fastmatch_1.1-3            
 [43] BiocStyle_2.26.0            treeio_1.22.0              
 [45] xfun_0.39                   clue_0.3-64                
 [47] gson_0.1.0                  cluster_2.1.4              
 [49] caTools_1.18.2              tidygraph_1.2.3            
 [51] KEGGREST_1.38.0             logging_0.10-108           
 [53] ape_5.7-1                   Biostrings_2.66.0          
 [55] png_0.1-8                   reshape_0.8.9              
 [57] withr_2.5.0                 bitops_1.0-7               
 [59] ggforce_0.4.1               RBGL_1.74.0                
 [61] plyr_1.8.8                  coda_0.19-4                
 [63] bumphunter_1.40.0           pillar_1.9.0               
 [65] GlobalOptions_0.1.2         cachem_1.0.8               
 [67] fs_1.6.2                    GetoptLong_1.0.5           
 [69] graphite_1.44.0             vctrs_0.6.3                
 [71] generics_0.1.3              tools_4.2.1                
 [73] foreign_0.8-84              beeswarm_0.4.0             
 [75] munsell_0.5.0               tweenr_2.0.2               
 [77] fgsea_1.24.0                DelayedArray_0.24.0        
 [79] fastmap_1.1.1               compiler_4.2.1             
 [81] abind_1.4-5                 httpuv_1.6.11              
 [83] rtracklayer_1.58.0          plotly_4.10.2              
 [85] GenomeInfoDbData_1.2.9      lattice_0.21-8             
 [87] utf8_1.2.3                  later_1.3.1                
 [89] BiocFileCache_2.6.1         jsonlite_1.8.7             
 [91] GGally_2.1.2                scales_1.2.1               
 [93] tidytree_0.4.2              carData_3.0-5              
 [95] lazyeval_0.2.2              promises_1.2.0.1           
 [97] car_3.1-2                   doParallel_1.0.17          
 [99] checkmate_2.2.0             rmarkdown_2.23             
[101] cowplot_1.1.1               webshot_0.5.5              
[103] downloader_0.4              BSgenome_1.66.3            
[105] igraph_1.5.0                survival_3.5-5             
[107] numDeriv_2016.8-1.1         yaml_2.3.7                 
[109] systemfonts_1.0.4           ashr_2.2-54                
[111] SQUAREM_2021.1              htmltools_0.5.5            
[113] memoise_2.0.1               VariantAnnotation_1.44.1   
[115] BiocIO_1.8.0                locfit_1.5-9.8             
[117] graphlayouts_1.0.0          digest_0.6.32              
[119] rappdirs_0.3.3              knitrBootstrap_1.0.2       
[121] emdbook_1.3.13              RSQLite_2.3.1              
[123] yulab.utils_0.0.6           derfinder_1.32.0           
[125] blob_1.2.4                  preprocessCore_1.60.2      
[127] labeling_0.4.2              splines_4.2.1              
[129] Formula_1.2-5               RCurl_1.98-1.12            
[131] broom_1.0.5                 hms_1.1.3                  
[133] ConsensusClusterPlus_1.62.0 colorspace_2.1-0           
[135] base64enc_0.1-3             mnormt_2.1.1               
[137] BiocManager_1.30.21         shape_1.4.6                
[139] aplot_0.1.10                nnet_7.3-19                
[141] sass_0.4.6                  Rcpp_1.0.10                
[143] circlize_0.4.15             mvtnorm_1.2-2              
[145] fansi_1.0.4                 tzdb_0.4.0                 
[147] truncnorm_1.0-9             R6_2.5.1                   
[149] grid_4.2.1                  lifecycle_1.0.3            
[151] curl_5.0.1                  ggsignif_0.6.4             
[153] jquerylib_0.1.4             Matrix_1.5-4.1             
[155] qvalue_2.30.0               org.Hs.eg.db_3.16.0        
[157] iterators_1.0.14            RefManageR_1.4.0           
[159] htmlwidgets_1.6.2           markdown_1.7               
[161] polyclip_1.10-4             shadowtext_0.1.2           
[163] timechange_0.2.0            gridGraphics_0.5-1         
[165] reactome.db_1.82.0          ComplexHeatmap_2.14.0      
[167] rvest_1.0.3                 htmlTable_2.4.1            
[169] patchwork_1.1.2             bdsmatrix_1.3-6            
[171] invgamma_1.1                codetools_0.2-19           
[173] gtools_3.9.4                getPass_0.2-2              
[175] prettyunits_1.1.1           psych_2.3.6                
[177] dbplyr_2.3.2                gtable_0.3.3               
[179] DBI_1.1.3                   git2r_0.32.0               
[181] highr_0.10                  ggfun_0.1.1                
[183] httr_1.4.6                  KernSmooth_2.23-21         
[185] stringi_1.7.12              progress_1.2.2             
[187] reshape2_1.4.4              farver_2.1.1               
[189] annotate_1.76.0             Rgraphviz_2.42.0           
[191] ggtree_3.6.2                xml2_1.3.4                 
[193] bbmle_1.0.25                restfulr_0.0.15            
[195] geneplotter_1.76.0          ggplotify_0.1.1            
[197] bit_4.0.5                   scatterpie_0.2.1           
[199] ggraph_2.1.0                pkgconfig_2.0.3            
[201] rstatix_0.7.2               GenomicFiles_1.34.0        </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
