---
title: "Microglial METH IVSA DeSeq2 RNA-Sequencing Analysis -- Samples Removed -- Final Product"
author: "Alexander Margetts"
date: "`r Sys.Date()`"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
  code_folding: hide
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)

library(gprofiler2)
library(DESeq2)
library(ggplot2)
library(clusterProfiler)
library(biomaRt)
library(ReactomePA)
library(DOSE)
options(connectionObserver = NULL)
library(org.Mm.eg.db)
library(tximport)
library(ggsci)
library(pheatmap)
library(genefilter)
library(RColorBrewer)
library(GO.db)
library(topGO)
library(dplyr)
library(gage)
library(pathview)
library(WGCNA)
library(tidyr)
library(enrichplot)
library(ggnewscale)
library(SPIA)
library(ggpubr)
library(apeglm)
library(GOSemSim)
library(edgeR)
library(tweeDEseqCountData)
library(statmod)
library(org.Mm.eg.db)
library(tidyverse)
library(pheatmap)
library(gplots)
library(Mus.musculus)
library(Glimma)
library(RColorBrewer)
library(clusterProfiler)
library(ReactomePA)
library(enrichplot)
library(ggnewscale)
library(knitr)
library(tables)
library(vidger)
library(kableExtra)
library(regionReport)
library(dplyr)
library(tidyr)
library(ggplot2)
library(ggpubr)
library(dplyr)
library(conflicted)
library(ggplot2)
library(ggrepel)
library(ggplot2)
library(ggrepel)
library(RColorBrewer)
library(pheatmap)
library(DEGreport)
library(data.table)
library(readr)
library(PoiClaClu)
library(ggbeeswarm)
library(viridis)
conflicts_prefer(dplyr::setdiff)
library("BiocParallel")
register(MulticoreParam())
```

# Methamphetamine IVSA


Male C57BL/6J mice underwent methamphetamine intravenous self-administration (METH IVSA). The striatum was dissected, and microglia were isolated immediately following the last maintenance session (Maintenance and Saline) or following 21 days of forced home-cage abstinence (Abstinence). The samples were then processed for RNA-sequencing. The subsequent analysis is the result of this RNA-sequencing experiment.

# DESeq2 Analysis

## Setup

Building DESeq2 input files from gene count matrices output by StringTie following removal of samples that are outliers.

```{r DESeq2 Build -- Final, message=FALSE, warning=FALSE, include=FALSE}
rm(list = ls(all.names = TRUE))
gc()

## Run DEeq2 from Matrix output using Stringtie to compare all groups

colData <- read.csv("/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/MethamphetamineIVSA_RNASequencing/data/sample_info_final.csv", sep = ",", row.names = 1)
colData
countData <-as.matrix(read.csv("/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/MethamphetamineIVSA_RNASequencing/data/MIVSA_gene_count_matrix.csv", row.names = "gene_id"))


colnames(countData)
rownames(colData)
all(rownames(colData) %in% colnames(countData))
countData <- countData[, rownames(colData)]
all(rownames(colData) == colnames(countData))
```


Running DeSeq2 analysis with factor levels of Saline and Maintenance

```{r DESeq2 Analysis, echo=FALSE, message=FALSE, warning=FALSE}
dds <- DESeqDataSetFromMatrix(countData = countData, colData = colData, design = ~condition)
dds2 <- DESeqDataSetFromMatrix(countData = countData, colData = colData, design = ~condition)

## Relevel Condition to Analyze Each group against Saline
dds$condition <- relevel(dds$condition, ref = "Saline")
dds <- DESeq(dds)
## Relevel Condition to Analyze Each group against Maintenance
dds2$condition <- relevel(dds2$condition, ref = "Maintenance")
dds2 <- DESeq(dds2)

plotDispEsts(dds)
```
# Plot Estimates and Gather Results

## Gather Results {.tabset}

These results include overview for each result file and dispersion estimates based on count values.


### Maintenance vs Saline
```{r Maintenance vs Saline Results, echo=FALSE, results='asis'}
## Look at Results
res <- results(dds, name = "condition_Maintenance_vs_Saline")

kable(head(res, 20), caption = "Maintenance vs Saline Microglia Results Overview", format = "html", align = "l", booktabs = TRUE) %>%
  kable_styling(html_font = "Arial", font_size = 10, bootstrap_options = c("striped", "hover", "condensed", "responsive")) %>%
  scroll_box(width = "1000px", height = "500px")

summary(res)
```

### Abstinence vs Saline

```{r Abstinence vs Saline Results, echo=FALSE, results='asis'}
res2 <- results(dds, name = "condition_Abstinence_vs_Saline")
kable(head(res2, 20), caption = "Abstinence vs Saline Microglia Results Overview", format = "html", align = "l", booktabs = TRUE) %>%
  kable_styling(html_font = "Arial", font_size = 10, bootstrap_options = c("striped", "hover", "condensed", "responsive")) %>%
  scroll_box(width = "1000px", height = "500px")

summary(res2)
```

### Abstinence vs Maintenance

```{r Abstinence vs Maintenance Results, echo=FALSE, results='asis'}
res3 <- results(dds2, name = "condition_Abstinence_vs_Maintenance")
kable(head(res3, 20), caption = "Abstinence vs Maintenance Microglia Results Overview", format = "html", align = "l", booktabs = TRUE) %>%
  kable_styling(html_font = "Arial", font_size = 10, bootstrap_options = c("striped", "hover", "condensed", "responsive")) %>%
  scroll_box(width = "1000px", height = "500px")

summary(res3)
```

## Annotate Results Files and Pull Significant DEGs {.tabset}

This subsection is for annotation the results files to include all necessary information for downstream analyses.

```{r Annotate DDS Matrices, message=FALSE, warning=FALSE, include=FALSE, results='asis'}
## Make new rownames file and split based on pipe from dds file

deseqrownames <- data.frame(rownames(dds))
newrownames <- deseqrownames %>% separate(rownames.dds., c("id", "symbol"), sep = "([|])")
deseqrownames2 <- data.frame(rownames(dds2))
newrownames2 <- deseqrownames2 %>% separate(rownames.dds2., c("id", "symbol"), sep = "([|])")


# Check to make sure they split correctly and do not overlap

all(newrownames$id == newrownames$symbol)
all(newrownames2$id == newrownames2$symbol)
sum(is.na(newrownames$symbol))
sum(duplicated(newrownames$symbol))
sum(is.na(newrownames2$symbol))
sum(duplicated(newrownames2$symbol))
mcols(dds) <- cbind(mcols(dds), newrownames)
mcols(dds2) <- cbind(mcols(dds2), newrownames2)

# Add gene symbol to results

res$symbol <- mcols(dds)$symbol

res2$symbol <- mcols(dds)$symbol

res3$symbol <- mcols(dds2)$symbol

## Add Annotations

## Add Description

res$description <- mapIds(
  x = org.Mm.eg.db,
  keys = res$symbol,
  column = "GENENAME",
  keytype = "SYMBOL",
  multiVals = "first"
)

res2$description <- mapIds(
  x = org.Mm.eg.db,
  keys = res2$symbol,
  column = "GENENAME",
  keytype = "SYMBOL",
  multiVals = "first"
)

res3$description <- mapIds(
  x = org.Mm.eg.db,
  keys = res3$symbol,
  column = "GENENAME",
  keytype = "SYMBOL",
  multiVals = "first"
)



# Add ENTREZ ID
res$entrez <- mapIds(
  x = org.Mm.eg.db,
  keys = res$symbol,
  column = "ENTREZID",
  keytype = "SYMBOL",
  multiVals = "first"
)

res2$entrez <- mapIds(
  x = org.Mm.eg.db,
  keys = res2$symbol,
  column = "ENTREZID",
  keytype = "SYMBOL",
  multiVals = "first"
)

res3$entrez <- mapIds(
  x = org.Mm.eg.db,
  keys = res3$symbol,
  column = "ENTREZID",
  keytype = "SYMBOL",
  multiVals = "first"
)


# Add ENSEMBL ID
res$ensembl <- mapIds(
  x = org.Mm.eg.db,
  keys = res$entrez,
  column = "ENSEMBL",
  keytype = "ENTREZID",
  multiVals = "first"
)

res2$ensembl <- mapIds(
  x = org.Mm.eg.db,
  keys = res2$entrez,
  column = "ENSEMBL",
  keytype = "ENTREZID",
  multiVals = "first"
)

res3$ensembl <- mapIds(
  x = org.Mm.eg.db,
  keys = res3$entrez,
  column = "ENSEMBL",
  keytype = "ENTREZID",
  multiVals = "first"
)
```

After annotation I subset the results to only view the significantly differentially expressed genes with an adjusted p-value \> 0.05.

```{r Subset Significant Results, echo=TRUE, message=FALSE, warning=FALSE, results='asis'}
## Subset for significant genes only

results_sig <- subset(res, padj < 0.05)
results_sig2 <- subset(res2, padj < 0.05)
results_sig3 <- subset(res3, padj < 0.05)
```

### Maintenance vs Saline

```{r Maintenance vs Saline Significant Results, echo=FALSE, results='asis'}
kable((head(results_sig, 20)), caption = "Significant DEGs Maintenance vs Saline", format = "html", align = "l", booktabs = TRUE) %>%
  kable_styling(html_font = "Arial", font_size = 10, bootstrap_options = c("striped", "hover", "condensed", "responsive")) %>%
  scroll_box(width = "1000px", height = "500px")
```

### Abstinence vs Saline

```{r Abstinence vs Saline Significant Results, echo=FALSE, results='asis'}
kable((head(results_sig2, 20)), caption = "Significant DEGs Abstinence vs Saline", format = "html", align = "l", booktabs = TRUE) %>%
  kable_styling(html_font = "Arial", font_size = 10, bootstrap_options = c("striped", "hover", "condensed", "responsive")) %>%
  scroll_box(width = "1000px", height = "500px")
```

### Abstinence vs Maintenance

```{r Abstinence vs Maintenance Significant Results, echo=FALSE, results='asis'}
kable((head(results_sig3, 20)), caption = "Significant Abstinence vs Maintenance", format = "html", align = "l", booktabs = TRUE) %>%
  kable_styling(html_font = "Arial", font_size = 10, bootstrap_options = c("striped", "hover", "condensed", "responsive")) %>%
  scroll_box(width = "1000px", height = "500px")
```

```{r Write Significant Results to CSV, eval=FALSE, include=FALSE}
write_csv((as.data.frame(results_sig)), "results/Sig_Res_Maintenace_vs_Saline_Final.csv")
write_csv((as.data.frame(results_sig2)), "results/Sig_Res_Abstinence_vs_Saline_Final.csv")
write_csv((as.data.frame(results_sig3)), "results/Sig_Res_Abstinence_vs_Maintenance_Final.csv")
```

## Visualize Shrinkage Estimations

Following subsetting, I want to see how the data looks based on effect size so I have run a shrinkage model using the "apeglm" method which normalizes counts based on effect size without major changes to data structure. I have included non-shrunk models as well for comparison.

```{r Shrinkage Estimation, echo=FALSE, message=FALSE, warning=FALSE, paged.print=TRUE}
## Log Fold Change Shrinkage Estimation

resLFC <- lfcShrink(dds, coef = "condition_Maintenance_vs_Saline", type = "ashr")
resLFC2 <- lfcShrink(dds, coef = "condition_Abstinence_vs_Saline", type = "ashr")
resLFC3 <- lfcShrink(dds2, coef = "condition_Abstinence_vs_Maintenance", type = "ashr")
```

### Non-Shrunken L2FC {.tabset}

#### Maintenance vs Saline

```{r Non-Shrunken Estimate Plots, echo=FALSE, message=FALSE, warning=FALSE}
DESeq2::plotMA(res, ylim = c(-5, 5), alpha = 0.05, main = "Non-Shrunken Maintenance vs Saline L2FC")
```

#### Abstinence vs Saline

```{r Non-Shrunken Estimate Plots2, echo=FALSE, message=FALSE, warning=FALSE}
DESeq2::plotMA(res2, ylim = c(-5, 5), alpha = 0.05, main = "Non-Shrunken Abstinence vs Saline L2FC")
```

#### Abstinence vs Maintenance

```{r Non-Shrunken Estimate Plots3, echo=FALSE, message=FALSE, warning=FALSE}
DESeq2::plotMA(res3, ylim = c(-5, 5), alpha = 0.05, main = "Non-Shrunken Abstinence vs Maintenance L2FC")
```

### Shrunken L2FC {.tabset}

#### Maintenance vs Saline

```{r Shrunken Estimate Plots, echo=FALSE, message=FALSE, warning=FALSE}
DESeq2::plotMA(resLFC, alpha = 0.05, main = "Shrunken LF2C Maintenance vs Saline")
```

#### Abstinence vs Saline

```{r Shrunken Estimate Plots2, echo=FALSE, message=FALSE, warning=FALSE}
DESeq2::plotMA(resLFC2, alpha = 0.05, main = "Shrunken LF2C Abstinence vs Saline")
```

#### Abstinence vs Maintenance

```{r Shrunken Estimate Plots3, echo=FALSE, message=FALSE, warning=FALSE}
DESeq2::plotMA(resLFC3, alpha = 0.05, main = "Shrunken LF2C Abstinence vs Maintenance")
```


# Looking at Differentially Expressed Genes

We can generate a matrix to identify where our differentially expressed genes lie between each group based on an alpha value of 0.05. 

```{r Matrix of Differentially Expressed genes across groups, echo=TRUE, message=FALSE, warning=FALSE}
## Matrix Visualizaattion
x <- vsDEGMatrix(
  data = dds, padj = 0.05, d.factor = "condition",
  type = "deseq", title = TRUE, legend = TRUE, grid = TRUE
)
```

```{r Writing matrix to CSV File, eval=FALSE, message=FALSE, warning=FALSE, include=FALSE}
## Write table to csv files
df_scatter <- x$data

write.table(
  x = as.data.frame(df_scatter),
  file = "results/DEGs_Flower_Matrix_MIVSA_Experiment_05.txt",
  sep = "\t",
  quote = F,
  col.names = NA
)
```

```{r Save All Counts and Results Files, eval=FALSE, message=FALSE, warning=FALSE, include=FALSE}
## Write table to csv files

write.table(
  x = as.data.frame(counts(dds), normalized = T),
  file = "results/normcounts/MIVSA_experiment_normalized_counts_Final.txt",
  sep = "\t",
  quote = F,
  col.names = NA
)


# Write significant normalized gene counts to a .txt file
write.table(
  x = counts(dds[rownames(results_sig)], normalized = T),
  file = "results/normcounts/Sig_Norm_Counts_Maintenance_vs_Saline.txt",
  sep = "\t",
  quote = F,
  col.names = NA
)

write.table(
  x = counts(dds[rownames(results_sig2)], normalized = T),
  file = "results/normcounts/Sig_Norm_Counts_Abstinence_vs_Saline.txt",
  sep = "\t",
  quote = F,
  col.names = NA
)

write.table(
  x = counts(dds[rownames(results_sig3)], normalized = T),
  file = "results/normcounts/Sig_Norm_Counts_Abstinence_vs_Maintenance.txt",
  sep = "\t",
  quote = F,
  col.names = NA
)
```

# Data Visualization

## Normalized Counts and Plots of Genes of Interest {.tabset}

```{r Obtain Normalized Counts, eval=FALSE, message=FALSE, warning=FALSE, include=FALSE}
theme_set(theme_pubr())

normalized_counts <- counts(dds, normalized = TRUE)
head(normalized_counts)

## dds
normrownames <- data.frame(rownames(normalized_counts))
newrownames <- normrownames %>% separate(rownames.normalized_counts., c("id", "symbol"), sep = "([|])")

all(newrownames$id == newrownames$symbol)

sum(is.na(newrownames$symbol))
sum(duplicated(newrownames$symbol))


## dds
mcols(dds) <- cbind(mcols(dds), newrownames)

# Add gene symbol to results

normalized_counts2 <- as.data.frame(normalized_counts)

normalized_counts2$symbol <- newrownames$symbol


dds@rowRanges@elementMetadata@listData[["Description"]] <- mapIds(
  x = org.Mm.eg.db,
  keys = dds@rowRanges@elementMetadata@listData[["id"]],
  column = "GENENAME",
  keytype = "SYMBOL",
  multiVals = "first"
)
# Add ENTREZ ID
dds@rowRanges@elementMetadata@listData[["Entrez"]] <- mapIds(
  x = org.Mm.eg.db,
  keys = dds@rowRanges@elementMetadata@listData[["id"]],
  column = "ENTREZID",
  keytype = "SYMBOL",
  multiVals = "first"
)

# Add ENSEMBL ID
dds@rowRanges@elementMetadata@listData[["ENSMBL"]] <- mapIds(
  x = org.Mm.eg.db,
  keys = dds@rowRanges@elementMetadata@listData[["id"]],
  column = "ENSEMBL",
  keytype = "SYMBOL",
  multiVals = "first"
)

normalized_counts2
normalized_counts3 <- as.data.frame(normalized_counts2)
write.table(
  x = as.data.frame(counts(dds), normalized = T),
  file = "results/normcounts/MIVSA_experiment_normalized_counts_annotated.txt",
  sep = "\t",
  quote = F,
  col.names = NA
)

write.table(
  x = as.data.frame(dds@rowRanges@elementMetadata@listData[["Entrez"]]),
  file = "results/normcounts/MIVSA_experiment_normalized_counts_annotated_Entrez.txt",
  sep = "\t",
  quote = F,
  col.names = NA
)
```

```{r Plot Counts -- Selected 1, echo=TRUE, message=FALSE, warning=FALSE}
level_order <- c("Saline", "Maintenance", "Abstinence") # this vector might be useful for other plots/analyses

normalized_counts <- counts(dds, normalized = TRUE)
```

### Hspd1

```{r Selected Genes, echo=TRUE, message=FALSE, warning=FALSE}
####################################### HSPA8

d2 <- plotCounts(dds,
  gene = "Hspd1|Hspd1", intgroup = "condition",
  returnData = TRUE
)

a <- ggplot(d2, aes(x = factor(condition, level = level_order), y = count, color = condition)) +
  geom_point(position = position_jitter(w = 0.1, h = 0)) +
  theme_bw() +
  ggtitle(
    label = "Hspd1 Expression",
    subtitle = "Normalized Gene Expression"
  )

a
```

### Cirbp 

```{r Selected Genes2, echo=TRUE, message=FALSE, warning=FALSE}
####################################### HSP90ab1


d2 <- plotCounts(dds,
  gene = "Cirbp|Cirbp", intgroup = "condition",
  returnData = TRUE
)

a <- ggplot(d2, aes(x = factor(condition, level = level_order), y = count, color = condition)) +
  geom_point(position = position_jitter(w = 0.1, h = 0)) +
  theme_bw() +
  ggtitle(
    label = "Cirbp Expression",
    subtitle = "Normalized Gene Expression"
  )

a
```

### Fus

```{r Selected Genes3, echo=TRUE, message=FALSE, warning=FALSE}
####################################### Dnajb1


d2 <- plotCounts(dds,
  gene = "Fus|Fus", intgroup = "condition",
  returnData = TRUE
)

a <- ggplot(d2, aes(x = factor(condition, level = level_order), y = count, color = condition)) +
  geom_point(position = position_jitter(w = 0.1, h = 0)) +
  theme_bw() +
  ggtitle(
    label = "Fus Expression",
    subtitle = "Normalized Gene Expression"
  )

a
```

### Cryab

```{r Selected Genes4, echo=TRUE, message=FALSE, warning=FALSE}
####################################### Dnajb1


d2 <- plotCounts(dds,
  gene = "Cryab|Cryab", intgroup = "condition",
  returnData = TRUE
)

a <- ggplot(d2, aes(x = factor(condition, level = level_order), y = count, color = condition)) +
  geom_point(position = position_jitter(w = 0.1, h = 0)) +
  theme_bw() +
  ggtitle(
    label = "Cryab Expression",
    subtitle = "Normalized Gene Expression"
  )

a
```

### Nr1d1

```{r Selected Genes5, echo=TRUE, message=FALSE, warning=FALSE}
####################################### Dnajb1


d2 <- plotCounts(dds,
  gene = "Nr1d1|Nr1d1", intgroup = "condition",
  returnData = TRUE
)

a <- ggplot(d2, aes(x = factor(condition, level = level_order), y = count, color = condition)) +
  geom_point(position = position_jitter(w = 0.1, h = 0)) +
  theme_bw() +
  ggtitle(
    label = "Nr1d1 Expression",
    subtitle = "Normalized Gene Expression"
  )

a
```


### Syt11

```{r Selected Genes6, echo=TRUE, message=FALSE, warning=FALSE}
####################################### Dnajb1


d2 <- plotCounts(dds,
  gene = "Syt11|Syt11", intgroup = "condition",
  returnData = TRUE
)

a <- ggplot(d2, aes(x = factor(condition, level = level_order), y = count, color = condition)) +
  geom_point(position = position_jitter(w = 0.1, h = 0)) +
  theme_bw() +
  ggtitle(
    label = "Syt11 Expression",
    subtitle = "Normalized Gene Expression"
  )

a
```

```{r plotcounts function, message=FALSE, warning=FALSE, include=FALSE}
#' Plot normalized counts for a gene
#'
#' Plot for normalized counts of a single gene, with jittered points superimposed
#' on the boxplot
#'
#' Note: this function relies on the \code{\link{plotCounts}} function of DESeq2,
#' therefore pseudocounts of 0.5 are added to each point
#'
#' @param dds A \code{\link{DESeqDataSet}} object.
#' @param gene A character, specifying the name of the gene to plot
#' @param intgroup Interesting groups: a character vector of
#' names in \code{colData(dds)} to use for grouping
#' @param annotation_obj A \code{data.frame} object, with \code{row.names} as gene
#' identifiers (e.g. ENSEMBL ids) and a column, \code{gene_name}, containing
#' e.g. HGNC-based gene symbols. Optional.
#' @param transform Logical value, corresponding whether to have log scale y-axis
#' or not. Defaults to TRUE.
#' @param labels_repel Logical value. Whether to use \code{ggrepel}'s functions to
#' place labels; defaults to TRUE.
#'
#' @return An object created by \code{ggplot}
#' @export
#'
#' @examples
#' library(airway)
#' data(airway)
#' airway
#' dds_airway <- DESeq2::DESeqDataSetFromMatrix(assay(airway),
#'   colData = colData(airway),
#'   design = ~ cell + dex
#' )
#' ggplotCounts(dds_airway,
#'   gene = "ENSG00000103196", # CRISPLD2 in the original publication
#'   intgroup = "dex"
#' )
ggplotCounts <- function(dds, gene, intgroup = "condition", annotation_obj = NULL,
                         transform = TRUE, labels_repel = TRUE) {
  df <- plotCounts(dds, gene, intgroup, returnData = TRUE)
  df$sampleID <- rownames(df)

  if (!is.null(annotation_obj)) {
    genesymbol <- annotation_obj$gene_name[match(gene, annotation_obj$gene_id)]
  } else {
    genesymbol <- ""
  }

  jittered_df <- df
  # jittered_df$conditionj <- jitter(as.numeric(factor(jittered_df$condition)))
  jittered_df$countj <- jitter(jittered_df$count)

  onlyfactors <- df[, match(intgroup, colnames(df))]
  df$plotby <- interaction(onlyfactors)

  # base_breaks <- function(n = 10){
  #   function(x) {
  #     axisTicks(log10(range(x, na.rm = TRUE)), log = TRUE, nint = n)
  #   }
  # }

  p <-
    ggplot(df, aes_string(x = "plotby", y = "count", col = "plotby")) +
    geom_boxplot(outlier.shape = NA) +
    # geom_text(data = jittered_df,aes(x=conditionj,y=countj,label=sampleID)) +
    scale_x_discrete(name = "") +
    geom_jitter(aes_string(x = "plotby", y = "count"),
      position = position_jitter(width = 0.1)
    ) +
    scale_color_discrete(name = "Experimental\nconditions")

  if (labels_repel) {
    p <- p + ggrepel::geom_text_repel(aes_string(label = "sampleID"))
  } else {
    p <- p + geom_text(aes_string(label = "sampleID"), hjust = -.1, vjust = 0)
  }

  if (transform) {
    p <- p + scale_y_log10(name = "Normalized counts (log10 scale)")
  } else {
    p <- p + scale_y_continuous(name = "Normalized counts")
  }

  # scale_y_log10(name="Normalized counts - log10 scale") +
  # coord_cartesian(ylim = c())# ,limits=c(0.1,NA)) +
  p <- p + theme_bw()

  if (!is.null(annotation_obj)) {
    p <- p + labs(title = paste0("Normalized counts for ", genesymbol, " - ", gene))
  } else {
    p <- p + labs(title = paste0("Normalized counts for ", gene))
  }

  p
}
```

```{r eval=FALSE, include=FALSE}
library(reshape)
library(dplyr)
conflicts_prefer(reshape::melt)
# obtaining normalized count read
dys_reg <- normalized_counts

# melting dataset for visualization
melted_norm_counts <- data.frame(melt(dys_reg))
colnames(melted_norm_counts) <- c("gene", "samplename", "normalized_counts")

melted_norm_counts <- melted_norm_counts %>%
  group_by(gene) %>%
  mutate(group = case_when(
    (samplename == "SAL01") ~ "Saline",
    (samplename == "SAL02") ~ "Saline",
    (samplename == "SAL03") ~ "Saline",
    (samplename == "SAL04") ~ "Saline",
    (samplename == "SAL05") ~ "Saline",
    (samplename == "SAL06") ~ "Saline",
    (samplename == "SAL07") ~ "Saline",
    (samplename == "SAL08") ~ "Saline",
    (samplename == "MN01") ~ "Maintenance",
    (samplename == "MN02") ~ "Maintenance",
    (samplename == "MN03") ~ "Maintenance",
    (samplename == "MN04") ~ "Maintenance",
    (samplename == "MN05") ~ "Maintenance",
    (samplename == "MN06") ~ "Maintenance",
    (samplename == "MN07") ~ "Maintenance",
    (samplename == "MN08") ~ "Maintenance",
    (samplename == "ABS01") ~ "Abstinence",
    (samplename == "ABS02") ~ "Abstinence",
    (samplename == "ABS03") ~ "Abstinence",
    (samplename == "ABS04") ~ "Abstinence",
    (samplename == "ABS05") ~ "Abstinence",
    (samplename == "ABS06") ~ "Abstinence",
    (samplename == "ABS07") ~ "Abstinence",
    (samplename == "ABS08") ~ "Abstinence",
  ))

# write.table to import in python
# write.table(melted_norm_counts, file = "results/melted_norm_counts.csv", sep = ",", row.names = F)


ggplotCounts(
  dds = dds,
  gene = "Hdac3|Hdac3", # CRISPLD2 in the original publication
  intgroup = "condition",
  labels_repel = FALSE
)

dat <- as.data.frame(normalized_counts)
dat$gene <- rownames(normalized_counts)


# ggplot(normalized_counts, aes(x = as.factor(gene), y = normalized_counts)) +
# geom_boxplot(aes(fill = group), position = position_dodge(0.9)) +
# scale_fill_manual(values = c("#09E359", "#E31009")) +
# theme_bw() +
# theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust=1))


# ggplot(melted_norm_counts, aes(x = as.factor(gene), y = normalized_counts)) +
# geom_boxplot(aes(fill = group), position = position_dodge(0.9)) +
# scale_fill_manual(values = c("#09E359", "#E31009")) +
# theme_bw() +
# theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust=1))
```

## Visualize Results: PCA

For visualization I am converting all analyses to logarithmic. Following this I have completed a PCA for each analyses based on all genes together.

```{r Plot PCA, echo=FALSE, message=FALSE, warning=FALSE, paged.print=TRUE}
# Convert all samples to rlog for data visualization

ddsMat_rlog <- DESeq2::rlog(dds, blind = FALSE)

# Plot PCA by column variable (dds)

# pdf(file = "plots/PCA_MIVSA_Final.pdf", width = 12, height = 12);

# pcaData2 <- plotPCA(ddsMat_rlog, intgroup = "condition", returnData = TRUE, ntop = 500)
# pcaData2$condition <- factor(pcaData2$condition, levels = c("Saline", "Maintenance", "Abstinence"))
# percentVar2 <- round(100 * attr(pcaData2, "percentVar"))
# ggplot(pcaData2, aes(PC1, PC2, color=condition, label = colnames(dds))) +
#  theme_bw() + # remove default ggplot2 theme
#  geom_point(size = 2) + # Increase point size
# scale_y_continuous(limits = c(-60, 60)) +# change limits to fix figure dimensions
# scale_x_continuous(limits = c(-80, 50)) +# change limits to fix figure dimensions
#  xlab(paste0("PC2: ",percentVar2[1],"% variance")) +
#  ylab(paste0("PC3: ",percentVar2[2],"% variance")) +
#  ggrepel::geom_label_repel(box.padding = 1, max.overlaps = Inf) +
#  coord_fixed() +
#  ggtitle(label = "Principal Component Analysis (PCA)",
#          subtitle = "All Genes (rlog transformed)")

# dev.off()


pcaData2 <- plotPCA(ddsMat_rlog, intgroup = "condition", returnData = TRUE, ntop = 500)
pcaData2$condition <- factor(pcaData2$condition, levels = c("Saline", "Maintenance", "Abstinence"))
percentVar2 <- round(100 * attr(pcaData2, "percentVar"))
ggplot(pcaData2, aes(PC1, PC2, color = condition, label = colnames(dds))) +
  theme_bw() + # remove default ggplot2 theme
  geom_point(size = 2) + # Increase point size
  # scale_y_continuous(limits = c(-60, 60)) +# change limits to fix figure dimensions
  # scale_x_continuous(limits = c(-80, 50)) +# change limits to fix figure dimensions
  xlab(paste0("PC2: ", percentVar2[1], "% variance")) +
  ylab(paste0("PC3: ", percentVar2[2], "% variance")) +
  ggrepel::geom_label_repel(box.padding = 1, max.overlaps = Inf) +
  coord_fixed() +
  ggtitle(
    label = "Principal Component Analysis (PCA)",
    subtitle = "All Genes (rlog transformed)"
  )
```

## Visualize Results: Heatmap (Data Generation)

This heatmap is generated by taking the row means of the normalized counts from the primary analysis (dds, All vs Control). The top genes were placed in decreasing order and selected for analysis after logarithmic transformation.The gene name column is then split by the "\|" delimeter in order to obtain a single gene name for each count. The heatmap was generated by scaling to genes for the z-scores following row/column analysis using a complete pearson correlation. The heatmap is clustered by row but not by column to maintain correct order of samples.The legend indicates logfold change values for each gene. Representative heatmaps are included for a number of selected genes.

```{r PheatMap, eval=FALSE, message=FALSE, warning=FALSE, include=FALSE}
conflicts_prefer(pheatmap::pheatmap)

samples <- as.tibble(read.csv("/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/final_samples_removed/sample_info_final.csv", sep = ","))

annot_col <- samples %>%
  column_to_rownames("samplename") %>%
  as.data.frame()

ann_colors <- list(
  Group = c(Saline = "blue", Maintenance = "orange", Abstinence = "red")
)

select <- order(rowMeans(counts(dds, normalized = TRUE)),
  decreasing = TRUE
) # [1:75]

mat <- assay(ddsMat_rlog)[select, ]
matrownames <- data.frame(rownames(mat))
matrownamesnew <- matrownames %>% separate(rownames.mat., c("id", "symbol"), sep = "([|])")

# pheatmap(mat = mat,
#         color = colorRampPalette(rev(brewer.pal(8, "RdYlBu")))(255),
#         scale = "row",# Scale genes to Z-score (how many standard deviations),
#         annotation_col = annot_col,
#         annotation_colors = ann_colors,# Change the default colors of the annotations
#         fontsize = 8, # Make fonts smaller
#         cellwidth = 20, # Make the cells wider
#         show_colnames = T,
#         cluster_rows = T,
#         cluster_cols = F,
#         annotation_legend = T,
#         legend = T,
#         show_rownames = F,
#         clustering_distance_rows = 'correlation',
#         clustering_method = 'ward',
#         main = "Heatmap of All Genes")
#         filename = "plots/Heatmaps_AllGenes/Heatmap_of_all_genes_MIVSA.pdf")



## 500 Genes

select3 <- order(rowMeans(counts(dds, normalized = TRUE)),
  decreasing = TRUE
)[1:500]

mat3 <- assay(ddsMat_rlog)[select3, ]


matrownames3 <- data.frame(rownames(mat3))

# head(matrownames3)

matrownamesnew3 <- matrownames3 %>% separate(rownames.mat3., c("id", "symbol"), sep = "([|])")

# matrownamesnew3


pheatmap(
  mat = mat3,
  color = colorRampPalette(rev(brewer.pal(11, "RdYlBu")))(255),
  scale = "row", # Scale genes to Z-score (how many standard deviations),
  annotation_col = annot_col,
  annotation_colors = ann_colors, # Change the default colors of the annotations
  fontsize = 8, # Make fonts smaller
  cellwidth = 20, # Make the cells wider
  show_colnames = T,
  cluster_rows = T,
  cluster_cols = T,
  show_rownames = F,
  annotation_legend = T,
  legend = T,
  clustering_distance_rows = "correlation",
  clustering_method = "ward",
  main = "Heatmap of Top 500 Most Variable Genes"
)
# filename = "plots/Heatmaps_AllGenes/Heatmpap_of_500_genes_MIVSA.pdf")


## 250 Genes

select4 <- order(rowMeans(counts(dds, normalized = TRUE)),
  decreasing = TRUE
)[1:250]

mat4 <- assay(ddsMat_rlog)[select4, ]


matrownames4 <- data.frame(rownames(mat4))

# head(matrownames4)

matrownamesnew4 <- matrownames4 %>% separate(rownames.mat4., c("id", "symbol"), sep = "([|])")

# matrownamesnew4

pheatmap(
  mat = mat4,
  color = colorRampPalette(rev(brewer.pal(11, "RdYlBu")))(255),
  scale = "row", # Scale genes to Z-score (how many standard deviations),
  annotation_col = annot_col,
  annotation_colors = ann_colors, # Change the default colors of the annotations
  fontsize = 8, # Make fonts smaller
  cellwidth = 20, # Make the cells wider
  show_colnames = T,
  cluster_rows = T,
  cluster_cols = T,
  annotation_legend = T,
  legend = T,
  show_rownames = F,
  clustering_distance_rows = "correlation",
  clustering_method = "ward",
  main = "Heatmap of Top 250 Most Variable Genes"
)
# filename = "plots/Heatmaps_AllGenes/Heatmpap_of_250_genes_MIVSA.pdf")


## 100 Genes

select2 <- order(rowMeans(counts(dds, normalized = TRUE)),
  decreasing = TRUE
)[1:100]

mat2 <- assay(ddsMat_rlog)[select2, ]


matrownames2 <- data.frame(rownames(mat2))

# head(matrownames2)

matrownamesnew2 <- matrownames2 %>% separate(rownames.mat2., c("id", "symbol"), sep = "([|])")

# matrownamesnew2


pheatmap(
  mat = mat2,
  color = colorRampPalette(rev(brewer.pal(11, "RdYlBu")))(255),
  scale = "row", # Scale genes to Z-score (how many standard deviations),
  annotation_col = annot_col,
  annotation_colors = ann_colors, # Change the default colors of the annotations
  fontsize = 4, # Make fonts smaller
  cellwidth = 20, # Make the cells wider
  show_colnames = T,
  cluster_rows = T,
  cluster_cols = F,
  annotation_legend = T,
  legend = T,
  show_rownames = T,
  clustering_distance_rows = "correlation",
  clustering_method = "ward",
  main = "Heatmap of Top 100 Most Variable Genes",
  # filename = "plots/Heatmaps_AllGenes/Heatmpap_of_100_genes_MIVSA.pdf",
  labels_row = matrownamesnew2$symbol
)


## 55 Genes

select5 <- order(rowMeans(counts(dds, normalized = TRUE)),
  decreasing = TRUE
)[1:55]

mat5 <- assay(ddsMat_rlog)[select5, ]


matrownames5 <- data.frame(rownames(mat5))

# head(matrownames5)

matrownamesnew5 <- matrownames5 %>% separate(rownames.mat5., c("id", "symbol"), sep = "([|])")

# matrownamesnew5


pheatmap(
  mat = mat5,
  color = colorRampPalette(rev(brewer.pal(11, "RdYlBu")))(255),
  scale = "row", # Scale genes to Z-score (how many standard deviations)
  annotation_col = annot_col,
  annotation_colors = ann_colors, # Change the default colors of the annotations
  fontsize = 8, # Make fonts smaller
  cellwidth = 20, # Make the cells wider
  show_colnames = T,
  cluster_rows = T,
  cluster_cols = T,
  annotation_legend = T,
  legend = T,
  show_rownames = T,
  clustering_distance_rows = "correlation",
  clustering_method = "ward",
  main = "Heatmap of Top 55 Most Variable Genes",
  # filename = "plots/Heatmaps_AllGenes/Heatmpap_of_55_genes_MIVSA.pdf",
  labels_row = matrownamesnew5$symbol
)
```

## Visualize Results: Heatmap {.tabset}

```{r Filter Heatmaps by DEGs, fig.height=8, fig.width=8, message=FALSE, warning=FALSE}
ressig1 <- as.data.frame(results_sig)
ressig2 <- as.data.frame(results_sig2)
ressig3 <- as.data.frame(results_sig3)

ressig1$Comparison <- "Maintenance vs Saline"
ressig2$Comparison <- "Abstinence vs Saline"
ressig3$Comparison <- "Abstinence vs Maintenance"

ressig1_highlfc_pos <- ressig1[ressig1$log2FoldChange >= 1.3, ]
ressig1_highlfc_neg <- ressig1[ressig1$log2FoldChange <= -1.3, ]
ressig2_highlfc_pos <- ressig2[ressig2$log2FoldChange >= 1.3, ]
ressig2_highlfc_neg <- ressig2[ressig2$log2FoldChange <= -1.3, ]
ressig3_highlfc_pos <- ressig3[ressig3$log2FoldChange >= 1.3, ]
ressig3_highlfc_neg <- ressig3[ressig3$log2FoldChange <= -1.3, ]

temp_sigdegshm <- rbind(ressig1_highlfc_pos, ressig1_highlfc_neg)
temp_sigdegshm2 <- rbind(temp_sigdegshm, ressig2_highlfc_pos)
temp_sigdegshm3 <- rbind(temp_sigdegshm2, ressig2_highlfc_neg)
temp_sigdegshm4 <- rbind(temp_sigdegshm3, ressig3_highlfc_pos)
temp_sigdegshm5 <- rbind(temp_sigdegshm4, ressig3_highlfc_neg)

temp_sigdegshm6 <- temp_sigdegshm5[temp_sigdegshm5$lfcSE <= 1.50, ]

# sum(is.na(temp_sigdegshm6$symbol))
# sum(duplicated(temp_sigdegshm6$symbol))

dup_rows <- duplicated(temp_sigdegshm6$symbol)
data_dedup <- temp_sigdegshm6[!dup_rows, ]

rows <- rownames(data_dedup)

select <- order(rowMeans(counts(dds, normalized = TRUE)),
  decreasing = TRUE
)

mat <- assay(ddsMat_rlog)[select, ]

data_mod <- mat[rownames(mat) %in% rows, ]

matrownames_data_mod <- data.frame(rownames(data_mod))

matrownames_data_mod <- matrownames_data_mod %>% separate(rownames.data_mod., c("id", "symbol"), sep = "([|])")


samples <- as.tibble(read.csv("/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/final_samples_removed/sample_info_final.csv", sep = ","))

annot_col <- samples %>%
  column_to_rownames("samplename") %>%
  as.data.frame()

ann_colors <- list(
  condition = as.factor(c(Saline = "slateblue2", Maintenance = "limegreen", Abstinence = "violetred2")),
  Group = as.factor(c(Saline = "slateblue2", Maintenance = "limegreen", Abstinence = "violetred2"))
)
```

### Unsupervised Clustering

```{r Filter Heatmaps by DEGs2Unsupervised, fig.height=8, fig.width=8, message=FALSE, warning=FALSE}
pheatmap::pheatmap(
  mat = data_mod,
  color = colorRampPalette(rev(brewer.pal(11, "RdYlBu")))(155),
  scale = "row", # Scale genes to Z-score (how many standard deviations),
  annotation_col = annot_col,
  # annotation_colors = ann_colors,# Change the default colors of the annotations
  fontsize = 3, # Make fonts smaller
  cellwidth = 15, # Make the cells wider
  cellheight = 3, # Make the cells wider
  show_colnames = T,
  cluster_rows = T,
  cluster_cols = T,
  annotation_legend = T,
  legend = T,
  show_rownames = T,
  clustering_distance_rows = "correlation",
  clustering_method = "ward.D",
  main = "Heatmap of Significant DEGs -- Unsupervised Clustering",
  cluster_row_slices = TRUE,
  show_row_dend = FALSE,
  cutree_rows = 4,
  cutree_cols = 3,
  treeheight_row = 10,
  treeheight_col = 10,
  # kmeans_k = 4,
  # display_numbers = TRUE,
  # filename = "plots/MIVSA_Heatmap_DEGs_Final.pdf",
  legend_labels = c("Saline", "Maintenance", "Abstinence"),
  labels_row = matrownames_data_mod$symbol
)
```

### Supervised Clustering

```{r Filter Heatmaps by DEGs2, fig.height=8, fig.width=8, message=FALSE, warning=FALSE}
pheatmap::pheatmap(
  mat = data_mod,
  color = colorRampPalette(rev(brewer.pal(11, "RdYlBu")))(155),
  scale = "row", # Scale genes to Z-score (how many standard deviations),
  annotation_col = annot_col,
  # annotation_colors = ann_colors,# Change the default colors of the annotations
  fontsize = 8, # Make fonts smaller
  cellwidth = 15, # Make the cells wider
  cellheight = 3, # Make the cells wider
  show_colnames = F,
  cluster_rows = T,
  cluster_cols = F,
  annotation_legend = T,
  legend = T,
  show_rownames = F,
  clustering_distance_rows = "correlation",
  clustering_method = "ward.D",
  main = "Heatmap of Significant DEGs -- Supervised Clustering",
  cluster_row_slices = TRUE,
  show_row_dend = FALSE,
  cutree_rows = 4,
  cutree_cols = 3,
  gaps_col = c(5, 10),
  treeheight_row = 0,
  treeheight_col = 10,
  # kmeans_k = 4,
  # display_numbers = TRUE,
  # filename = "plots/MIVSA_Heatmap_DEGs_Final_Unclustered_Nolabs.pdf",
  legend_labels = c("Saline", "Maintenance", "Abstinence")
)
# labels_row = matrownames_data_mod$symbol)
```

### Supervised Clustering and Condensed

```{r Filter Heatmaps by DEGs3, fig.height=8, fig.width=8, message=FALSE, warning=FALSE}
pheatmap::pheatmap(
  mat = data_mod,
  color = colorRampPalette(rev(brewer.pal(11, "RdYlBu")))(155),
  scale = "row", # Scale genes to Z-score (how many standard deviations),
  annotation_col = annot_col,
  # annotation_colors = ann_colors,# Change the default colors of the annotations
  fontsize = 12, # Make fonts smaller
  # cellwidth = 15, # Make the cells wider
  # cellheight = 3, # Make the cells wider
  show_colnames = F,
  cluster_rows = T,
  cluster_cols = F,
  annotation_legend = T,
  legend = T,
  show_rownames = F,
  clustering_distance_rows = "correlation",
  clustering_method = "ward.D",
  main = "Heatmap of Significant DEGs -- Supervised and Condensed",
  legend_labels = c("Saline", "Maintenance", "Abstinence"),
  cluster_row_slices = TRUE,
  show_row_dend = FALSE,
  gaps_col = c(5, 10),
  cutree_rows = 4,
  cutree_cols = 3,
  treeheight_row = 15,
  treeheight_col = 10
)
# kmeans_k = 4,
# display_numbers = TRUE,
# filename = "plots/MIVSA_Heatmap_DEGs_Final_Unclustered_Nolabs_Condensed.pdf")
# labels_row = matrownames_data_mod$symbol)
```

## Volcano Plots

### Visualize Results: Volcano Plots (Non-Interactive)

Volcano plots are generated by gathering the FDR corrected p-values from each analysis. The adjusted p-values undergo a -log10 transformation to generate these FDR corrected values. Any rows containing "NA" are ommitted from analysis. Data points are colored based on increasing or decreasing value and plotted using ggplot2. Adjusted p-value cutoff is 1.3 following log transformation.(-log10 0.05 = 1.3)

```{r Volcano Plots, echo=TRUE, paged.print=TRUE}
# Gather Log-fold change and FDR-corrected pvalues from DESeq2 results
# Change pvalues to -log10 (1.3 = 0.05)
data <- data.frame(
  gene = (res$symbol),
  pval = -log10(res$padj),
  lfc = res$log2FoldChange
)

data2 <- data.frame(
  gene = (res2$symbol),
  pval = -log10(res2$padj),
  lfc = res2$log2FoldChange
)

data3 <- data.frame(
  gene = (res3$symbol),
  pval = -log10(res3$padj),
  lfc = res3$log2FoldChange
)


# Remove any rows that have NA as an entry
data <- na.omit(data)

data2 <- na.omit(data2)

data3 <- na.omit(data3)

# Color the points which are up or down
## If fold-change > 0 and pvalue > 1.3 (Increased significant)
## If fold-change < 0 and pvalue > 1.3 (Decreased significant)

data <- dplyr::mutate(data, color = case_when(
  data$lfc > 0 & data$pval > 1.3 ~ "Increased in Maintenance",
  data$lfc < 0 & data$pval > 1.3 ~ "Decreased in Maintenance",
  data$pval < 1.3 ~ "Nonsignificant"
))

data2 <- dplyr::mutate(data2, color = case_when(
  data2$lfc > 0 & data2$pval > 1.3 ~ "Increased in Abstinence",
  data2$lfc < 0 & data2$pval > 1.3 ~ "Decreased in Abstinence",
  data2$pval < 1.3 ~ "Nonsignificant"
))

data3 <- dplyr::mutate(data3, color = case_when(
  data3$lfc > 0 & data3$pval > 1.3 ~ "Increased in Abstinence",
  data3$lfc < 0 & data3$pval > 1.3 ~ "Decreased in Abstinence",
  data3$pval < 1.3 ~ "Nonsignificant"
))



# Make a basic ggplot2 object with x-y values
vol <- ggplot(data, aes(x = lfc, y = pval, color = color))

vol2 <- ggplot(data2, aes(x = lfc, y = pval, color = color))

vol3 <- ggplot(data3, aes(x = lfc, y = pval, color = color))



# Add ggplot2 layers

y.axis.text <- element_text(face = "bold", color = "black", size = 7)
x.axis.text <- element_text(face = "bold", color = "black", size = 7)
plot.title.text <- element_text(face = "bold", color = "black", size = 9, hjust = 0.5)
legend.text <- element_text(face = "bold", color = "black", size = 5)
legend.title.text <- element_text(face = "bold", color = "black", size = 6)
axis.title <- element_text(face = "bold", color = "black", size = 6)
```

```{r Volcano Plots -- Visualize, echo=TRUE, message=FALSE, warning=FALSE, paged.print=TRUE}
vol_final <- vol +
  ggtitle(label = "Maintenance vs Saline") +
  geom_point(size = 1.5, alpha = 0.8, na.rm = T) +
  scale_color_manual(
    name = "Directionality",
    values = c("Increased in Maintenance" = "#008B00", "Decreased in Maintenance" = "#CD4F39", "Nonsignificant" = "darkgray")
  ) +
  theme_bw(base_size = 14) + # change overall theme+
  theme(
    legend.position = "right",
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  xlab(expression(log[2]("Fold Change"))) + # Change X-Axis label
  ylab(expression(-log[10]("adjusted p-value"))) + # Change Y-Axis label
  geom_hline(yintercept = 1.3, colour = "darkgrey") + # Add p-adj value cutoff line
  scale_y_continuous(trans = "log1p") # Scale y-axis due to large p-values

vol2_final <- vol2 +
  ggtitle(label = "Abstinence vs Saline") +
  geom_point(size = 1.5, alpha = 0.8, na.rm = T) +
  scale_color_manual(
    name = "Directionality",
    values = c("Increased in Abstinence" = "#008B00", "Decreased in Abstinence" = "#CD4F39", "Nonsignificant" = "darkgray")
  ) +
  theme_bw(base_size = 14) + # change overall theme
  theme(
    legend.position = "right",
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  xlab(expression(log[2]("Fold Change"))) + # Change X-Axis label
  ylab(expression(-log[10]("adjusted p-value"))) + # Change Y-Axis label
  geom_hline(yintercept = 1.3, colour = "darkgrey") + # Add p-adj value cutoff line
  scale_y_continuous(trans = "log1p") # Scale y-axis due to large p-values

vol3_final <- vol3 +
  ggtitle(label = "Abstinence vs Maintenance") +
  geom_point(size = 1.5, alpha = 0.8, na.rm = T) +
  scale_color_manual(
    name = "Directionality",
    values = c("Increased in Abstinence" = "#008B00", "Decreased in Abstinence" = "#CD4F39", "Nonsignificant" = "darkgray")
  ) +
  theme_bw(base_size = 14) + # change overall theme
  theme(
    legend.position = "right",
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  xlab(expression(log[2]("fold change"))) + # Change X-Axis label
  ylab(expression(-log[10]("adjusted p-value"))) + # Change Y-Axis label
  geom_hline(yintercept = 1.3, colour = "darkgrey") + # Add p-adj value cutoff line
  scale_y_continuous(trans = "log1p") # Scale y-axis due to large p-values

ggarrange(vol_final, vol2_final, vol3_final, labels = c("A", "B", "C"))
```

# Pathway Analysis

## Pathway Analysis GO {.tabset}

First, the background for the GO universe must be set. In order to do so we will subset the first results file "res" for any genes that map to entrez IDs and then we only take the unique ones. This leaves us with ~31,000 genes for the remaining background which we list as our microglial expressed genes.

```{r GO Analyses -- Setting up Background, echo=FALSE, message=FALSE, warning=FALSE}
# Generate Background for KEGG/GO
background_mm10 <- subset(res, is.na(entrez) == FALSE)
gene_matrix_background <- unique(sort(background_mm10$entrez))
```

Pathway analysis is conducted using enrichGo and enrichKEGG based on a p-value of 0.05 and a q-value of 0.10 and including all significantly differentially expressed genes.

```{r GO Analyses 1, echo=TRUE, message=FALSE, warning=FALSE}
# Remove any genes that have greater than adj.pvalue 0.05
results_sig_entrez <- subset(results_sig, lfcSE <= 1.5)
results_sig_entrez2 <- subset(results_sig2, lfcSE <= 1.5)
results_sig_entrez3 <- subset(results_sig3, lfcSE <= 1.5)

# Remove any genes that have greater than 1.5 LFC Standard Error
results_sig_entrez <- subset(results_sig_entrez, padj <= 0.05)
results_sig_entrez2 <- subset(results_sig_entrez2, padj <= 0.05)
results_sig_entrez3 <- subset(results_sig_entrez3, padj <= 0.05)

# Remove any genes that do not have any entrez identifiers
results_sig_entrez <- subset(results_sig_entrez, is.na(entrez) == FALSE)
results_sig_entrez2 <- subset(results_sig_entrez2, is.na(entrez) == FALSE)
results_sig_entrez3 <- subset(results_sig_entrez3, is.na(entrez) == FALSE)

# Create a matrix of gene log2 fold changes
gene_matrix <- results_sig_entrez$log2FoldChange
gene_matrix2 <- results_sig_entrez2$log2FoldChange
gene_matrix3 <- results_sig_entrez3$log2FoldChange

# Add the entrezID's as names for each logFC entry
names(gene_matrix) <- results_sig_entrez$entrez
names(gene_matrix2) <- results_sig_entrez2$entrez
names(gene_matrix3) <- results_sig_entrez3$entrez


go <- enrichGO(
  gene = names(gene_matrix),
  OrgDb = "org.Mm.eg.db",
  readable = T,
  universe = gene_matrix_background,
  ont = "ALL",
  pAdjustMethod = "fdr",
  pvalueCutoff = 0.05,
  qvalueCutoff = 0.10
)

go2 <- enrichGO(
  gene = names(gene_matrix2),
  OrgDb = "org.Mm.eg.db",
  readable = T,
  universe = gene_matrix_background,
  ont = "ALL",
  pAdjustMethod = "fdr",
  pvalueCutoff = 0.05,
  qvalueCutoff = 0.10
)

go3 <- enrichGO(
  gene = names(gene_matrix3),
  OrgDb = "org.Mm.eg.db",
  readable = T,
  universe = gene_matrix_background,
  ont = "ALL",
  pAdjustMethod = "fdr",
  pvalueCutoff = 0.05,
  qvalueCutoff = 0.10
)
```

```{r Merging GO For Barplots, eval=FALSE, include=FALSE}
## Merge GO Pathways and Write to CSV

MN_vs_SAL_GO <- as.data.frame(go)
CRV_vs_SAL_GO <- as.data.frame(go2)
CRV_vs_MN_GO <- as.data.frame(go3)

MN_vs_SAL_GO$Comparison <- "MaintenancevsSaline"
CRV_vs_SAL_GO$Comparison <- "AbstinencevsSaline"
CRV_vs_MN_GO$Comparison <- "AbstinencevsMaintenance"


GoMerged_1 <- rbind(MN_vs_SAL_GO, CRV_vs_SAL_GO)
GoMerged_Final <- rbind(GoMerged_1, CRV_vs_MN_GO)
```


```{r Writing Tables GO Results, eval=FALSE, include=FALSE}
write.table(
  x = as.data.frame(GoMerged_Final),
  file = "/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/MethamphetamineIVSA_RNASequencing/output/GO_all_Combined_Comparisons.txt",
  sep = "\t",
  quote = F,
  col.names = NA
)


write.table(
  x = as.data.frame(gene_matrix_background),
  file = "/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/MethamphetamineIVSA_RNASequencing/output/Gene_Matrix_EntrezIDs_BACKGROUND.txt",
  sep = "\t",
  quote = F,
  col.names = NA
)


write.table(
  x = as.data.frame(names(gene_matrix)),
  file = "/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/MethamphetamineIVSA_RNASequencing/output/Gene_Matrix_EntrezIDs_MNvsSAL.txt",
  sep = "\t",
  quote = F,
  col.names = NA
)
write.table(
  x = as.data.frame(names(gene_matrix2)),
  file = "/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/MethamphetamineIVSA_RNASequencing/output/Gene_Matrix_EntrezIDs_ABSvsSAL.txt",
  sep = "\t",
  quote = F,
  col.names = NA
)
write.table(
  x = as.data.frame(names(gene_matrix3)),
  file = "/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/MethamphetamineIVSA_RNASequencing/output/Gene_Matrix_EntrezIDs_ABSvsMN.txt",
  sep = "\t",
  quote = F,
  col.names = NA
)
```

### Maintenance vs Saline

```{r Maintenance vs Saline GO}
kable(head(go, 10), caption = "Top 10 GO Pathways for Maintenance vs Saline", format = "html", align = "l", booktabs = TRUE) %>%
  kable_styling(html_font = "Arial", font_size = 10, bootstrap_options = c("striped", "hover", "condensed", "responsive")) %>%
  scroll_box(width = "1000px", height = "500px")
```


### Abstinence vs Saline 
```{r Abstinence vs Saline GO}
kable(head(go2, 10), caption = "Top 10 GO Pathways for Abstinence vs Saline", format = "html", align = "l", booktabs = TRUE) %>%
  kable_styling(html_font = "Arial", font_size = 10, bootstrap_options = c("striped", "hover", "condensed", "responsive")) %>%
  scroll_box(width = "1000px", height = "500px")
```


### Abstinence vs Maintenance

```{r Abstinence vs Mainteance GO}
kable(head(go3, 10), caption = "Top 10 GO Pathways for Abstinence vs Maintenance", format = "html", align = "l", booktabs = TRUE) %>%
  kable_styling(html_font = "Arial", font_size = 10, bootstrap_options = c("striped", "hover", "condensed", "responsive")) %>%
  scroll_box(width = "1000px", height = "500px")
```

```{r Write GO Pathways to CSV, eval=FALSE, include=FALSE}
# write.table(go, file = "results/PathwayAnalysis/GO_MIVSA_Maintenance_vs_Saline.txt")
# write.table(go2, file = "results/PathwayAnalysis/GO_MIVSA_Abstinence_vs_Saline.txt")
# write.table(go3, file = "results/PathwayAnalysis/GO_MIVSA_Abstinence_vs_Maintenance.txt")

go_df <- as.data.frame(go)
go_df2 <- as.data.frame(go2)
go_df3 <- as.data.frame(go3)

go_df$Comparison <- "MaintenancevsSaline"
go_df2$Comparison <- "AbstinencevsSaline"
go_df3$Comparison <- "AbstinencevsMaintenance"


Go_Combined1 <- rbind(go_df, go_df2)
Go_Combined_final <- rbind(Go_Combined1, go_df3)

# write.csv(Go_Combined_final, file = "results/PathwayAnalysis/GO_MIVSA_PathwaysCombined.csv")
```

```{r eval=FALSE, message=FALSE, warning=FALSE, include=FALSE}
## GO Enrich

write.table(
  x = as.data.frame(go),
  file = "GO_all_Maintenance_vs_Saline.txt",
  sep = "\t",
  quote = F,
  col.names = NA
)
write.table(
  x = as.data.frame(go2),
  file = "GO_all_Abstinence_vs_Saline.txt",
  sep = "\t",
  quote = F,
  col.names = NA
)
write.table(
  x = as.data.frame(go3),
  file = "GO_all_Abstinence_vs_Maintenance.txt",
  sep = "\t",
  quote = F,
  col.names = NA
)

## Merge GO Pathways and Write to CSV

MN_vs_SAL_GO <- as.data.frame(go)
CRV_vs_SAL_GO <- as.data.frame(go2)
CRV_vs_MN_GO <- as.data.frame(go3)

MN_vs_SAL_GO$Comparison <- "Maintenance vs Saline"
CRV_vs_SAL_GO$Comparison <- "Abstinence vs Saline"
CRV_vs_MN_GO$Comparison <- "Abstinence vs Maintenance"


GoMerged_1 <- rbind(MN_vs_SAL_GO, MN_vs_SAL_GO)
GoMerged_Final <- rbind(GoMerged_1, CRV_vs_MN_GO)

write.table(
  x = as.data.frame(GoMerged_Final),
  file = "GO_all_Combined_Comparisons.txt",
  sep = "\t",
  quote = F,
  col.names = NA
)
```


# GO Pathways Visualization {.tabset}
## GO DotPlot
```{r Combined GO DotPlots, echo=FALSE, message=FALSE, warning=FALSE}
library(ggplot2)
library(stringr)

data <- read.csv("/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/final_samples_removed/Filtered_GO_Combined_Final.csv", sep = ",", header = TRUE, stringsAsFactors = FALSE)

# Turn your 'treatment' column into a character vector
data$Comparison <- as.character(data$Comparison)
# Then turn it back into a factor with the levels in the correct order
data$Comparison <- factor(data$Comparison, levels = c("Maintenance vs Saline", "Abstinence vs Saline", "Abstinence vs Maintenance"))

data$Description_title_case <- str_to_title(data$Description)
data$GOPathways <- str_to_title(data$Description_title_case)
data$GOPathways <- str_wrap(data$GOPathways, width = 10)


y.axis.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 6)
x.axis.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 7)
plot.title.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 7, hjust = 0.5)
legend.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 8)
legend.title.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 8)
axis.title <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 9)


S1 <- ggplot(data, aes(x = Comparison, y = Description, size = Count, color = qvalue, group = Comparison)) +
  geom_point(alpha = 0.8) +
  theme_bw() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  scale_x_discrete(labels = function(x) str_wrap(x, width = 15)) +
  scale_y_discrete(labels = function(x) str_wrap(x, width = 50)) +
  ggtitle("Enriched Pathways in Striatal Microglia Through METH Abstinence")

S2 <- S1 + scale_color_gradient(low = "red2", high = "mediumblue", space = "Lab") + scale_size(range = c(1, 5))
S2
```
## Chord Plots
```{r Circle Plots, echo=TRUE, fig.height=15, fig.width=15, message=FALSE, warning=FALSE}
library(GOplot)
library(data.table)

go_circle <- read.delim("/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/final_samples_removed/results/PathwayAnalysis/GO_MIVSA_Maintenance_vs_Saline_circle.txt", sep = "\t")
go2_circle <- read.delim("/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/final_samples_removed/results/PathwayAnalysis/GO_MIVSA_Abstinence_vs_Saline_circle.txt", sep = "\t")
go3_circle <- read.delim("/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/final_samples_removed/results/PathwayAnalysis/GO_MIVSA_Abstinence_vs_Maintenance_circle.txt", sep = "\t")

processes1 <- data[c(1:7), ]

processes1_final_circle <- processes1$Description

ids_final_circle <- processes1$ID


processes2 <- data[c(8:14), ]

processes2_final_circle <- processes2$Description

ids2_final_circle <- processes2$ID


processes3 <- data[c(15:21), ]

processes3_final_circle <- processes3$Description

ids3_final_circle <- processes3$ID



circletest_genes1 <- data.frame(matrix(ncol = 2, nrow = 432))
colnames(circletest_genes1) <- c("Genes", "L2FC")

circletest_DEGsgenes1 <- as.data.table(results_sig)
circletest_DEGsgenes1$ID <- circletest_DEGsgenes1$symbol
circletest_DEGsgenes1$logFC <- circletest_DEGsgenes1$log2FoldChange
circletest_DEGsgenes1$AveExpr <- circletest_DEGsgenes1$baseMean
circletest_DEGsgenes1$t <- circletest_DEGsgenes1$stat
circletest_DEGsgenes1$P.Value <- circletest_DEGsgenes1$pvalue
circletest_DEGsgenes1$adj.P.Value <- circletest_DEGsgenes1$padj
circletest_DEGsgenes1$B <- circletest_DEGsgenes1$stat



circletest_DEGsgenes1_final <- circletest_DEGsgenes1[, 11:17, with = FALSE]

circletest_genes1$Genes <- names(gene_matrix)
circletest_genes1$ID <- mapIds(
  x = org.Mm.eg.db,
  keys = circletest_genes1$Genes,
  column = "SYMBOL",
  keytype = "ENTREZID",
  multiVals = "first"
)
circletest_genes1$logFC <- (gene_matrix)


circletest_genes1 <- mutate_all(circletest_genes1, .funs = toupper)
circletest_genes1_final <- circletest_genes1[, 3:4]
circletest_genes1_final$logFC <- as.numeric(circletest_genes1_final$logFC)



circletest_genes2 <- data.frame(matrix(ncol = 2, nrow = 227))
colnames(circletest_genes2) <- c("Genes", "L2FC")

circletest_DEGsgenes2 <- as.data.table(results_sig2)
circletest_DEGsgenes2$ID <- circletest_DEGsgenes2$symbol
circletest_DEGsgenes2$logFC <- circletest_DEGsgenes2$log2FoldChange
circletest_DEGsgenes2$AveExpr <- circletest_DEGsgenes2$baseMean
circletest_DEGsgenes2$t <- circletest_DEGsgenes2$stat
circletest_DEGsgenes2$P.Value <- circletest_DEGsgenes2$pvalue
circletest_DEGsgenes2$adj.P.Value <- circletest_DEGsgenes2$padj
circletest_DEGsgenes2$B <- circletest_DEGsgenes2$stat



circletest_DEGsgenes2_final <- circletest_DEGsgenes2[, 11:17, with = FALSE]

circletest_genes2$Genes <- names(gene_matrix2)
circletest_genes2$ID <- mapIds(
  x = org.Mm.eg.db,
  keys = circletest_genes2$Genes,
  column = "SYMBOL",
  keytype = "ENTREZID",
  multiVals = "first"
)
circletest_genes2$logFC <- (gene_matrix2)

circletest_genes2 <- mutate_all(circletest_genes2, .funs = toupper)
circletest_genes2_final <- circletest_genes2[, 3:4]
circletest_genes2_final$logFC <- as.numeric(circletest_genes2_final$logFC)





circletest_genes3 <- data.frame(matrix(ncol = 2, nrow = 549))
colnames(circletest_genes3) <- c("Genes", "L2FC")

circletest_DEGsgenes3 <- as.data.table(results_sig3)
circletest_DEGsgenes3$ID <- circletest_DEGsgenes3$symbol
circletest_DEGsgenes3$logFC <- circletest_DEGsgenes3$log2FoldChange
circletest_DEGsgenes3$AveExpr <- circletest_DEGsgenes3$baseMean
circletest_DEGsgenes3$t <- circletest_DEGsgenes3$stat
circletest_DEGsgenes3$P.Value <- circletest_DEGsgenes3$pvalue
circletest_DEGsgenes3$adj.P.Value <- circletest_DEGsgenes3$padj
circletest_DEGsgenes3$B <- circletest_DEGsgenes3$stat



circletest_DEGsgenes3_final <- circletest_DEGsgenes3[, 11:17, with = FALSE]

circletest_genes3$Genes <- names(gene_matrix3)
circletest_genes3$ID <- mapIds(
  x = org.Mm.eg.db,
  keys = circletest_genes3$Genes,
  column = "SYMBOL",
  keytype = "ENTREZID",
  multiVals = "first"
)
circletest_genes3$logFC <- (gene_matrix3)


circletest_genes3 <- mutate_all(circletest_genes3, .funs = toupper)
circletest_genes3_final <- circletest_genes3[, 3:4]
circletest_genes3_final$logFC <- as.numeric(circletest_genes3_final$logFC)
```

```{r echo=TRUE, fig.height=20, fig.width=60, message=FALSE, warning=FALSE}
circ <- circle_dat(go_circle, circletest_DEGsgenes1_final)
circ2 <- circle_dat(go2_circle, circletest_DEGsgenes2_final)
circ3 <- circle_dat(go3_circle, circletest_DEGsgenes3_final)

chord <- chord_dat(circ, circletest_genes1_final, processes1_final_circle)

chord2 <- chord_dat(circ2, circletest_genes2_final, processes2_final_circle)

chord3 <- chord_dat(circ3, circletest_genes3_final, processes3_final_circle)


p5 <- GOChord(chord, space = 0, gene.order = "logFC", gene.space = 0.15, gene.size = 4, nlfc = 1, process.label = 14, limit = c(1, 0), ribbon.col = c("#478FCD", "#FEC111", "#8D268B", "#0FB34C", "#39C0C4", "#F16739", "#607D8B"), border.size = 0, lfc.col = c("#E31A1C", "#33A02C", "#125AD9"), lfc.min = -2, lfc.max = 2)
p6 <- GOChord(chord2, space = 0, gene.order = "logFC", gene.space = 0.15, gene.size = 4, nlfc = 1, process.label = 14, limit = c(1, 0), ribbon.col = c("#478FCD", "#FEC111", "#8D268B", "#0FB34C", "#39C0C4", "#F16739", "#607D8B"), border.size = 0, lfc.col = c("#E31A1C", "#33A02C", "#125AD9"), lfc.min = -2, lfc.max = 2)
p7 <- GOChord(chord3, space = 0, gene.order = "logFC", gene.space = 0.15, gene.size = 4, nlfc = 1, process.label = 14, limit = c(1, 0), ribbon.col = c("#478FCD", "#FEC111", "#8D268B", "#0FB34C", "#39C0C4", "#F16739", "#607D8B"), border.size = 0, lfc.col = c("#E31A1C", "#33A02C", "#125AD9"), lfc.min = -2, lfc.max = 2)



y.axis.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 7)
x.axis.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 7)
plot.title.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 9, hjust = 0.5)
legend.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 6)
legend.title.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 10)
axis.title <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 6)
text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 6)


p8 <- p5 +
  theme(
    legend.position = "bottom",
    legend.text = legend.text,
    legend.title = legend.title.text,
    text = text,
    legend.key.width = unit(3, "cm")
  ) # change legend key width

p9 <- p6 +
  theme(
    legend.position = "bottom",
    legend.text = legend.text,
    legend.title = legend.title.text,
    text = text,
    legend.key.width = unit(3, "cm")
  ) # change legend key width

p10 <- p7 +
  theme(
    legend.position = "bottom",
    legend.text = legend.text,
    legend.title = legend.title.text,
    text = text,
    legend.key.width = unit(3, "cm")
  ) # change legend key width


cowplot::plot_grid(p8, p9, p10,
  ncol = 3, labels = c("A. Maintenance vs Saline", "B. Abstinence vs Saline", "C. Abstinence vs Maintenance"),
  label_fontfamily = "arial",
  label_fontface = "bold",
  label_colour = "black",
  label_size = 50
)
```

```{r eval=FALSE, include=FALSE}
p5
p6
p7
```

```{r Write Circularized Files to txt, eval=FALSE, include=FALSE}
write.table(chord, file = "results/PathwayAnalysis/GOChord_MIVSA_Maintenance_vs_Saline.txt")
write.table(chord2, file = "results/PathwayAnalysis/GOChord_MIVSA_Abstinence_vs_Saline.txt")
write.table(chord3, file = "results/PathwayAnalysis/GOChord_Abstinence_vs_Maintenance.txt")
```

# Determining Directionality of GOPathways {.tabset}

```{r Setting up updown Barplots from GO, include=FALSE}
# Remove any genes that have greater than 1.5 LFC Standard Error
df <- subset(results_sig, lfcSE <= 1.50)
df2 <- subset(results_sig2, lfcSE <= 1.50)
df3 <- subset(results_sig3, lfcSE <= 1.50)


# Remove any genes that do not have any entrez identifiers
df <- subset(df, is.na(entrez) == FALSE)
df2 <- subset(df2, is.na(entrez) == FALSE)
df3 <- subset(df3, is.na(entrez) == FALSE)


# Create a matrix of gene log2 fold changes
df_L2FC <- as.data.frame(df)
df2_L2FC <- as.data.frame(df2)
df3_L2FC <- as.data.frame(df3)

df_L2FC_values <- df_L2FC %>%
  mutate(Expression = ifelse(log2FoldChange > 0, "Upregulated", "Downregulated"))
df2_L2FC_values <- df2_L2FC %>%
  mutate(Expression = ifelse(log2FoldChange > 0, "Upregulated", "Downregulated"))
df3_L2FC_values <- df3_L2FC %>%
  mutate(Expression = ifelse(log2FoldChange > 0, "Upregulated", "Downregulated"))
```

```{r Making updown barplots from GO, fig.height=10, fig.width=10, include=FALSE}
barplots <- read.csv("/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/MethamphetamineIVSA_RNASequencing/data/Comparison_GO_GeneID_barplots.csv")
```

```{r Maintenance vs Saline GO Plots of Barplots Gene L2FC, fig.height=24, fig.width=24, include=FALSE}
## Maintenance vs Saline
barplots$Comparison
barplots_MNvsSAL <- barplots[1:166, ]
barplots_MNvsSAL$ID
barplots_MNvsSAL$geneID <- barplots_MNvsSAL$GeneID
barplots_MNvsSAL1 <- barplots_MNvsSAL[1:18, ]
barplots_MNvsSAL2 <- barplots_MNvsSAL[19:50, ]
barplots_MNvsSAL3 <- barplots_MNvsSAL[51:69, ]
barplots_MNvsSAL4 <- barplots_MNvsSAL[70:89, ]
barplots_MNvsSAL5 <- barplots_MNvsSAL[90:117, ]
barplots_MNvsSAL6 <- barplots_MNvsSAL[118:132, ]
barplots_MNvsSAL7 <- barplots_MNvsSAL[133:146, ]
barplots_MNvsSAL8 <- barplots_MNvsSAL[147:166, ]

rows1 <- barplots_MNvsSAL$geneID
rows1

rows2 <- barplots_MNvsSAL1$geneID
rows2

rows3 <- barplots_MNvsSAL2$geneID
rows3

rows4 <- barplots_MNvsSAL3$geneID
rows4

rows5 <- barplots_MNvsSAL4$geneID
rows5

rows6 <- barplots_MNvsSAL5$geneID
rows6

rows7 <- barplots_MNvsSAL6$geneID
rows7

rows8 <- barplots_MNvsSAL7$geneID
rows8

rows9 <- barplots_MNvsSAL8$geneID
rows9

data_mod_barplots_MNvsSAL <- df_L2FC_values[df_L2FC_values$symbol %in% rows1, ]
# Replicate original data
data_mod_barplots_MNvsSAL$symbol <- factor(data_mod_barplots_MNvsSAL$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_MNvsSAL$symbol[order(data_mod_barplots_MNvsSAL$log2FoldChange, decreasing = FALSE)]
)


data_mod_barplots_MNvsSAL1 <- df_L2FC_values[df_L2FC_values$symbol %in% rows2, ]

data_mod_barplots_MNvsSAL1$symbol <- factor(data_mod_barplots_MNvsSAL1$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_MNvsSAL1$symbol[order(data_mod_barplots_MNvsSAL1$log2FoldChange, decreasing = FALSE)]
)


data_mod_barplots_MNvsSAL2 <- df_L2FC_values[df_L2FC_values$symbol %in% rows3, ]

data_mod_barplots_MNvsSAL2$symbol <- factor(data_mod_barplots_MNvsSAL2$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_MNvsSAL2$symbol[order(data_mod_barplots_MNvsSAL2$log2FoldChange, decreasing = FALSE)]
)

data_mod_barplots_MNvsSAL3 <- df_L2FC_values[df_L2FC_values$symbol %in% rows4, ]

data_mod_barplots_MNvsSAL3$symbol <- factor(data_mod_barplots_MNvsSAL3$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_MNvsSAL3$symbol[order(data_mod_barplots_MNvsSAL3$log2FoldChange, decreasing = FALSE)]
)

data_mod_barplots_MNvsSAL4 <- df_L2FC_values[df_L2FC_values$symbol %in% rows5, ]

data_mod_barplots_MNvsSAL4$symbol <- factor(data_mod_barplots_MNvsSAL4$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_MNvsSAL4$symbol[order(data_mod_barplots_MNvsSAL4$log2FoldChange, decreasing = FALSE)]
)

data_mod_barplots_MNvsSAL5 <- df_L2FC_values[df_L2FC_values$symbol %in% rows6, ]

data_mod_barplots_MNvsSAL5$symbol <- factor(data_mod_barplots_MNvsSAL5$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_MNvsSAL5$symbol[order(data_mod_barplots_MNvsSAL5$log2FoldChange, decreasing = FALSE)]
)

data_mod_barplots_MNvsSAL6 <- df_L2FC_values[df_L2FC_values$symbol %in% rows7, ]

data_mod_barplots_MNvsSAL6$symbol <- factor(data_mod_barplots_MNvsSAL6$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_MNvsSAL6$symbol[order(data_mod_barplots_MNvsSAL6$log2FoldChange, decreasing = FALSE)]
)

data_mod_barplots_MNvsSAL7 <- df_L2FC_values[df_L2FC_values$symbol %in% rows8, ]

data_mod_barplots_MNvsSAL7$symbol <- factor(data_mod_barplots_MNvsSAL7$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_MNvsSAL7$symbol[order(data_mod_barplots_MNvsSAL7$log2FoldChange, decreasing = FALSE)]
)

data_mod_barplots_MNvsSAL8 <- df_L2FC_values[df_L2FC_values$symbol %in% rows9, ]

data_mod_barplots_MNvsSAL8$symbol <- factor(data_mod_barplots_MNvsSAL8$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_MNvsSAL8$symbol[order(data_mod_barplots_MNvsSAL8$log2FoldChange, decreasing = FALSE)]
)



### Plot

y.axis.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 8)
x.axis.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 6)
plot.title.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 6, hjust = 0.5)
legend.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 5)
legend.title.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 5)
axis.title <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 10)

p <- ggplot(data = data_mod_barplots_MNvsSAL, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in All Pathways (Maintenance vs Saline)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Maintenance over Saline)")


p1 <- ggplot(data = data_mod_barplots_MNvsSAL1, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Oligodendrocyte differentiation (GO:0048709) (Maintenance vs Saline)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Maintenance over Saline)")

# p1

p2 <- ggplot(data = data_mod_barplots_MNvsSAL2, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in mRNA Processing (GO:0006397) (Maintenance vs Saline)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Maintenance over Saline)")

# p2

p3 <- ggplot(data = data_mod_barplots_MNvsSAL3, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Protein folding (GO:0006457) (Maintenance vs Saline)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Maintenance over Saline)")

# p3

cowplot::plot_grid(p, p1, p2, p3, ncol = 2, labels = "AUTO", label_size = 14)



p4 <- ggplot(data = data_mod_barplots_MNvsSAL4, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Glial cell differentiation (GO:0010001) (Maintenance vs Saline)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Maintenance over Saline)")


p5 <- ggplot(data = data_mod_barplots_MNvsSAL5, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Chromosome organization (GO:0051276) (Maintenance vs Saline)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Maintenance over Saline)")


p6 <- ggplot(data = data_mod_barplots_MNvsSAL6, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Negative regulation of supramolecular fiber organization (GO:1902904) (Maintenance vs Saline)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Maintenance over Saline)")

p7 <- ggplot(data = data_mod_barplots_MNvsSAL7, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Regulation of microtubule cytoskeleton organization (GO:0070507) (Maintenance vs Saline)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Maintenance over Saline)")

p8 <- ggplot(data = data_mod_barplots_MNvsSAL8, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Immune response-regulating signaling pathway (GO:0002764) (Maintenance vs Saline)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Maintenance over Saline)")



## Combined Plots

data_mod_barplots_MNvsSAL1$Pathway <- "Oligodendrocyte differentiation"
data_mod_barplots_MNvsSAL2$Pathway <- "mRNA Processing"
data_mod_barplots_MNvsSAL3$Pathway <- "Protein folding"
data_mod_barplots_MNvsSAL4$Pathway <- "Glial cell differentiation"
data_mod_barplots_MNvsSAL5$Pathway <- "Chromosome organization"
data_mod_barplots_MNvsSAL6$Pathway <- "Negative regulation of supramolecular fiber organization"
data_mod_barplots_MNvsSAL7$Pathway <- "Regulation of microtubule cytoskeleton organization"
data_mod_barplots_MNvsSAL8$Pathway <- "Immune response regulating signaling pathway"


data_combined_barplot_MNvsSAL_1 <- rbind(data_mod_barplots_MNvsSAL1, data_mod_barplots_MNvsSAL2)
data_combined_barplot_MNvsSAL_2 <- rbind(data_combined_barplot_MNvsSAL_1, data_mod_barplots_MNvsSAL3)
data_combined_barplot_MNvsSAL_3 <- rbind(data_combined_barplot_MNvsSAL_2, data_mod_barplots_MNvsSAL4)
data_combined_barplot_MNvsSAL_4 <- rbind(data_combined_barplot_MNvsSAL_3, data_mod_barplots_MNvsSAL5)
data_combined_barplot_MNvsSAL_5 <- rbind(data_combined_barplot_MNvsSAL_4, data_mod_barplots_MNvsSAL6)
data_combined_barplot_MNvsSAL_6 <- rbind(data_combined_barplot_MNvsSAL_5, data_mod_barplots_MNvsSAL7)
data_combined_barplot_MNvsSAL <- rbind(data_combined_barplot_MNvsSAL_6, data_mod_barplots_MNvsSAL8)


plot.title.text.all <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 14, hjust = 0.5)

p9 <- ggplot(data = data_combined_barplot_MNvsSAL, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text.all,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Pathways  (Maintenance vs Saline)") +
  coord_flip() +
  labs(y = "Gene Name", x = "log2FoldChange (Maintenance over Saline)") +
  facet_wrap(~Pathway, scales = "free")
# p6



cowplot::plot_grid(p1, p2, p3, p4, ncol = 2, labels = "AUTO", label_size = 14)

cowplot::plot_grid(p5, p6, p7, p8, ncol = 2, labels = "AUTO", label_size = 14)

p9
```

```{r Abstinence vs Saline GO Plots of Barplots Gene L2FC, fig.height=7, fig.width=7, include=FALSE}
## Abstinence vs Saline
barplots$Comparison
barplots_ABSvsSAL <- barplots[167:247, ]
barplots_ABSvsSAL$ID
barplots$geneID <- barplots$GeneID
barplots_ABSvsSAL1 <- barplots[167:186, ]
barplots_ABSvsSAL2 <- barplots[187:206, ]
barplots_ABSvsSAL3 <- barplots[208:218, ]
barplots_ABSvsSAL4 <- barplots[219:228, ]
barplots_ABSvsSAL5 <- barplots[229:242, ]
barplots_ABSvsSAL6 <- barplots[243:247, ]


rows1 <- barplots_ABSvsSAL$geneID
rows1

rows2 <- barplots_ABSvsSAL1$geneID
rows2

rows3 <- barplots_ABSvsSAL2$geneID
rows3

rows4 <- barplots_ABSvsSAL3$geneID
rows4

rows5 <- barplots_ABSvsSAL4$geneID
rows5

rows6 <- barplots_ABSvsSAL5$geneID
rows6

rows7 <- barplots_ABSvsSAL6$geneID
rows7

data_mod_barplots_ABSvsSAL <- df2_L2FC_values[df2_L2FC_values$symbol %in% rows1, ]
# Replicate original data
data_mod_barplots_ABSvsSAL$symbol <- factor(data_mod_barplots_ABSvsSAL$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_ABSvsSAL$symbol[order(data_mod_barplots_ABSvsSAL$log2FoldChange, decreasing = FALSE)]
)


data_mod_barplots_ABSvsSAL1 <- df2_L2FC_values[df2_L2FC_values$symbol %in% rows2, ]

data_mod_barplots_ABSvsSAL1$symbol <- factor(data_mod_barplots_ABSvsSAL1$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_ABSvsSAL1$symbol[order(data_mod_barplots_ABSvsSAL1$log2FoldChange, decreasing = FALSE)]
)


data_mod_barplots_ABSvsSAL2 <- df2_L2FC_values[df2_L2FC_values$symbol %in% rows3, ]

data_mod_barplots_ABSvsSAL2$symbol <- factor(data_mod_barplots_ABSvsSAL2$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_ABSvsSAL2$symbol[order(data_mod_barplots_ABSvsSAL2$log2FoldChange, decreasing = FALSE)]
)

data_mod_barplots_ABSvsSAL3 <- df2_L2FC_values[df2_L2FC_values$symbol %in% rows4, ]

data_mod_barplots_ABSvsSAL3$symbol <- factor(data_mod_barplots_ABSvsSAL3$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_ABSvsSAL3$symbol[order(data_mod_barplots_ABSvsSAL3$log2FoldChange, decreasing = FALSE)]
)

data_mod_barplots_ABSvsSAL4 <- df2_L2FC_values[df2_L2FC_values$symbol %in% rows5, ]

data_mod_barplots_ABSvsSAL4$symbol <- factor(data_mod_barplots_ABSvsSAL4$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_ABSvsSAL4$symbol[order(data_mod_barplots_ABSvsSAL4$log2FoldChange, decreasing = FALSE)]
)

data_mod_barplots_ABSvsSAL5 <- df2_L2FC_values[df2_L2FC_values$symbol %in% rows6, ]

data_mod_barplots_ABSvsSAL5$symbol <- factor(data_mod_barplots_ABSvsSAL5$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_ABSvsSAL5$symbol[order(data_mod_barplots_ABSvsSAL5$log2FoldChange, decreasing = FALSE)]
)

data_mod_barplots_ABSvsSAL6 <- df2_L2FC_values[df2_L2FC_values$symbol %in% rows7, ]

data_mod_barplots_ABSvsSAL6$symbol <- factor(data_mod_barplots_ABSvsSAL6$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_ABSvsSAL6$symbol[order(data_mod_barplots_ABSvsSAL6$log2FoldChange, decreasing = FALSE)]
)


data_mod_barplots_ABSvsSAL1$Pathway <- "Oligodendrocyte differentiation"
data_mod_barplots_ABSvsSAL2$Pathway <- "Synapse organization"
data_mod_barplots_ABSvsSAL3$Pathway <- "Regulation of nervous system process"
data_mod_barplots_ABSvsSAL4$Pathway <- "Protein folding"
data_mod_barplots_ABSvsSAL5$Pathway <- "Cell-substrate adhesion"
data_mod_barplots_ABSvsSAL6$Pathway <- "Microglial cell activation"

### Plot

y.axis.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 8)
x.axis.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 6)
plot.title.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 6, hjust = 0.5)
legend.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 5)
legend.title.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 5)
axis.title <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 10)

p10 <- ggplot(data = data_mod_barplots_ABSvsSAL, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in All Pathways (Abstinence vs Saline)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Abstinence over Saline)")


p11 <- ggplot(data = data_mod_barplots_ABSvsSAL1, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Oligocyte differentiation (GO:0048709) (Abstinence vs Saline)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Abstinence over Saline)")

# p11

p12 <- ggplot(data = data_mod_barplots_ABSvsSAL2, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Synapse organization (GO:0050808) (Abstinence vs Saline)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Abstinence over Saline)")

# p12

p13 <- ggplot(data = data_mod_barplots_ABSvsSAL3, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Regulation of nervous system process (GO:0031644) (Abstinence vs Saline)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Abstinence over Saline)")

# p13


p14 <- ggplot(data = data_mod_barplots_ABSvsSAL4, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Protein folding (GO:0006457) (Abstinence vs Saline)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Abstinence over Saline)")



p15 <- ggplot(data = data_mod_barplots_ABSvsSAL5, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Cell-substrate adhesion (GO:0031589) (Abstinence vs Saline)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Abstinence over Saline)")


p16 <- ggplot(data = data_mod_barplots_ABSvsSAL6, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Microglial cell activation (GO:0001774) (Abstinence vs Saline)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Abstinence over Saline)")


## Combined Plots

data_combined_barplot_ABSvsSAL_1 <- rbind(data_mod_barplots_ABSvsSAL1, data_mod_barplots_ABSvsSAL2)
data_combined_barplot_ABSvsSAL_2 <- rbind(data_combined_barplot_ABSvsSAL_1, data_mod_barplots_ABSvsSAL3)
data_combined_barplot_ABSvsSAL_3 <- rbind(data_combined_barplot_ABSvsSAL_2, data_mod_barplots_ABSvsSAL4)
data_combined_barplot_ABSvsSAL_4 <- rbind(data_combined_barplot_ABSvsSAL_3, data_mod_barplots_ABSvsSAL5)
data_combined_barplot_ABSvsSAL <- rbind(data_combined_barplot_ABSvsSAL_4, data_mod_barplots_ABSvsSAL6)

p17 <- ggplot(data = data_combined_barplot_ABSvsSAL, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text.all,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Pathways  (Abstinence vs Saline)") +
  coord_flip() +
  labs(y = "Gene Name", x = "log2FoldChange (Abstinence over Saline)") +
  facet_wrap(~Pathway, scales = "free")
# p6



cowplot::plot_grid(p11, p12, p13, p14, ncol = 2, labels = "AUTO", label_size = 14)

cowplot::plot_grid(p15, p16, ncol = 2, labels = "AUTO", label_size = 14)

p17
```

```{r Abstinence vs Maintence GO Plots of Barplots Gene L2FC, include=FALSE}
## Abstinence vs Maintenance
barplots$Comparison
barplots_ABSvsMN <- barplots[248:441, ]
barplots_ABSvsMN$ID
barplots$geneID <- barplots$GeneID
barplots_ABSvsMN1 <- barplots[248:279, ]
barplots_ABSvsMN2 <- barplots[280:299, ]
barplots_ABSvsMN3 <- barplots[300:324, ]
barplots_ABSvsMN4 <- barplots[325:354, ]
barplots_ABSvsMN5 <- barplots[355:384, ]
barplots_ABSvsMN6 <- barplots[385:412, ]
barplots_ABSvsMN7 <- barplots[413:441, ]


rows1 <- barplots_ABSvsMN$geneID
rows1

rows2 <- barplots_ABSvsMN1$geneID
rows2

rows3 <- barplots_ABSvsMN2$geneID
rows3

rows4 <- barplots_ABSvsMN3$geneID
rows4

rows5 <- barplots_ABSvsMN4$geneID
rows5

rows6 <- barplots_ABSvsMN5$geneID
rows6

rows7 <- barplots_ABSvsMN6$geneID
rows7

rows8 <- barplots_ABSvsMN7$geneID
rows8

data_mod_barplots_ABSvsMN <- df3_L2FC_values[df3_L2FC_values$symbol %in% rows1, ]
# Replicate original data
data_mod_barplots_ABSvsMN$symbol <- factor(data_mod_barplots_ABSvsMN$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_ABSvsMN$symbol[order(data_mod_barplots_ABSvsMN$log2FoldChange, decreasing = FALSE)]
)


data_mod_barplots_ABSvsMN1 <- df3_L2FC_values[df3_L2FC_values$symbol %in% rows2, ]

data_mod_barplots_ABSvsMN1$symbol <- factor(data_mod_barplots_ABSvsMN1$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_ABSvsMN1$symbol[order(data_mod_barplots_ABSvsMN1$log2FoldChange, decreasing = FALSE)]
)


data_mod_barplots_ABSvsMN2 <- df3_L2FC_values[df3_L2FC_values$symbol %in% rows3, ]

data_mod_barplots_ABSvsMN2$symbol <- factor(data_mod_barplots_ABSvsMN2$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_ABSvsMN2$symbol[order(data_mod_barplots_ABSvsMN2$log2FoldChange, decreasing = FALSE)]
)

data_mod_barplots_ABSvsMN3 <- df3_L2FC_values[df3_L2FC_values$symbol %in% rows4, ]

data_mod_barplots_ABSvsMN3$symbol <- factor(data_mod_barplots_ABSvsMN3$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_ABSvsMN3$symbol[order(data_mod_barplots_ABSvsMN3$log2FoldChange, decreasing = FALSE)]
)

data_mod_barplots_ABSvsMN4 <- df3_L2FC_values[df3_L2FC_values$symbol %in% rows5, ]

data_mod_barplots_ABSvsMN4$symbol <- factor(data_mod_barplots_ABSvsMN4$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_ABSvsMN4$symbol[order(data_mod_barplots_ABSvsMN4$log2FoldChange, decreasing = FALSE)]
)

data_mod_barplots_ABSvsMN5 <- df3_L2FC_values[df3_L2FC_values$symbol %in% rows6, ]

data_mod_barplots_ABSvsMN5$symbol <- factor(data_mod_barplots_ABSvsMN5$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_ABSvsMN5$symbol[order(data_mod_barplots_ABSvsMN5$log2FoldChange, decreasing = FALSE)]
)

data_mod_barplots_ABSvsMN6 <- df3_L2FC_values[df3_L2FC_values$symbol %in% rows7, ]

data_mod_barplots_ABSvsMN6$symbol <- factor(data_mod_barplots_ABSvsMN6$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_ABSvsMN6$symbol[order(data_mod_barplots_ABSvsMN6$log2FoldChange, decreasing = FALSE)]
)

data_mod_barplots_ABSvsMN7 <- df3_L2FC_values[df3_L2FC_values$symbol %in% rows8, ]

data_mod_barplots_ABSvsMN7$symbol <- factor(data_mod_barplots_ABSvsMN7$symbol, # Factor levels in decreasing order
  levels = data_mod_barplots_ABSvsMN7$symbol[order(data_mod_barplots_ABSvsMN7$log2FoldChange, decreasing = FALSE)]
)



data_mod_barplots_ABSvsMN1$Pathway <- "Immune response-regulating signaling pathway"
data_mod_barplots_ABSvsMN2$Pathway <- "Protein folding"
data_mod_barplots_ABSvsMN3$Pathway <- "Intrinsic apoptotic signaling pathway"
data_mod_barplots_ABSvsMN4$Pathway <- "Neuron death"
data_mod_barplots_ABSvsMN5$Pathway <- "mRNA Processing"
data_mod_barplots_ABSvsMN6$Pathway <- "Response to radiation"
data_mod_barplots_ABSvsMN7$Pathway <- "Chromosome organization"

### Plot

y.axis.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 8)
x.axis.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 6)
plot.title.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 6, hjust = 0.5)
legend.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 5)
legend.title.text <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 5)
axis.title <- element_text(family = "NimbusSanCond", face = "bold", color = "black", size = 10)

p18 <- ggplot(data = data_mod_barplots_ABSvsMN, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in All Pathways (Abstinence vs Maintenance)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Abstinence over Maintenance)")


p19 <- ggplot(data = data_mod_barplots_ABSvsMN1, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Immune response-regulating signaling pathway (GO:0002764) (Abstinence vs Maintenance)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Abstinence over Maintenance)")

# p19

p20 <- ggplot(data = data_mod_barplots_ABSvsMN2, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Protein folding (GO:0006457) (Abstinence vs Maintenance)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Abstinence over Maintenance)")

# p20

p21 <- ggplot(data = data_mod_barplots_ABSvsMN3, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Intrinsic apoptotic signlaing pathway (GO:0097193) (Abstinence vs Maintenance)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Abstinence over Maintenance)")

# p21


p22 <- ggplot(data = data_mod_barplots_ABSvsMN4, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Neuron death (GO:0070997) (Abstinence vs Maintenance)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Abstinence over Maintenance)")



p23 <- ggplot(data = data_mod_barplots_ABSvsMN5, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in mRNA processing (GO:0006397) (Abstinence vs Maintenance)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Abstinence over Maintenance)")


p24 <- ggplot(data = data_mod_barplots_ABSvsMN6, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Response to radiation (GO:0009314) (Abstinence vs Maintenance)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Abstinence over Maintenance)")


p25 <- ggplot(data = data_mod_barplots_ABSvsMN7, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Chromosome organization (GO:0051276) (Abstinence vs Maintenance)") +
  coord_flip() +
  labs(x = "Gene Name", y = "log2FoldChange (Abstinence over Maintenance)")

## Combined Plots

data_combined_barplot_ABSvsMN_1 <- rbind(data_mod_barplots_ABSvsMN1, data_mod_barplots_ABSvsMN2)
data_combined_barplot_ABSvsMN_2 <- rbind(data_combined_barplot_ABSvsMN_1, data_mod_barplots_ABSvsMN3)
data_combined_barplot_ABSvsMN_3 <- rbind(data_combined_barplot_ABSvsMN_2, data_mod_barplots_ABSvsMN4)
data_combined_barplot_ABSvsMN_4 <- rbind(data_combined_barplot_ABSvsMN_3, data_mod_barplots_ABSvsMN5)
data_combined_barplot_ABSvsMN_5 <- rbind(data_combined_barplot_ABSvsMN_4, data_mod_barplots_ABSvsMN6)
data_combined_barplot_ABSvsMN <- rbind(data_combined_barplot_ABSvsMN_5, data_mod_barplots_ABSvsMN7)

p26 <- ggplot(data = data_combined_barplot_ABSvsMN, aes(x = symbol, y = log2FoldChange, color = Expression, fill = Expression)) +
  geom_bar(stat = "identity", width = 0.8) +
  scale_color_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  scale_fill_manual(breaks = c("Downregulated", "Upregulated"), values = c("red", "green")) +
  theme_classic() +
  theme(
    axis.text.y = y.axis.text,
    axis.text.x = x.axis.text,
    axis.title = axis.title,
    plot.title = plot.title.text.all,
    legend.text = legend.text,
    legend.title = legend.title.text,
    legend.key.size = unit(0.5, "cm"), # change legend key size
    legend.key.height = unit(0.5, "cm"), # change legend key height
    legend.key.width = unit(0.5, "cm")
  ) + # change legend key width
  ggtitle("Log2Fold Change of Genes in Pathways  (Abstinence vs Maintenance)") +
  coord_flip() +
  labs(y = "Gene Name", x = "log2FoldChange (Abstinence over Maintenance)") +
  facet_wrap(~Pathway, scales = "free")
# p6

cowplot::plot_grid(p19, p20, p21, p22, ncol = 2, labels = "AUTO", label_size = 14)

cowplot::plot_grid(p23, p24, p25, ncol = 2, labels = "AUTO", label_size = 14)

p26
```

## Log2FoldChange of Genes in Maintenance vs Saline Gene Ontologies

```{r Barplots Maintenance vs Saline, fig.height=6, fig.width=12}
p9
```

## Log2FoldChange of Genes in Abstinence vs Saline Gene Ontologies

```{r Barplots Abstinence vs Saline, fig.height=6, fig.width=12}
p17
```

## Log2FoldChange of Genes in Abstinence vs Maintenance Gene Ontologies

```{r Barplots Abstinence vs Maintenance, fig.height=6, fig.width=12}
p26
```


```{r PDF save overall plots -- barplots, eval=FALSE, fig.height=14, fig.width=14, include=FALSE}
pdf(file = "/home/avm27/Documents/Raw_Sequencing_Data/MIVSA/MIVSA_R/MethamphetamineIVSA_RNASequencing/output/GOBarplots_MIVSA.pdf", width = 25, height = 14)
cowplot::plot_grid(p9, p17, p26, ncol = 2, labels = "AUTO", label_size = 12)
dev.off()
```

# Session Info

```{r Print Session Info, echo=TRUE, message=TRUE, warning=FALSE}
sessionInfo()
```
